<!DOCTYPE html>
<html lang="en"><head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="twitter:card" content="summary_large_image" /><!-- Begin Jekyll SEO tag v2.6.1 -->
<title>[Paper List]Optimization for Biological Networks | 肥叉不吃肥叉</title>
<meta name="generator" content="Jekyll v4.1.1" />
<meta property="og:title" content="[Paper List]Optimization for Biological Networks" />
<meta name="author" content="BjChacha" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Only BMC Bioinformatics" />
<meta property="og:description" content="Only BMC Bioinformatics" />
<link rel="canonical" href="https://bjxx.ltd/translations/deeplearning/2020/09/26/papers-BMC-optimization-for-biological-networks.html" />
<meta property="og:url" content="https://bjxx.ltd/translations/deeplearning/2020/09/26/papers-BMC-optimization-for-biological-networks.html" />
<meta property="og:site_name" content="肥叉不吃肥叉" />
<meta property="og:type" content="article" />
<meta property="article:published_time" content="2020-09-26T00:00:00-05:00" />
<script type="application/ld+json">
{"url":"https://bjxx.ltd/translations/deeplearning/2020/09/26/papers-BMC-optimization-for-biological-networks.html","@type":"BlogPosting","headline":"[Paper List]Optimization for Biological Networks","dateModified":"2020-09-26T00:00:00-05:00","datePublished":"2020-09-26T00:00:00-05:00","mainEntityOfPage":{"@type":"WebPage","@id":"https://bjxx.ltd/translations/deeplearning/2020/09/26/papers-BMC-optimization-for-biological-networks.html"},"author":{"@type":"Person","name":"BjChacha"},"description":"Only BMC Bioinformatics","@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->
<link rel="stylesheet" href="/assets/css/style.css"><link type="application/atom+xml" rel="alternate" href="https://bjxx.ltd/feed.xml" title="肥叉不吃肥叉" /><!-- the google_analytics_id gets auto inserted from the config file -->



<script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','//www.google-analytics.com/analytics.js','ga');ga('create','UA-160825107-1','auto');ga('require','displayfeatures');ga('send','pageview');</script>

<link rel="shortcut icon" type="image/x-icon" href="/images/favicon.ico"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/Primer/15.2.0/primer.css" integrity="sha512-xTz2ys4coGAOz8vuV1NcQBkgVmKhsSEtjbqyMJbBHRplFuvKIUo6xhLHpAyPt9mfR6twHJgn9OgVLuqOvjeBhg==" crossorigin="anonymous" />
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css" integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==" crossorigin="anonymous" />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.12.0/katex.min.css" integrity="sha512-h7nl+xz8wgDlNM4NqKEM4F1NkIRS17M9+uJwIGwuo8vGqIl4BhuCKdxjWEINm+xyrUjNCnK5dCrhM0sj+wTIXw==" crossorigin="anonymous" />
    <script src="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.12.0/katex.min.js" integrity="sha512-/CMIhXiDA3m2c9kzRyd97MTb3MC6OVnx4TElQ7fkkoRghwDf6gi41gaT1PwF270W6+J60uTmwgeRpNpJdRV6sg==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.12.0/contrib/auto-render.min.js" integrity="sha512-Do7uJAaHZm5OLrIv/yN4w0iG1dbu01kzdMNnFfu/mAqgUk6Nniv2JYHcwH+cNwjqgLcqcuBBk+JRvprLVI8azg==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha512-0doc9hKxR3PYwso42RD1p5ySZpzzuDiOwMrdCEh2WdJZCjcmFKc/wEnL+z8fBQrnHoiNWbo+3fiGkOYXBdQp4A==" crossorigin="anonymous"></script>
    <script>
    document.addEventListener("DOMContentLoaded", function() {
        renderMathInElement( document.body, {
        delimiters: [
            {left: "$$", right: "$$", display: true},
            {left: "[%", right: "%]", display: true},
            {left: "$", right: "$", display: false}
        ]}
        );
    });
    </script>


<script>
function wrap_img(fn) {
    if (document.attachEvent ? document.readyState === "complete" : document.readyState !== "loading") {
        var elements = document.querySelectorAll(".post img");
        Array.prototype.forEach.call(elements, function(el, i) {
            if (el.getAttribute("title") && (el.className != "emoji")) {
                const caption = document.createElement('figcaption');
                var node = document.createTextNode(el.getAttribute("title"));
                caption.appendChild(node);
                const wrapper = document.createElement('figure');
                wrapper.className = 'image';
                el.parentNode.insertBefore(wrapper, el);
                el.parentNode.removeChild(el);
                wrapper.appendChild(el);
                wrapper.appendChild(caption);
            }
        });
    } else { document.addEventListener('DOMContentLoaded', fn); }
}
window.onload = wrap_img;
</script>

<script>
    document.addEventListener("DOMContentLoaded", function(){
    // add link icon to anchor tags
    var elem = document.querySelectorAll(".anchor-link")
    elem.forEach(e => (e.innerHTML = '<i class="fas fa-link fa-xs"></i>'));
    });
</script>
</head>
<body><header class="site-header">

  <div class="wrapper"><a class="site-title" rel="author" href="/">肥叉不吃肥叉</a><nav class="site-nav">
        <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.032C17.335,0,18,0.665,18,1.484L18,1.484z M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.032C17.335,6.031,18,6.696,18,7.516L18,7.516z M18,13.516C18,14.335,17.335,15,16.516,15H1.484 C0.665,15,0,14.335,0,13.516l0,0c0-0.82,0.665-1.483,1.484-1.483h15.032C17.335,12.031,18,12.695,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

        <div class="trigger"><a class="page-link" href="/about/">About Me</a><a class="page-link" href="/search/">Search</a><a class="page-link" href="/categories/">Tags</a></div>
      </nav></div>
</header>
<main class="page-content" aria-label="Content">
      <div class="wrapper">
        <article class="post h-entry" itemscope itemtype="http://schema.org/BlogPosting">

  <header class="post-header">
    <h1 class="post-title p-name" itemprop="name headline">[Paper List]Optimization for Biological Networks</h1><p class="page-description">Only BMC Bioinformatics</p><p class="post-meta post-meta-title"><time class="dt-published" datetime="2020-09-26T00:00:00-05:00" itemprop="datePublished">
        Sep 26, 2020
      </time>• 
          <span itemprop="author" itemscope itemtype="http://schema.org/Person">
            <span class="p-author h-card" itemprop="name">BjChacha</span></span>
       • <span class="read-time" title="Estimated read time">
    
    
      85 min read
    
</span></p>

    
      <p class="category-tags"><i class="fas fa-tags category-tags-icon"></i></i> 
      
        <a class="category-tags-link" href="/categories/#translations">translations</a>
        &nbsp;
      
        <a class="category-tags-link" href="/categories/#deeplearning">deeplearning</a>
        
      
      </p>
    

    </header>

  <div class="post-content e-content" itemprop="articleBody">
    <ul class="section-nav">
<li class="toc-entry toc-h1"><a href="#bmc-bioinformatics">BMC Bioinformatics</a>
<ul>
<li class="toc-entry toc-h2"><a href="#2020">2020</a></li>
<li class="toc-entry toc-h2"><a href="#2019">2019</a></li>
<li class="toc-entry toc-h2"><a href="#2018">2018</a></li>
<li class="toc-entry toc-h2"><a href="#2017">2017</a></li>
<li class="toc-entry toc-h2"><a href="#2016">2016</a></li>
<li class="toc-entry toc-h2"><a href="#2015">2015</a></li>
<li class="toc-entry toc-h2"><a href="#2014">2014</a></li>
<li class="toc-entry toc-h2"><a href="#2013">2013</a></li>
<li class="toc-entry toc-h2"><a href="#2012">2012</a></li>
<li class="toc-entry toc-h2"><a href="#2011">2011</a></li>
<li class="toc-entry toc-h2"><a href="#2010">2010</a></li>
<li class="toc-entry toc-h2"><a href="#2009">2009</a></li>
<li class="toc-entry toc-h2"><a href="#2008">2008</a></li>
<li class="toc-entry toc-h2"><a href="#2007">2007</a></li>
<li class="toc-entry toc-h2"><a href="#2006">2006</a></li>
<li class="toc-entry toc-h2"><a href="#2005">2005</a></li>
<li class="toc-entry toc-h2"><a href="#2004">2004</a></li>
<li class="toc-entry toc-h2"><a href="#2003">2003</a></li>
<li class="toc-entry toc-h2"><a href="#2002">2002</a></li>
</ul>
</li>
</ul><h1 id="bmc-bioinformatics">
<a class="anchor" href="#bmc-bioinformatics" aria-hidden="true"><span class="octicon octicon-link"></span></a>BMC Bioinformatics</h1>

<p>关键字：optimization for biological networks</p>

<h2 id="2020">
<a class="anchor" href="#2020" aria-hidden="true"><span class="octicon octicon-link"></span></a>2020</h2>
<ul>
  <li>–2020–Jane Hung–BMC–<strong>Keras R-CNN: library for cell detection in biological images using deep neural networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03635-x">PDF</a>
</li>
  <li>–2020–Yuanke Zhong–BMC–<strong>Twadn: an efficient alignment algorithm based on time warping for pairwise dynamic networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03672-6">PDF</a>
</li>
  <li>–2020–Ricardo Andrade–BMC–<strong>MOMO - multi-objective metabolic mixed integer optimization: application to yeast strain engineering</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3377-1">PDF</a>
</li>
  <li>–2020–Xihui Lin–BMC–<strong>Optimization and expansion of non-negative matrix factorization</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3312-5">PDF</a>
</li>
  <li>–2020–Yan Li–BMC–<strong>Bayesian differential analysis of gene regulatory networks exploiting genetic perturbations</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3314-3">PDF</a>
</li>
  <li>–2020–Bruna Piereck–BMC–<strong>LAITOR4HPC: A text mining pipeline based on HPC for building interaction networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03620-4">PDF</a>
</li>
  <li>–2020–Michael Sekula–BMC–<strong>A sparse Bayesian factor model for the construction of gene co-expression networks from single-cell RNA sequencing count data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03707-y">PDF</a>
</li>
  <li>–2020–Tania Timmermann–BMC–<strong>Reconstruction of a gene regulatory network of the induced systemic resistance defense response in Arabidopsis using boolean networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3472-3">PDF</a>
</li>
  <li>–2020–Luisa Cutillo–BMC–<strong>OscoNet: inferring oscillatory gene networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03561-y">PDF</a>
</li>
  <li>–2020–Sudipta Acharya–BMC–<strong>A consensus multi-view multi-objective gene selection approach for improved sample classification</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03681-5">PDF</a>
</li>
  <li>–2020–Bihai Zhao–BMC–<strong>NPF:network propagation for protein function prediction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03663-7">PDF</a>
</li>
  <li>–2020–Chunyan Tang–BMC–<strong>Drug-target interactions prediction using marginalized denoising model on heterogeneous networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03662-8">PDF</a>
</li>
  <li>–2020–Juncheng Guo–BMC–<strong>An embedded gene selection method using knockoffs optimizing neural network</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03717-w">PDF</a>
</li>
  <li>–2020–Ana Cernea–BMC–<strong>Robust pathway sampling in phenotype prediction. Application to triple negative breast cancer</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3356-6">PDF</a>
</li>
  <li>–2020–Mojtaba Tefagh–BMC–<strong>SWIFTCORE: a tool for the context-specific reconstruction of genome-scale metabolic networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3440-y">PDF</a>
</li>
  <li>–2020–Anna M. Nia–BMC–<strong>Efficient identification of multiple pathways: RNA-Seq analysis of livers from 56Fe ion irradiated mice</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3446-5">PDF</a>
</li>
  <li>–2020–Thevaa Ch–BMC–<strong>Lag penalized weighted correlation for time series clustering</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3324-1">PDF</a>
</li>
  <li>–2020–Robert A. Dromms–BMC–<strong>LK-DFBA: a linear programming-based modeling strategy for capturing dynamics and metabolite-dependent regulation in metabolism</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3422-0">PDF</a>
</li>
  <li>–2020–Yosef Masoudi-Sobhanzadeh–BMC–<strong>Synthetic repurposing of drugs against hypertension: a datamining method based on association rules and a novel discrete algorithm</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03644-w">PDF</a>
</li>
  <li>–2020–Lvxing Zhu–BMC–<strong>Biomedical event extraction with a novel combination strategy based on hybrid deep neural networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3376-2">PDF</a>
</li>
  <li>–2020–Wei Du–BMC–<strong>CapsNet-SSP: multilane capsule network for predicting human saliva-secretory proteins</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03579-2">PDF</a>
</li>
  <li>–2020–Cheng Ye–BMC–<strong>The corrected gene proximity map for analyzing the 3D genome organization using Hi-C data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03545-y">PDF</a>
</li>
  <li>–2020–Zhijing Li–BMC–<strong>Bio-semantic relation extraction with attention-based external knowledge reinforcement</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3540-8">PDF</a>
</li>
  <li>–2020–Hynek Bachratý–BMC–<strong>Applications of machine learning for simulations of red blood cells in microfluidic devices</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3357-5">PDF</a>
</li>
  <li>–2020–Isabella Mendolia–BMC–<strong>Convolutional architectures for virtual screening</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03645-9">PDF</a>
</li>
  <li>–2020–Jan Klosa–BMC–<strong>Seagull: lasso, group lasso and sparse-group lasso regularization for linear regression models via proximal gradient descent</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03725-w">PDF</a>
</li>
  <li>–2020–Jian Wang–BMC–<strong>Biomedical document triage using a hierarchical attention-based capsule network</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03673-5">PDF</a>
</li>
  <li>–2020–Hasindu Gamaarachchi–BMC–<strong>GPU accelerated adaptive banded event alignment for rapid comparative nanopore signal analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03697-x">PDF</a>
</li>
  <li>–2020–Melanie Fachet–BMC–<strong>Reconstruction and analysis of a carbon-core metabolic network for Dunaliella salina</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3325-0">PDF</a>
</li>
  <li>–2020–Yahya Bokhari–BMC–<strong>QuaDMutNetEx: a method for detecting cancer driver genes with low mutation frequency</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3449-2">PDF</a>
</li>
  <li>–2020–Fang Yang–BMC–<strong>Graph-based prediction of Protein-protein interactions with attributed signed graph embedding</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03646-8">PDF</a>
</li>
  <li>–2020–Paolo Castagno–BMC–<strong>A computational framework for modeling and studying pertussis epidemiology and vaccination</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03648-6">PDF</a>
</li>
  <li>–2020–Mikołaj Rybiński–BMC–<strong>TopoFilter: a MATLAB package for mechanistic model identification in systems biology</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3343-y">PDF</a>
</li>
  <li>–2020–Marta B. Lopes–BMC–<strong>Tracking intratumoral heterogeneity in glioblastoma via regularized classification of single-cell RNA-Seq data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3390-4">PDF</a>
</li>
  <li>–2020–Tiziana Castrignanò–BMC–<strong>ELIXIR-IT HPC@CINECA: high performance computing resources for the bioinformatics community</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03565-8">PDF</a>
</li>
  <li>–2020–Eun Jeong Min–BMC–<strong>Sparse multiple co-Inertia analysis with application to integrative analysis of multi -Omics data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3455-4">PDF</a>
</li>
  <li>–2020–Amina Lemsara–BMC–<strong>PathME: pathway based multi-modal sparse autoencoders for clustering of patient-level multi-omics data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3465-2">PDF</a>
</li>
  <li>–2020–Ali Foroughi pour–BMC–<strong>High dimensional model representation of log-likelihood ratio: binary classification with expression data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3486-x">PDF</a>
</li>
  <li>–2020–Pavlos Stephanos Bekiaris–BMC–<strong>Automatic construction of metabolic models with enzyme constraints</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3329-9">PDF</a>
</li>
  <li>–2020–Henri Riihimäki–BMC–<strong>A topological data analysis based classification method for multiple measurements</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03659-3">PDF</a>
</li>
  <li>–2020–Xin Guan–BMC–<strong>Dynamic incorporation of prior knowledge from multiple domains in biomarker discovery</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3344-x">PDF</a>
</li>
  <li>–2020–Hesham ElAbd–BMC–<strong>Amino acid encoding for deep learning applications</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03546-x">PDF</a>
</li>
  <li>–2020–Zhen Gao–BMC–<strong>Graph regularized L2,1-nonnegative matrix factorization for miRNA-disease association prediction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3409-x">PDF</a>
</li>
  <li>–2020–Alessio Mancini–BMC–<strong>Machine learning models predicting multidrug resistant urinary tract infections using “DsaaS”</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03566-7">PDF</a>
</li>
  <li>–2020–Aaron M. Smith–BMC–<strong>Standard machine learning approaches outperform deep representation learning on phenotype prediction from transcriptomics data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3427-8">PDF</a>
</li>
  <li>–2020–Joungmin Choi–BMC–<strong>methCancer-gen: a DNA methylome dataset generator for user-specified cancer type based on conditional variational autoencoder</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3516-8">PDF</a>
</li>
  <li>–2020–Joshua J. Levy–BMC–<strong>MethylNet: an automated and modular deep learning approach for DNA methylation analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3443-8">PDF</a>
</li>
  <li>–2020–Kitsuchart Pasupa–BMC–<strong>Discovery of significant porcine SNPs for swine breed identification by a hybrid of information gain, genetic algorithm, and frequency feature selection technique</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3471-4">PDF</a>
</li>
  <li>–2020–Yuliang Pan–BMC–<strong>Computationally identifying hot spots in protein-DNA binding interfaces using an ensemble approach</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03675-3">PDF</a>
</li>
  <li>–2020–Rita T. Sousa–BMC–<strong>Evolving knowledge graph similarity for supervised learning in complex biomedical domains</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3296-1">PDF</a>
</li>
  <li>–2020–Muaaz G. Awan–BMC–<strong>ADEPT: a domain independent sequence alignment strategy for gpu architectures</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03720-1">PDF</a>
</li>
  <li>–2020–Attila Csala–BMC–<strong>Multiset sparse partial least squares path modeling for high dimensional omics data analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3286-3">PDF</a>
</li>
  <li>–2020–Prasannavenkatesh Durai–BMC–<strong>Evolutionary chemical binding similarity approach integrated with 3D-QSAR method for effective virtual screening</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03643-x">PDF</a>
</li>
  <li>–2020–Xingheng Yu–BMC–<strong>Heterogeneous multiple kernel learning for breast cancer outcome evaluation</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3483-0">PDF</a>
</li>
  <li>–2020–Katja Ovchinnikova–BMC–<strong>OffsampleAI: artificial intelligence approach to recognize off-sample mass spectrometry images</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3425-x">PDF</a>
</li>
  <li>–2020–Niraj Thapa–BMC–<strong>DeepSuccinylSite: a deep learning based approach for protein succinylation site prediction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3342-z">PDF</a>
</li>
  <li>–2020–Lei Du–BMC–<strong>Subcellular location prediction of apoptosis proteins using two novel feature extraction methods based on evolutionary information and LDA</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3539-1">PDF</a>
</li>
  <li>–2020–Julie Hurel–BMC–<strong>DUGMO: tool for the detection of unknown genetically modified organisms with high-throughput sequencing data for pure bacterial samples</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03611-5">PDF</a>
</li>
  <li>–2020–Cheng Yan–BMC–<strong>PESM: predicting the essentiality of miRNAs based on gradient boosting machines and sequences</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3426-9">PDF</a>
</li>
  <li>–2020–Lei Li–BMC–<strong>Multi-task learning sparse group lasso: a method for quantifying antigenicity of influenza A(H1N1) virus using mutations and variations in glycosylation of Hemagglutinin</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3527-5">PDF</a>
</li>
  <li>–2020–Samuel E. Barnett–BMC–<strong>Simulating metagenomic stable isotope probing datasets with MetaSIPSim</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3372-6">PDF</a>
</li>
  <li>–2020–Matthew D. Koslovsky–BMC–<strong>MicroBVS: Dirichlet-tree multinomial regression models with Bayesian variable selection - an R package</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03640-0">PDF</a>
</li>
  <li>–2020–Mattia Chiesa–BMC–<strong>GARS: Genetic Algorithm for the identification of a Robust Subset of features in high-dimensional datasets</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3400-6">PDF</a>
</li>
  <li>–2020–Lu Cao–BMC–<strong>Automated image analysis system for studying cardiotoxicity in human pluripotent stem cell-Derived cardiomyocytes</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3466-1">PDF</a>
</li>
  <li>–2020–Behrend Heeren–BMC–<strong>Statistical shape analysis of tap roots: a methodological case study on laser scanned sugar beets</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03654-8">PDF</a>
</li>
  <li>–2020–Qiaoyue Liu–BMC–<strong>Deep learning improves the ability of sgRNA off-target propensity prediction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3395-z">PDF</a>
</li>
  <li>–2020–Matthew Merski–BMC–<strong>Self-analysis of repeat proteins reveals evolutionarily conserved patterns</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3493-y">PDF</a>
</li>
  <li>–2020–Feixiao Long–BMC–<strong>Microscopy cell nuclei segmentation with enhanced U-Net</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3332-1">PDF</a>
</li>
  <li>–2020–Audrey Hulot–BMC–<strong>Fast tree aggregation for consensus hierarchical clustering</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3453-6">PDF</a>
</li>
  <li>–2020–Sinead E Morris–BMC–<strong>ushr: Understanding suppression of HIV in R</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3389-x">PDF</a>
    <h2 id="2019">
<a class="anchor" href="#2019" aria-hidden="true"><span class="octicon octicon-link"></span></a>2019</h2>
  </li>
  <li>–2019–Wei Zheng–BMC–<strong>Control of multilayer biological networks and applied to target identification of complex diseases</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2841-2">PDF</a>
</li>
  <li>–2019–Sara Rahiminejad–BMC–<strong>Topological and functional comparison of community detection algorithms in biological networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2746-0">PDF</a>
</li>
  <li>–2019–Hsiang-Yun Wu–BMC–<strong>Metabopolis: scalable network layout for biological pathway diagrams in urban map style</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2779-4">PDF</a>
</li>
  <li>–2019–Chen Wang–BMC–<strong>Efficient proximal gradient algorithm for inference of differential gene networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2749-x">PDF</a>
</li>
  <li>–2019–Florian Martin–BMC–<strong>NPA: an R package for computing network perturbation amplitudes using gene expression data and two-layer networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3016-x">PDF</a>
</li>
  <li>–2019–Tom W. Hiscock–BMC–<strong>Adapting machine-learning algorithms to design gene circuits</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2788-3">PDF</a>
</li>
  <li>–2019–Kipoong Kim–BMC–<strong>Incorporating genetic networks into case-control association studies with high-dimensional DNA methylation data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3040-x">PDF</a>
</li>
  <li>–2019–Zolzaya Dashdorj–BMC–<strong>An application of convolutional neural networks with salient features for relation classification</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2808-3">PDF</a>
</li>
  <li>–2019–Savvas Kinalis–BMC–<strong>Deconvolution of autoencoders to learn biological regulatory modules from single cell mRNA sequencing data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2952-9">PDF</a>
</li>
  <li>–2019–Yifeng Cui–BMC–<strong>Predicting protein-ligand binding residues with deep convolutional neural networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2672-1">PDF</a>
</li>
  <li>–2019–Yongtian Wang–BMC–<strong>LncDisAP: a computation model for LncRNA-disease association prediction based on multiple biological datasets</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3081-1">PDF</a>
</li>
  <li>–2019–Tongle Ma–BMC–<strong>Periodicity and dosage optimization of an RNAi model in eukaryotes cells</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2925-z">PDF</a>
</li>
  <li>–2019–Eun-Youn Kim–BMC–<strong>Identification of critical connectors in the directed reaction-centric graphs of microbial metabolic networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2897-z">PDF</a>
</li>
  <li>–2019–Fei Han–BMC–<strong>A hybrid gene selection method based on gene scoring strategy and improved particle swarm optimization</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2773-x">PDF</a>
</li>
  <li>–2019–Han Yu–BMC–<strong>Integration of probabilistic regulatory networks into constraint-based models of metabolism with applications to Alzheimer’s disease</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2872-8">PDF</a>
</li>
  <li>–2019–Xiujuan Lei–BMC–<strong>Protein complex detection based on flower pollination mechanism in multi-relation reconstructed dynamic protein networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2649-0">PDF</a>
</li>
  <li>–2019–Denise Thiel–BMC–<strong>Identifying lncRNA-mediated regulatory modules via ChIA-PET network analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2900-8">PDF</a>
</li>
  <li>–2019–Min Luo–BMC–<strong>Screening drug target combinations in disease-related molecular networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2730-8">PDF</a>
</li>
  <li>–2019–Ryohei Eguchi–BMC–<strong>Classification of alkaloids according to the starting substances of their biosynthetic pathways using graph convolutional neural networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2963-6">PDF</a>
</li>
  <li>–2019–Mahroo Moridi–BMC–<strong>The assessment of efficient representation of drug features using deep learning for drug repositioning</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3165-y">PDF</a>
</li>
  <li>–2019–Joseph Luttrell IV–BMC–<strong>Predicting protein residue-residue contacts using random forests and deep networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2627-6">PDF</a>
</li>
  <li>–2019–Jake Alan Pitt–BMC–<strong>Parameter estimation in models of biological oscillators: an automated regularised estimation approach</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2630-y">PDF</a>
</li>
  <li>–2019–Byung-Jun Yoon–BMC–<strong>Selected research articles from the 2018 International Workshop on Computational Network Biology: Modeling, Analysis, and Control (CNB-MAC)</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2830-5">PDF</a>
</li>
  <li>–2019–Binh P. Nguyen–BMC–<strong>iProDNA-CapsNet: identifying protein-DNA binding residues using capsule neural networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3295-2">PDF</a>
</li>
  <li>–2019–Zheng Gao–BMC–<strong>edge2vec: Representation learning using edge semantics for biomedical knowledge discovery</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2914-2">PDF</a>
</li>
  <li>–2019–Bihai Zhao–BMC–<strong>An iteration method for identifying yeast essential proteins from heterogeneous network</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2930-2">PDF</a>
</li>
  <li>–2019–Miriam Bern–BMC–<strong>Network-based prediction of polygenic disease genes involved in cell motility</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2834-1">PDF</a>
</li>
  <li>–2019–Ali Navid–BMC–<strong>System-level analysis of metabolic trade-offs during anaerobic photoheterotrophic growth in Rhodopseudomonas palustris</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2844-z">PDF</a>
</li>
  <li>–2019–Weizhong Lu–BMC–<strong>Predicting RNA secondary structure via adaptive deep recurrent neural networks with energy-based filter</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3258-7">PDF</a>
</li>
  <li>–2019–Qiu Xiao–BMC–<strong>CeModule: an integrative framework for discovering regulatory patterns from genomic data in cancer</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2654-3">PDF</a>
</li>
  <li>–2019–Huan Jin–BMC–<strong>Moiety modeling framework for deriving moiety abundances from mass spectrometry measured isotopologues</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3096-7">PDF</a>
</li>
  <li>–2019–Da Zhang–BMC–<strong>Multimodal deep representation learning for protein interaction identification and protein family classification</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3084-y">PDF</a>
</li>
  <li>–2019–Yuanjie Liu–BMC–<strong>A general-purpose signal processing algorithm for biological profiles using only first-order derivative information</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3188-4">PDF</a>
</li>
  <li>–2019–Chieh Lo–BMC–<strong>MetaNN: accurate classification of host phenotypes from metagenomic data using neural networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2833-2">PDF</a>
</li>
  <li>–2019–Lifan Liang–BMC–<strong>Integrating data and knowledge to identify functional modules of genes: a multilayer approach</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2800-y">PDF</a>
</li>
  <li>–2019–Yahui Long–BMC–<strong>WMGHMDA: a novel weighted meta-graph-based model for predicting human microbe-disease association on heterogeneous information network</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3066-0">PDF</a>
</li>
  <li>–2019–Felipe Kenji Nakano–BMC–<strong>Machine learning for discovering missing or wrong protein function annotations</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3060-6">PDF</a>
</li>
  <li>–2019–Thomas A. Geddes–BMC–<strong>Autoencoder-based cluster ensembles for single-cell RNA-seq data analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3179-5">PDF</a>
</li>
  <li>–2019–Chun-Mei Feng–BMC–<strong>PCA via joint graph Laplacian and sparse constraint: Identification of differentially expressed genes and sample clustering on gene expression data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3229-z">PDF</a>
</li>
  <li>–2019–Jing Xu–BMC–<strong>A hierarchical integration deep flexible neural forest framework for cancer subtype classification by integrating multi-omics data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3116-7">PDF</a>
</li>
  <li>–2019–Trevor Cickovski–BMC–<strong>MATria: a unified centrality algorithm</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2820-7">PDF</a>
</li>
  <li>–2019–Hokuto Hirano–BMC–<strong>Difficulty in inferring microbial community structure based on co-occurrence network approaches</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2915-1">PDF</a>
</li>
  <li>–2019—-BMC–<strong>Third international conference “Bioinformatics: from Algorithms to Applications” (BiATA 2019)</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3122-9">PDF</a>
</li>
  <li>–2019–Jiesi Luo–BMC–<strong>Prediction of activity and specificity of CRISPR-Cpf1 using convolutional deep learning neural networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2939-6">PDF</a>
</li>
  <li>–2019–Hongda Bu–BMC–<strong>DEEPSEN: a convolutional neural network based method for super-enhancer prediction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3180-z">PDF</a>
</li>
  <li>–2019–Jol–BMC–<strong>A network-based approach for identifying suitable biomarkers for oral immunotherapy of food allergy</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2802-9">PDF</a>
</li>
  <li>–2019–Pascal Dolejsch–BMC–<strong>Extensions of ℓ1 regularization increase detection specificity for cell-type specific parameters in dynamic models</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2976-1">PDF</a>
</li>
  <li>–2019–Fortunato Bianconi–BMC–<strong>CRA toolbox: software package for conditional robustness analysis of cancer systems biology models in MATLAB</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2933-z">PDF</a>
</li>
  <li>–2019–Karsten Wüllems–BMC–<strong>Detection and visualization of communities in mass spectrometry imaging data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2890-6">PDF</a>
</li>
  <li>–2019–Tongxin Wang–BMC–<strong>Generalized gene co-expression analysis via subspace clustering using low-rank representation</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2733-5">PDF</a>
</li>
  <li>–2019–Ilaria Granata–BMC–<strong>Integration of transcriptomic data in a genome-scale metabolic model to investigate the link between obesity and breast cancer</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2685-9">PDF</a>
</li>
  <li>–2019–Jie Zhao–BMC–<strong>Detecting overlapping protein complexes in weighted PPI network based on overlay network chain in quotient space</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3256-9">PDF</a>
</li>
  <li>–2019–Vítor Vieira–BMC–<strong>Comparison of pathway analysis and constraint-based methods for cell factory design</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2934-y">PDF</a>
</li>
  <li>–2019–Nikolay Martyushenko–BMC–<strong>ModelExplorer - software for visual inspection and inconsistency correction of genome-scale metabolic reconstructions</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2615-x">PDF</a>
</li>
  <li>–2019–Leanne S. Whitmore–BMC–<strong>RetSynth: determining all optimal and sub-optimal synthetic pathways that facilitate synthesis of target compounds in chassis organisms</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3025-9">PDF</a>
</li>
  <li>–2019–Marta B. Lopes–BMC–<strong>Twiner: correlation-based regularization for identifying common cancer gene signatures</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2937-8">PDF</a>
</li>
  <li>–2019–Wei Li Wang–BMC–<strong>Robustness of signal detection in cryo-electron microscopy via a bi-objective-function approach</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2714-8">PDF</a>
</li>
  <li>–2019–Zhen Cui–BMC–<strong>The computational prediction of drug-disease interactions using the dual-network L2,1-CMF method</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2575-6">PDF</a>
</li>
  <li>–2019–Marie Postel–BMC–<strong>A multiscale mathematical model of cell dynamics during neurogenesis in the mouse cerebral cortex</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3018-8">PDF</a>
</li>
  <li>–2019–Yang Guo–BMC–<strong>Epi-GTBN: an approach of epistasis mining based on genetic Tabu algorithm and Bayesian network</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3022-z">PDF</a>
</li>
  <li>–2019–Xiaochuan Wang–BMC–<strong>SIMLIN: a bioinformatics tool for prediction of S-sulphenylation in the human proteome based on multi-stage ensemble-learning models</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3178-6">PDF</a>
</li>
  <li>–2019–Dan Liu–BMC–<strong>Predicting virus-host association by Kernelized logistic matrix factorization and similarity network fusion</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3082-0">PDF</a>
</li>
  <li>–2019–Martin Palazzo–BMC–<strong>A pan-cancer somatic mutation embedding using autoencoders</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3298-z">PDF</a>
</li>
  <li>–2019–Qingxin Xiao–BMC–<strong>Occurrence prediction of pests and diseases in cotton on the basis of weather factors by long short term memory network</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3262-y">PDF</a>
</li>
  <li>–2019–Roman Hornung–BMC–<strong>Block Forests: random forests for blocks of clinical and omics covariate data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2942-y">PDF</a>
</li>
  <li>–2019–Chih-Hao Fang–BMC–<strong>AIKYATAN: mapping distal regulatory elements using convolutional learning on GPU</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3049-1">PDF</a>
</li>
  <li>–2019–Tobias B. Alter–BMC–<strong>Determination of growth-coupling strategies and their underlying principles</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2946-7">PDF</a>
</li>
  <li>–2019–Stefan Hammer–BMC–<strong>Fixed-parameter tractable sampling for RNA design with multiple target structures</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2784-7">PDF</a>
</li>
  <li>–2019–Le–BMC–<strong>Parametric and non-parametric gradient matching for network inference: a comparison</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2590-7">PDF</a>
</li>
  <li>–2019–Ying-Lian Gao–BMC–<strong>NPCMF: Nearest Profile-based Collaborative Matrix Factorization method for predicting miRNA-disease associations</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2956-5">PDF</a>
</li>
  <li>–2019–Ruohan Wang–BMC–<strong>SpliceFinder: ab initio prediction of splice sites using convolutional neural network</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3306-3">PDF</a>
</li>
  <li>–2019–Yongtian Wang–BMC–<strong>Prioritizing candidate diseases-related metabolites based on literature and functional similarity</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3127-4">PDF</a>
</li>
  <li>–2019–Tianyi Zhao–BMC–<strong>Identifying Alzheimer’s disease-related proteins by LRRGD</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3124-7">PDF</a>
</li>
  <li>–2019–Michael Mayers–BMC–<strong>Time-resolved evaluation of compound repositioning predictions on a text-mined knowledge network</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3297-0">PDF</a>
</li>
  <li>–2019–Tomas Vicar–BMC–<strong>Cell segmentation methods for label-free contrast microscopy: review and comprehensive comparison</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2880-8">PDF</a>
</li>
  <li>–2019–Yong Wang–BMC–<strong>Systems biology intertwines with single cell and AI</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2731-7">PDF</a>
</li>
  <li>–2019–Stefano Teso–BMC–<strong>Combining learning and constraints for genome-wide protein annotation</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2875-5">PDF</a>
</li>
  <li>–2019–Najaf A. Shah–BMC–<strong>Variable cellular decision-making behavior in a constant synthetic network topology</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2866-6">PDF</a>
</li>
  <li>–2019–Kevin Michael Boehm–BMC–<strong>Predicting peptide presentation by major histocompatibility complex class I: an improved machine learning approach to the immunopeptidome</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2561-z">PDF</a>
</li>
  <li>–2019–Ya-Wei Niu–BMC–<strong>Integrating random walk and binary regression to identify novel miRNA-disease association</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2640-9">PDF</a>
</li>
  <li>–2019–Nguyen Quoc Khanh Le–BMC–<strong>ET-GRU: using multi-layer gated recurrent units to identify electron transport proteins</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2972-5">PDF</a>
</li>
  <li>–2019–Ying Xiong–BMC–<strong>An efficient gene selection method for microarray data based on LASSO and BPSO</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3228-0">PDF</a>
</li>
  <li>–2019–Chuang Li–BMC–<strong>MCtandem: an efficient tool for large-scale peptide identification on many integrated core (MIC) architecture</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2980-5">PDF</a>
</li>
  <li>–2019–Thomas Gatter–BMC–<strong>Ryūtō: network-flow based transcriptome reconstruction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2786-5">PDF</a>
</li>
  <li>–2019–Aditya Sai–BMC–<strong>Exploring the information transmission properties of noise-induced dynamics: application to glioma differentiation</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2970-7">PDF</a>
</li>
  <li>–2019–Amarin Jettakul–BMC–<strong>Relation extraction between bacteria and biotopes from biomedical texts with attention mechanisms and domain-specific contextual representations</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3217-3">PDF</a>
</li>
  <li>–2019–Rudi Alberts–BMC–<strong>ZDOG: zooming in on dominating genes with mutations in cancer pathways</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3326-z">PDF</a>
</li>
  <li>–2019–Luis Torada–BMC–<strong>ImaGene: a convolutional neural network to quantify natural selection from genomic data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2927-x">PDF</a>
</li>
  <li>–2019–Marina Esteban-Medina–BMC–<strong>Exploring the druggable space around the Fanconi anemia pathway using machine learning and mechanistic models</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2969-0">PDF</a>
</li>
  <li>–2019–Ye Wang–BMC–<strong>Semi-supervised prediction of protein interaction sites from unlabeled sample information</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3274-7">PDF</a>
</li>
  <li>–2019–Thorsten Will–BMC–<strong>Differential analysis of combinatorial protein complexes with CompleXChange</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2852-z">PDF</a>
</li>
  <li>–2019–Hyang-Mi Lee–BMC–<strong>Computational determination of hERG-related cardiotoxicity of drug candidates</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2814-5">PDF</a>
</li>
  <li>–2019–Rahul Shaw–BMC–<strong>A mass and charge balanced metabolic model of Setaria viridis revealed mechanisms of proton balancing in C4 plants</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2941-z">PDF</a>
</li>
  <li>–2019–Yosef Masoudi-Sobhanzadeh–BMC–<strong>FeatureSelect: a software for feature selection based on machine learning approaches</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2754-0">PDF</a>
</li>
  <li>–2019–Ine Melckenbeeck–BMC–<strong>Optimising orbit counting of arbitrary order by equation selection</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2483-9">PDF</a>
</li>
  <li>–2019–Yiran Zhou–BMC–<strong>NmSEER V2.0: a prediction tool for 2′-O-methylation sites based on random forest and multi-encoding combination</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3265-8">PDF</a>
</li>
  <li>–2019–Alex–BMC–<strong>Walking pathways with positive feedback loops reveal DNA methylation biomarkers of colorectal cancer</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2687-7">PDF</a>
</li>
  <li>–2019–Heinrich Roder–BMC–<strong>Robust identification of molecular phenotypes using semi-supervised learning</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2885-3">PDF</a>
</li>
  <li>–2019–Menghan Wang–BMC–<strong>A probabilistic method for leveraging functional annotations to enhance estimation of the temporal order of pathway mutations during carcinogenesis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3218-2">PDF</a>
</li>
  <li>–2019–Xinxing Yang–BMC–<strong>Additional Neural Matrix Factorization model for computational drug repositioning</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2983-2">PDF</a>
</li>
  <li>–2019–Jin Yao–BMC–<strong>Predicting clinically promising therapeutic hypotheses using tensor factorization</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2664-1">PDF</a>
</li>
  <li>–2019–Tianjun Li–BMC–<strong>Quality control of imbalanced mass spectra from isotopic labeling experiments</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3170-1">PDF</a>
</li>
  <li>–2019–Alex–BMC–<strong>Modelling TERT regulation across 19 different cancer types based on the MIPRIP 2.0 gene regulatory network approach</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3323-2">PDF</a>
</li>
  <li>–2019–Debangana Chakravorty–BMC–<strong>In silico modeling of phosphorylation dependent and independent c-Myc degradation</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2846-x">PDF</a>
</li>
  <li>–2019–Yunyun Dong–BMC–<strong>MLW-gcForest: a multi-weighted gcForest model towards the staging of lung adenocarcinoma based on multi-modal genetic data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3172-z">PDF</a>
</li>
  <li>–2019–Xusheng Li–BMC–<strong>A hybrid deep learning framework for bacterial named entity recognition with domain features</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3071-3">PDF</a>
</li>
  <li>–2019–Lei Cai–BMC–<strong>DeepSV: accurate calling of genomic deletions from high-throughput sequencing data using deep convolutional neural network</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3299-y">PDF</a>
</li>
  <li>–2019–Hiroki Konishi–BMC–<strong>Capturing the differences between humoral immunity in the normal and tumor environments from repertoire-seq of B-cell receptors using supervised machine learning</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2853-y">PDF</a>
</li>
  <li>–2019–Christian Carsten Sachs–BMC–<strong>mycelyso – high-throughput analysis of Streptomyces mycelium live cell imaging data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3004-1">PDF</a>
</li>
  <li>–2019–Yoann Dufresne–BMC–<strong>SLIM: a flexible web application for the reproducible processing of environmental DNA metabarcoding data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2663-2">PDF</a>
</li>
  <li>–2019–Olalekan Agbolade–BMC–<strong>3-Dimensional facial expression recognition in human using multi-points warping</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3153-2">PDF</a>
</li>
  <li>–2019–Marco Seel–BMC–<strong>Image-based classification of plant genus and family for trained and untrained plant species</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2474-x">PDF</a>
</li>
  <li>–2019–Gregory M. Parkes–BMC–<strong>Uncovering extensive post-translation regulation during human cell cycle progression by integrative multi-’omics analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3150-5">PDF</a>
</li>
  <li>–2019–Arezo Torang–BMC–<strong>An elastic-net logistic regression approach to generate classifiers and gene signatures for types of immune cells and T helper cell subsets</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2994-z">PDF</a>
</li>
  <li>–2019–Zhen Cui–BMC–<strong>L2,1-GRMF: an improved graph regularized matrix factorization method to predict drug-target interactions</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2768-7">PDF</a>
</li>
  <li>–2019–Francesco Pappalardo–BMC–<strong>Toward computational modelling on immune system function</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3239-x">PDF</a>
</li>
  <li>–2019–Minghan Chen–BMC–<strong>Analysis and remedy of negativity problem in hybrid stochastic simulation algorithm and its application</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2836-z">PDF</a>
</li>
  <li>–2019–Qiang Yan–BMC–<strong>Metabolic characterization of the chitinolytic bacterium Serratia marcescens using a genome-scale metabolic model</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2826-1">PDF</a>
</li>
  <li>–2019–Paul Müller–BMC–<strong>nanite: using machine learning to assess the quality of atomic force microscopy-enabled nano-indentation data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3010-3">PDF</a>
</li>
  <li>–2019–Haohan Wang–BMC–<strong>Deep mixed model for marginal epistasis detection and population stratification correction in genome-wide association studies</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3300-9">PDF</a>
</li>
  <li>–2019–N. Özlem ÖZCAN ŞİMŞEK–BMC–<strong>Statistical representation models for mutation information within genomic data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2868-4">PDF</a>
</li>
  <li>–2019–Damiano Verda–BMC–<strong>Analyzing gene expression data for pediatric and adult cancer diagnosis using logic learning machine and standard supervised methods</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2953-8">PDF</a>
</li>
  <li>–2019–Yi Zheng–BMC–<strong>Inverse similarity and reliable negative samples for drug side-effect prediction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2563-x">PDF</a>
</li>
  <li>–2019–Zhen Cui–BMC–<strong>RCMF: a robust collaborative matrix factorization method to predict miRNA-disease associations</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3260-0">PDF</a>
</li>
  <li>–2019–Licheng Wang–BMC–<strong>DDmap: a MATLAB package for the double digest problem using multiple genetic operators</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2862-x">PDF</a>
</li>
  <li>–2019–Beatriz Peñalver Bernabé–BMC–<strong>Dynamic genome-scale cell-specific metabolic models reveal novel inter-cellular and intra-cellular metabolic communications during ovarian follicle development</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2825-2">PDF</a>
</li>
  <li>–2019–Daphne Ezer–BMC–<strong>NITPicker: selecting time points for follow-up experiments</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2717-5">PDF</a>
</li>
  <li>–2019–Chuanyan Wu–BMC–<strong>PTPD: predicting therapeutic peptides by deep learning and word2vec</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3006-z">PDF</a>
</li>
  <li>–2019–Beiji Zou–BMC–<strong>A novel glaucomatous representation method based on Radon and wavelet transform</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3267-6">PDF</a>
</li>
  <li>–2019–Elies Ramon–BMC–<strong>HIV drug resistance prediction with weighted categorical kernel functions</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2991-2">PDF</a>
</li>
  <li>–2019–Zhongwang Yu–BMC–<strong>ANDIS: an atomic angle- and distance-dependent statistical potential for protein structure quality assessment</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2898-y">PDF</a>
</li>
  <li>–2019–Christine P’ng–BMC–<strong>BPG: Seamless, automated and interactive visualization of scientific data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2610-2">PDF</a>
</li>
  <li>–2019–Saugato Rahman Dhruba–BMC–<strong>Recursive model for dose-time responses in pharmacological studies</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2831-4">PDF</a>
</li>
  <li>–2019–Kohei Amano–BMC–<strong>Metabolic disassembler for understanding and predicting the biosynthetic units of natural products</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3183-9">PDF</a>
</li>
  <li>–2019–Sarah Machado–BMC–<strong>LimeSeg: a coarse-grained lipid membrane simulation for 3D image segmentation</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2471-0">PDF</a>
</li>
  <li>–2019–Grzegorz M. Boratyn–BMC–<strong>Magic-BLAST, an accurate RNA-seq aligner for long and short reads</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2996-x">PDF</a>
</li>
  <li>–2019–Yannick Martin–BMC–<strong>WellInverter: a web application for the analysis of fluorescent reporter gene data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2920-4">PDF</a>
</li>
  <li>–2019–Josefa Díaz Álvarez–BMC–<strong>An application of machine learning with feature selection to improve diagnosis and classification of neurodegenerative disorders</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3027-7">PDF</a>
    <h2 id="2018">
<a class="anchor" href="#2018" aria-hidden="true"><span class="octicon octicon-link"></span></a>2018</h2>
  </li>
  <li>–2018–Peizhuo Wang–BMC–<strong>Feature related multi-view nonnegative matrix factorization for identifying conserved functional modules in multiple biological networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2434-5">PDF</a>
</li>
  <li>–2018–Marek Ostaszewski–BMC–<strong>Clustering approaches for visual knowledge exploration in molecular interaction networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2314-z">PDF</a>
</li>
  <li>–2018–Yuanfang Ren–BMC–<strong>ProMotE: an efficient algorithm for counting independent motifs in uncertain network topologies</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2236-9">PDF</a>
</li>
  <li>–2018–Bo Xu–BMC–<strong>Protein complexes identification based on go attributed network embedding</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2555-x">PDF</a>
</li>
  <li>–2018–Tong Liu–BMC–<strong>Reconstructing high-resolution chromosome three-dimensional structures by Hi-C complex networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2464-z">PDF</a>
</li>
  <li>–2018–Yong He–BMC–<strong>A new insight into underlying disease mechanism through semi-parametric latent differential network model</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2461-2">PDF</a>
</li>
  <li>–2018–Victor A. O. Carmelo–BMC–<strong>WISH-R– a fast and efficient tool for construction of epistatic networks for complex traits and diseases</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2291-2">PDF</a>
</li>
  <li>–2018–Jie Hao–BMC–<strong>PASNet: pathway-associated sparse deep neural network for prognosis prediction from high-throughput data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2500-z">PDF</a>
</li>
  <li>–2018–Christopher I Cooper–BMC–<strong>Valection: design optimization for validation and verification studies</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2391-z">PDF</a>
</li>
  <li>–2018–Francesco Bardozzo–BMC–<strong>A study on multi-omic oscillations in Escherichia coli metabolic networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2175-5">PDF</a>
</li>
  <li>–2018–Marco Fornili–BMC–<strong>Modeling the covariates effects on the hazard function by piecewise exponential artificial neural networks: an application to a controlled clinical trial on renal carcinoma</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2179-1">PDF</a>
</li>
  <li>–2018–Helen N. Catanese–BMC–<strong>A nearest-neighbors network model for sequence data reveals new insight into genotype distribution of a pathogen</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2453-2">PDF</a>
</li>
  <li>–2018–Lei Huang–BMC–<strong>Completing sparse and disconnected protein-protein network by deep learning</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2112-7">PDF</a>
</li>
  <li>–2018–Justin M. Wozniak–BMC–<strong>CANDLE/Supervisor: a workflow framework for machine learning applied to cancer research</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2508-4">PDF</a>
</li>
  <li>–2018–Cong Pian–BMC–<strong>Discovering the ‘Dark matters’ in expression data of miRNA based on the miRNA-mRNA and miRNA-lncRNA networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2410-0">PDF</a>
</li>
  <li>–2018–Pau Farré–BMC–<strong>Dense neural networks for predicting chromatin conformation</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2286-z">PDF</a>
</li>
  <li>–2018–Mina Khoshdeli–BMC–<strong>Fusion of encoder-decoder deep networks improves delineation of multiple nuclear phenotypes</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2285-0">PDF</a>
</li>
  <li>–2018–Shuonan Chen–BMC–<strong>Evaluating methods of inferring gene regulatory networks highlights their lack of performance for single cell gene expression data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2217-z">PDF</a>
</li>
  <li>–2018–Kazunori D. Yamada–BMC–<strong>De novo profile generation based on sequence context specificity with the long short-term memory network</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2284-1">PDF</a>
</li>
  <li>–2018–Katarina Elez–BMC–<strong>Distinguishing crystallographic from biological interfaces in protein complexes: role of intermolecular contacts and energetics for classification</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2414-9">PDF</a>
</li>
  <li>–2018–Diego Fioravanti–BMC–<strong>Phylogenetic convolutional neural networks in metagenomics</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2033-5">PDF</a>
</li>
  <li>–2018–Gamal Crichton–BMC–<strong>Neural networks for link prediction in realistic biomedical graphs: a multi-dimensional evaluation of graph embedding-based approaches</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2163-9">PDF</a>
</li>
  <li>–2018–Yoichi Takenaka–BMC–<strong>Automated transition analysis of activated gene regulation during diauxic nutrient shift in Escherichia coli and adipocyte differentiation in mouse cells</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2072-y">PDF</a>
</li>
  <li>–2018–Faliu Yi–BMC–<strong>Microvessel prediction in H&amp;E Stained Pathology Images using fully convolutional neural networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2055-z">PDF</a>
</li>
  <li>–2018–Lokmane Chebouba–BMC–<strong>Discriminate the response of Acute Myeloid Leukemia patients to treatment by using proteomics data and Answer Set Programming</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2034-4">PDF</a>
</li>
  <li>–2018–Phan Nguyen–BMC–<strong>Time-lagged Ordered Lasso for network inference</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2558-7">PDF</a>
</li>
  <li>–2018–Christine Sinoque–BMC–<strong>A method combining a random forest-based technique with the modeling of linkage disequilibrium through latent variables, to run multilocus genome-wide association studies</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2054-0">PDF</a>
</li>
  <li>–2018–Yaogong Zhang–BMC–<strong>Prioritizing disease genes with an improved dual label propagation framework</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2040-6">PDF</a>
</li>
  <li>–2018–Régis Garnier–BMC–<strong>Comparison of metaheuristics to measure gene effects on phylogenetic supports and topologies</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2172-8">PDF</a>
</li>
  <li>–2018–Makbule Guclin Ozsoy–BMC–<strong>Realizing drug repositioning by adapting a recommendation system to handle the process</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2142-1">PDF</a>
</li>
  <li>–2018–Guido Zampieri–BMC–<strong>Scuba: scalable kernel-based gene prioritization</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2025-5">PDF</a>
</li>
  <li>–2018–Shunyun Yang–BMC–<strong>cmFSM: a scalable CPU-MIC coordinated drug-finding tool by frequent subgraph mining</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2071-z">PDF</a>
</li>
  <li>–2018–Yiqian Zhang–BMC–<strong>DeepM6ASeq: prediction and characterization of m6A-containing sequences using deep learning</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2516-4">PDF</a>
</li>
  <li>–2018–Jonathan Ozik–BMC–<strong>High-throughput cancer hypothesis testing with an integrated PhysiCell-EMEWS workflow</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2510-x">PDF</a>
</li>
  <li>–2018–Guifeng Tang–BMC–<strong>Sequence-based bacterial small RNAs prediction using ensemble learning strategies</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2535-1">PDF</a>
</li>
  <li>–2018–Jian-Yu Shi–BMC–<strong>BMCMDA: a novel model for predicting human microbe-disease associations via binary matrix completion</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2274-3">PDF</a>
</li>
  <li>–2018–Wen Zhang–BMC–<strong>Predicting drug-disease associations by using similarity constrained matrix factorization</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2220-4">PDF</a>
</li>
  <li>–2018–Reza Rezazadegan–BMC–<strong>Degeneracy and genetic assimilation in RNA evolution</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2497-3">PDF</a>
</li>
  <li>–2018–Abhik Seal–BMC–<strong>Netpredictor: R and Shiny package to perform drug-target network analysis and prediction of missing links</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2254-7">PDF</a>
</li>
  <li>–2018–Jianing Xi–BMC–<strong>Discovering mutated driver genes through a robust and sparse co-regularized matrix factorization framework with prior information from mRNA expression patterns and interaction network</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2218-y">PDF</a>
</li>
  <li>–2018–Chi Xiao–BMC–<strong>Effective automated pipeline for 3D reconstruction of synapses based on deep learning</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2232-0">PDF</a>
</li>
  <li>–2018–Nhat Tran–BMC–<strong>MicroRNA dysregulational synergistic network: discovering microRNA dysregulatory modules across subtypes in non-small cell lung cancers</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2536-0">PDF</a>
</li>
  <li>–2018–Debsindhu Bhowmik–BMC–<strong>Deep clustering of protein folding simulations</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2507-5">PDF</a>
</li>
  <li>–2018–Simone Dinarelli–BMC–<strong>FC_analysis: a tool for investigating atomic force microscopy maps of force curves</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2265-4">PDF</a>
</li>
  <li>–2018–Yözen Hernández–BMC–<strong>BpWrapper: BioPerl-based sequence and tree utilities for rapid prototyping of bioinformatics pipelines</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2074-9">PDF</a>
</li>
  <li>–2018–Maciej Antczak–BMC–<strong>RNAfitme: a webserver for modeling nucleobase and nucleoside residue conformation in fixed-backbone RNA structures</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2317-9">PDF</a>
</li>
  <li>–2018–Riccardo Colombo–BMC–<strong>Emerging ensembles of kinetic parameters to characterize observed metabolic phenotypes</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2181-7">PDF</a>
</li>
  <li>–2018–Peiying Ruan–BMC–<strong>Improving prediction of heterodimeric protein complexes using combination with pairwise kernel</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2017-5">PDF</a>
</li>
  <li>–2018–Senthil B. Girimurugan–BMC–<strong>iSeg: an efficient algorithm for segmentation of genomic and epigenomic data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2140-3">PDF</a>
</li>
  <li>–2018–Buzhong Zhang–BMC–<strong>Prediction of 8-state protein secondary structures by a novel deep learning architecture</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2280-5">PDF</a>
</li>
  <li>–2018–Alperen Dalkiran–BMC–<strong>ECPred: a tool for the prediction of the enzymatic functions of protein sequences based on the EC nomenclature</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2368-y">PDF</a>
</li>
  <li>–2018–Michail Tsagris–BMC–<strong>Feature selection for high-dimensional temporal data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2023-7">PDF</a>
</li>
  <li>–2018–Lacramioara Dranca–BMC–<strong>Using Kinect to classify Parkinson’s disease stages related to severity of gait impairment</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2488-4">PDF</a>
</li>
  <li>–2018–Yujuan Gao–BMC–<strong>RaptorX-Angle: real-value prediction of protein backbone dihedral angles through a hybrid method of clustering and deep learning</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2065-x">PDF</a>
</li>
  <li>–2018–Yifeng Li–BMC–<strong>Genome-wide prediction of cis-regulatory regions using supervised deep learning methods</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2187-1">PDF</a>
</li>
  <li>–2018–Yousef Al-Kofahi–BMC–<strong>A deep learning-based algorithm for 2-D cell segmentation in microscopy images</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2375-z">PDF</a>
</li>
  <li>–2018–Xiaoming Liu–BMC–<strong>The International Conference on Intelligent Biology and Medicine (ICIBM) 2018: bioinformatics towards translational applications</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2460-3">PDF</a>
</li>
  <li>–2018–Ali Foroughi pour–BMC–<strong>Heuristic algorithms for feature selection under Bayesian models with block-diagonal covariance structure</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2059-8">PDF</a>
</li>
  <li>–2018–Viachaslau Tsyvina–BMC–<strong>Fast estimation of genetic relatedness between members of heterogeneous populations of closely related genomic variants</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2333-9">PDF</a>
</li>
  <li>–2018–Sebastian Daberdaku–BMC–<strong>Exploring the potential of 3D Zernike descriptors and SVM for protein–protein interface prediction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2043-3">PDF</a>
</li>
  <li>–2018–Konstantina Dimitrakopoulou–BMC–<strong>Deblender: a semi−/unsupervised multi-operational computational method for complete deconvolution of expression data from heterogeneous samples</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2442-5">PDF</a>
</li>
  <li>–2018–Roman Prytuliak–BMC–<strong>SLALOM, a flexible method for the identification and statistical analysis of overlapping continuous sequence elements in sequence- and time-series data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2020-x">PDF</a>
</li>
  <li>–2018–Xianchao Zhu–BMC–<strong>Nonlinear expression and visualization of nonmetric relationships in genetic diseases and microbiome data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2537-z">PDF</a>
</li>
  <li>–2018–Xingang Jia–BMC–<strong>Analyzing the similarity of samples and genes by MG-PCC algorithm, t-SNE-SS and t-SNE-SG maps</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2495-5">PDF</a>
</li>
  <li>–2018–Raza-Ur Rahman–BMC–<strong>Oasis 2: improved online analysis of small RNA-seq data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2047-z">PDF</a>
</li>
  <li>–2018–Sinan Abo Alchamlat–BMC–<strong>Aggregation of experts: an application in the field of “interactomics” (detection of interactions on the basis of genomic data)</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2447-0">PDF</a>
</li>
  <li>–2018–Armen Abnousi–BMC–<strong>Alignment-free clustering of large data sets of unannotated protein conserved regions using minhashing</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2080-y">PDF</a>
</li>
  <li>–2018–Yanhua Qiao–BMC–<strong>Protein-protein interface hot spots prediction based on a hybrid feature selection strategy</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2009-5">PDF</a>
</li>
  <li>–2018–Skylar W. Marvel–BMC–<strong>ToxPi Graphical User Interface 2.0: Dynamic exploration, visualization, and sharing of integrated data models</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2089-2">PDF</a>
</li>
  <li>–2018–Muxin Gu–BMC–<strong>Semi-supervised machine learning for automated species identification by collagen peptide mass fingerprinting</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2221-3">PDF</a>
</li>
  <li>–2018–Florian Burger–BMC–<strong>PCRdrive: the largest qPCR assay archive to date and endless potential for lab workflow revitalization</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2452-3">PDF</a>
    <h2 id="2017">
<a class="anchor" href="#2017" aria-hidden="true"><span class="octicon octicon-link"></span></a>2017</h2>
  </li>
  <li>–2017–Jing Liu–BMC–<strong>A time series driven decomposed evolutionary optimization approach for reconstructing large-scale gene regulatory networks based on fuzzy cognitive maps</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1657-1">PDF</a>
</li>
  <li>–2017–Shahin Boluki–BMC–<strong>Incorporating biological prior knowledge for Bayesian learning via maximal knowledge-driven information priors</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1893-4">PDF</a>
</li>
  <li>–2017–Yiming Zuo–BMC–<strong>Incorporating prior biological knowledge for network-based differential gene expression analysis using differentially weighted graphical LASSO</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1515-1">PDF</a>
</li>
  <li>–2017–Yusuke Azuma–BMC–<strong>Biologically constrained optimization based cell membrane segmentation in C. elegans embryos</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1717-6">PDF</a>
</li>
  <li>–2017–Tianyu Kang–BMC–<strong>A biological network-based regularized artificial neural network model for robust phenotype prediction from gene expression data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1984-2">PDF</a>
</li>
  <li>–2017–Ziyi Li–BMC–<strong>Incorporating biological information in sparse principal component analysis with application to genomic data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1740-7">PDF</a>
</li>
  <li>–2017–Shirin Taghipour–BMC–<strong>Improving protein complex prediction by reconstructing a high-confidence protein-protein interaction network of Escherichia coli from different physical interaction data sources</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1422-x">PDF</a>
</li>
  <li>–2017–Chuanchao Zhang–BMC–<strong>Comparative network stratification analysis for identifying functional interpretable network biomarkers</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1462-x">PDF</a>
</li>
  <li>–2017–Le Ou-Yang–BMC–<strong>A multi-network clustering method for detecting protein complexes from multiple heterogeneous networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1877-4">PDF</a>
</li>
  <li>–2017–Hyundoo Jeong–BMC–<strong>CUFID-query: accurate network querying through random walk based network flow estimation</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1899-y">PDF</a>
</li>
  <li>–2017–Marianna Milano–BMC–<strong>An extensive assessment of network alignment algorithms for comparison of brain connectomes</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1635-7">PDF</a>
</li>
  <li>–2017–Wen Zhang–BMC–<strong>Predicting potential drug-drug interactions by integrating chemical, biological, phenotypic and network data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1415-9">PDF</a>
</li>
  <li>–2017–Shupeng Gui–BMC–<strong>A scalable algorithm for structure identification of complex gene regulatory network from temporal expression data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1489-z">PDF</a>
</li>
  <li>–2017–Xu Min–BMC–<strong>Predicting enhancers with deep convolutional neural networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1878-3">PDF</a>
</li>
  <li>–2017–Li Ma–BMC–<strong>CURE-SMOTE algorithm and hybrid algorithm for feature selection and parameter optimization based on random forests</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1578-z">PDF</a>
</li>
  <li>–2017–Kazuhiro Takemoto–BMC–<strong>Limitations of a metabolic network-based reverse ecology method for inferring host–pathogen interactions</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1696-7">PDF</a>
</li>
  <li>–2017–Hao He–BMC–<strong>Comparison of statistical methods for subnetwork detection in the integration of gene expression and protein interaction network</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1567-2">PDF</a>
</li>
  <li>–2017–Byung-Jun Yoon–BMC–<strong>Selected research articles from the 2016 International Workshop on Computational Network Biology: Modeling, Analysis, and Control (CNB-MAC)</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1521-3">PDF</a>
</li>
  <li>–2017–Kang K. Yan–BMC–<strong>A comparison of graph- and kernel-based –omics data integration algorithms for classifying complex traits</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1982-4">PDF</a>
</li>
  <li>–2017–Jie Tan–BMC–<strong>ADAGE signature analysis: differential expression analysis with data-defined gene sets</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1905-4">PDF</a>
</li>
  <li>–2017–Kevin Matlock–BMC–<strong>Combination therapy design for maximizing sensitivity and minimizing toxicity</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1523-1">PDF</a>
</li>
  <li>–2017–Xiaoyong Pan–BMC–<strong>RNA-protein binding motifs mining with a new hybrid deep learning based cross-domain knowledge integration approach</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1561-8">PDF</a>
</li>
  <li>–2017–Garrett W. Birkel–BMC–<strong>The JBEI quantitative metabolic modeling library (jQMM): a python library for modeling microbial metabolism</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1615-y">PDF</a>
</li>
  <li>–2017–Lisa Katharina Blaß–BMC–<strong>Network design and analysis for multi-enzyme biocatalysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1773-y">PDF</a>
</li>
  <li>–2017–David R. Penas–BMC–<strong>Parameter estimation in large-scale systems biology models: a parallel and self-adaptive cooperative strategy</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1452-4">PDF</a>
</li>
  <li>–2017–Lei Deng–BMC–<strong>A sparse autoencoder-based deep neural network for protein solvent accessibility and contact number prediction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1971-7">PDF</a>
</li>
  <li>–2017–Atif Khan–BMC–<strong>Meta-analysis of cell- specific transcriptomic data using fuzzy c-means clustering discovers versatile viral responsive genes</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1669-x">PDF</a>
</li>
  <li>–2017–James C. Wilgenbusch–BMC–<strong>Visualizing phylogenetic tree landscapes</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1479-1">PDF</a>
</li>
  <li>–2017–Alex M. Ascensión–BMC–<strong>NaviSE: superenhancer navigator integrating epigenomics signal algebra</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1698-5">PDF</a>
</li>
  <li>–2017–Jonathan D. Young–BMC–<strong>Unsupervised deep learning reveals prognostically relevant subtypes of glioblastoma</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1798-2">PDF</a>
</li>
  <li>–2017–Zhi-Ping Liu–BMC–<strong>Structure alignment-based classification of RNA-binding pockets reveals regional RNA recognition motifs on protein surfaces</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1410-1">PDF</a>
</li>
  <li>–2017–Nicola Lazzarini–BMC–<strong>RGIFE: a ranked guided iterative feature elimination heuristic for the identification of biomarkers</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1729-2">PDF</a>
</li>
  <li>–2017–Zhen Zhang–BMC–<strong>An integrated enhancement and reconstruction strategy for the quantitative extraction of actin stress fibers from fluorescence micrographs</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1684-y">PDF</a>
</li>
  <li>–2017–Sunny Raj–BMC–<strong>A theorem proving approach for automatically synthesizing visualizations of flow cytometry data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1662-4">PDF</a>
</li>
  <li>–2017–Clarence White–BMC–<strong>CNN-BLPred: a Convolutional neural network based predictor for β-Lactamases (BL) and their classes</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1972-6">PDF</a>
</li>
  <li>–2017–Yanan Zhu–BMC–<strong>A deep convolutional neural network approach to single-particle recognition in cryo-electron microscopy</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1757-y">PDF</a>
</li>
  <li>–2017–Michael I. Klein–BMC–<strong>GRAPE: a pathway template method to characterize tissue-specific functionality from gene expression profiles</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1711-z">PDF</a>
</li>
  <li>–2017–Wei Zheng–BMC–<strong>An attention-based effective neural model for drug-drug interactions extraction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1855-x">PDF</a>
</li>
  <li>–2017–Yahya Bokhari–BMC–<strong>QuaDMutEx: quadratic driver mutation explorer</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1869-4">PDF</a>
</li>
  <li>–2017–Biswanath Chowdhury–BMC–<strong>An optimized approach for annotation of large eukaryotic genomic sequences using genetic algorithm</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1874-7">PDF</a>
</li>
  <li>–2017–Bo Wen–BMC–<strong>metaX: a flexible and comprehensive software for processing metabolomics data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1579-y">PDF</a>
</li>
  <li>–2017–Herty Liany–BMC–<strong>MultiDCoX: Multi-factor analysis of differential co-expression</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1963-7">PDF</a>
</li>
  <li>–2017–Andrea Tangherloni–BMC–<strong>LASSIE: simulating large-scale models of biochemical systems on GPUs</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1666-0">PDF</a>
</li>
  <li>–2017–Sinan Abo Alchamlat–BMC–<strong>KNN-MDR: a learning approach for improving interactions mapping performances in genome wide association studies</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1599-7">PDF</a>
</li>
  <li>–2017–Pratyaydipta Rudra–BMC–<strong>Model based heritability scores for high-throughput sequencing data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1539-6">PDF</a>
</li>
  <li>–2017–Alex–BMC–<strong>Vermont: a multi-perspective visual interactive platform for mutational analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1789-3">PDF</a>
</li>
  <li>–2017–Tao Huang–BMC–<strong>MOST: most-similar ligand based approach to target prediction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1586-z">PDF</a>
</li>
  <li>–2017–Peter Bajcsy–BMC–<strong>Modeling, validation and verification of three-dimensional cell-scaffold contacts from terabyte-sized images</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1928-x">PDF</a>
</li>
  <li>–2017–Michael Kern–BMC–<strong>Interactive visual exploration and refinement of cluster assignments</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1813-7">PDF</a>
</li>
  <li>–2017–W. Duncan Wadsworth–BMC–<strong>An integrative Bayesian Dirichlet-multinomial regression model for the analysis of taxonomic abundances in microbiome data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1516-0">PDF</a>
</li>
  <li>–2017–José Cleydson F. Silva–BMC–<strong>Fangorn Forest (F2): a machine learning approach to classify genes and genera in the family Geminiviridae</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1839-x">PDF</a>
</li>
  <li>–2017–Sophie Engle–BMC–<strong>Unboxing cluster heatmaps</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1442-6">PDF</a>
</li>
  <li>–2017–Sugeerth Murugesan–BMC–<strong>Multi-scale visual analysis of time-varying electrocorticography data via clustering of brain regions</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1633-9">PDF</a>
</li>
  <li>–2017–Kyoungyeul Lee–BMC–<strong>Utilizing random Forest QSAR models with optimized parameters for target identification and its application to target-fishing server</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1960-x">PDF</a>
</li>
  <li>–2017–Juan C. Aledo–BMC–<strong>A machine learning approach for predicting methionine oxidation sites</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1848-9">PDF</a>
</li>
  <li>–2017–Eleni Ioanna Delatola–BMC–<strong>SegCorr a statistical procedure for the detection of genomic regions of correlated expression</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1742-5">PDF</a>
</li>
  <li>–2017–Santosh Philips–BMC–<strong>Using machine learning algorithms to identify genes essential for cell survival</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1799-1">PDF</a>
</li>
  <li>–2017–Richard J. Cotton–BMC–<strong>flippant–An R package for the automated analysis of fluorescence-based scramblase assays</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1542-y">PDF</a>
</li>
  <li>–2017–Onur Erbilgin–BMC–<strong>Dynamic substrate preferences predict metabolic properties of a simple microbial consortium</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1478-2">PDF</a>
</li>
  <li>–2017–Kuang-Lim Chan–BMC–<strong>Seqping: gene prediction pipeline for plant genomes using self-training gene models and transcriptomic data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1426-6">PDF</a>
</li>
  <li>–2017–Daniel Ian McSkimming–BMC–<strong>Classifying kinase conformations using a machine learning approach</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1506-2">PDF</a>
</li>
  <li>–2017–Alvin Farrel–BMC–<strong>An efficient algorithm for improving structure-based prediction of transcription factor binding sites</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1755-0">PDF</a>
</li>
  <li>–2017–Youyi Fong–BMC–<strong>chngpt: threshold regression model estimation and inference</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1863-x">PDF</a>
</li>
  <li>–2017–Xue Jiang–BMC–<strong>Identify Huntington’s disease associated genes based on restricted Boltzmann machine with RNA-seq data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1859-6">PDF</a>
</li>
  <li>–2017–Mengdi Zhao–BMC–<strong>Segmentation and classification of two-channel C. elegans nucleus-labeled fluorescence images</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1817-3">PDF</a>
</li>
  <li>–2017–Linlin Gao–BMC–<strong>Brain medical image diagnosis based on corners with importance-values</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1903-6">PDF</a>
</li>
  <li>–2017–Raquel Dias–BMC–<strong>Improving the accuracy of high-throughput protein-protein affinity prediction may require better training data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1533-z">PDF</a>
    <h2 id="2016">
<a class="anchor" href="#2016" aria-hidden="true"><span class="octicon octicon-link"></span></a>2016</h2>
  </li>
  <li>–2016–Julien Dorier–BMC–<strong>Boolean regulatory network reconstruction using literature based knowledge with a genetic algorithm optimization method</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1287-z">PDF</a>
</li>
  <li>–2016–Sapna Kumari–BMC–<strong>Bottom-up GGM algorithm for constructing multilayered hierarchical gene regulatory networks that govern biological pathways or processes</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0981-1">PDF</a>
</li>
  <li>–2016–Francesco Iorio–BMC–<strong>Efficient randomization of biological networks while preserving functional characterization of individual nodes</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1402-1">PDF</a>
</li>
  <li>–2016–Veronica Vinciotti–BMC–<strong>Consistency of biological networks inferred from microarray and sequencing data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1136-0">PDF</a>
</li>
  <li>–2016–Xiao-Fei Zhang–BMC–<strong>Comparative analysis of housekeeping and tissue-specific driver nodes in human protein interaction networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1233-0">PDF</a>
</li>
  <li>–2016–Joeri Ruyssinck–BMC–<strong>Netter: re-ranking gene network inference predictions using structural network properties</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0913-0">PDF</a>
</li>
  <li>–2016–Matteo Bersanelli–BMC–<strong>Methods for the integration of multi-omics data: mathematical aspects</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0857-9">PDF</a>
</li>
  <li>–2016–Ehsan Kazemi–BMC–<strong>PROPER: global protein interaction network alignment through percolation matching</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1395-9">PDF</a>
</li>
  <li>–2016–Daniele Muraro–BMC–<strong>An integrative analysis of gene expression and molecular interaction data to identify dys-regulated sub-networks in inflammatory bowel disease</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0886-z">PDF</a>
</li>
  <li>–2016–Meng-Yun Wu–BMC–<strong>Regularized logistic regression with network-based pairwise interaction for biomarker identification in breast cancer</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0951-7">PDF</a>
</li>
  <li>–2016–Andrew J. Sedgewick–BMC–<strong>Learning mixed graphical models with separate sparsity parameters and stability-based model selection</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1039-0">PDF</a>
</li>
  <li>–2016–André C. A. Nascimento–BMC–<strong>A multiple kernel learning algorithm for drug-target interaction prediction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0890-3">PDF</a>
</li>
  <li>–2016–Chee Yee Lim–BMC–<strong>BTR: training asynchronous Boolean models using single-cell expression data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1235-y">PDF</a>
</li>
  <li>–2016–Séverine Affeldt–BMC–<strong>3off2: A network reconstruction algorithm based on 2-point and 3-point information statistics</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0856-x">PDF</a>
</li>
  <li>–2016–Sefa Kılıç–BMC–<strong>Assessment of transfer methods for comparative genomics of regulatory networks in bacteria</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1113-7">PDF</a>
</li>
  <li>–2016–Vicente Acuña–BMC–<strong>Deciphering transcriptional regulations coordinating the response to environmental changes</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0885-0">PDF</a>
</li>
  <li>–2016–Patrick K. O’Neill–BMC–<strong>Parametric bootstrapping for biological sequence motifs</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1246-8">PDF</a>
</li>
  <li>–2016–Hussein A. Hejase–BMC–<strong>A scalability study of phylogenetic network inference methods using empirical datasets and simulations involving a single reticulation</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1277-1">PDF</a>
</li>
  <li>–2016–Sumanta Ray–BMC–<strong>A NMF based approach for integrating multiple data sources to predict HIV-1–human PPIs</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0952-6">PDF</a>
</li>
  <li>–2016–Hirotaka Matsumoto–BMC–<strong>SCOUP: a probabilistic model based on the Ornstein–Uhlenbeck process to analyze single-cell expression data during differentiation</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1109-3">PDF</a>
</li>
  <li>–2016–Lujia Chen–BMC–<strong>Learning a hierarchical representation of the yeast transcriptomic machinery using an autoencoder model</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0852-1">PDF</a>
</li>
  <li>–2016–André Veríssimo–BMC–<strong>DegreeCox – a network-based regularization method for survival analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1310-4">PDF</a>
</li>
  <li>–2016–Le Ou-Yang–BMC–<strong>Protein complex detection based on partially shared multi-view clustering</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1164-9">PDF</a>
</li>
  <li>–2016–Ricardo Cerri–BMC–<strong>Reduction strategies for hierarchical multi-label classification in protein function prediction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1232-1">PDF</a>
</li>
  <li>–2016–Shun Guo–BMC–<strong>Gene regulatory network inference using PLS-based methods</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1398-6">PDF</a>
</li>
  <li>–2016–Eric C. Rouchka–BMC–<strong>Proceedings of the 15th Annual UT-KBRIN Bioinformatics Summit 2016</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1154-y">PDF</a>
</li>
  <li>–2016–Arnav Kapur–BMC–<strong>Gene expression prediction using low-rank matrix completion</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1106-6">PDF</a>
</li>
  <li>–2016–Shibiao Wan–BMC–<strong>Sparse regressions for predicting and interpreting subcellular localization of multi-label proteins</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0940-x">PDF</a>
</li>
  <li>–2016–Pi-Jing Wei–BMC–<strong>LNDriver: identifying driver genes by integrating mutation and expression data based on gene-gene interaction network</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1332-y">PDF</a>
</li>
  <li>–2016–Laurence Yang–BMC–<strong>solveME: fast and reliable solution of nonlinear ME models</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1240-1">PDF</a>
</li>
  <li>–2016–Yijie Ding–BMC–<strong>Predicting protein-protein interactions via multivariate mutual information of protein sequences</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1253-9">PDF</a>
</li>
  <li>–2016–Neel Prabh–BMC–<strong>Are orphan genes protein-coding, prediction artifacts, or non-coding RNAs?</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1102-x">PDF</a>
</li>
  <li>–2016–Le Ou-Yang–BMC–<strong>A two-layer integration framework for protein complex detection</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0939-3">PDF</a>
</li>
  <li>–2016–Moritz von Stosch–BMC–<strong>A principal components method constrained by elementary flux modes: analysis of flux data sets</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1063-0">PDF</a>
</li>
  <li>–2016–Mirela Domijan–BMC–<strong>PeTTSy: a computational tool for perturbation analysis of complex systems biology models</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0972-2">PDF</a>
</li>
  <li>–2016–Ryan Eshleman–BMC–<strong>Leveraging graph topology and semantic context for pharmacovigilance through twitter-streams</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1220-5">PDF</a>
</li>
  <li>–2016–Yang Liu–BMC–<strong>PGen: large-scale genomic variations analysis workflow and browser in SoyKB</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1227-y">PDF</a>
</li>
  <li>–2016–Ariful Azad–BMC–<strong>flowVS: channel-specific variance stabilization in flow cytometry</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1083-9">PDF</a>
</li>
  <li>–2016–Wei Cheng–BMC–<strong>Sparse regression models for unraveling group and individual associations in eQTL mapping</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0986-9">PDF</a>
</li>
  <li>–2016–Yang Song–BMC–<strong>Bioimage classification with subcategory discriminant transform of high dimensional visual descriptors</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1318-9">PDF</a>
</li>
  <li>–2016–Youzhong Liu–BMC–<strong>MetICA: independent component analysis for high-resolution mass-spectrometry based non-targeted metabolomics</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0970-4">PDF</a>
</li>
  <li>–2016–Otte Heinävaara–BMC–<strong>On the inconsistency of ℓ
1-penalised sparse precision matrix estimation</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1309-x">PDF</a>
</li>
  <li>–2016–Said el Bouhaddani–BMC–<strong>Evaluation of O2PLS in Omics data integration</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0854-z">PDF</a>
</li>
  <li>–2016–Junliang Shang–BMC–<strong>CINOEDV: a co-information based method for detecting and visualizing n-order epistatic interactions</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1076-8">PDF</a>
</li>
  <li>–2016–David A. duVerle–BMC–<strong>CellTree: an R/bioconductor package to infer the hierarchical structure of cell populations from single-cell RNA-seq data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1175-6">PDF</a>
</li>
  <li>–2016–Javier Pérez-Rodríguez–BMC–<strong>Stepwise approach for combining many sources of evidence for site-recognition in genomic sequences</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0968-y">PDF</a>
</li>
  <li>–2016–Nicolas Rodriguez–BMC–<strong>The systems biology format converter</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1000-2">PDF</a>
</li>
  <li>–2016–Chao Fan–BMC–<strong>PredRSA: a gradient boosted regression trees approach for predicting protein solvent accessibility</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0851-2">PDF</a>
</li>
  <li>–2016–Yuan Chen–BMC–<strong>Informative gene selection and the direct classification of tumors based on relative simplicity</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0893-0">PDF</a>
</li>
  <li>–2016–David Gomez-Cabrero–BMC–<strong>From comorbidities of chronic obstructive pulmonary disease to identification of shared molecular mechanisms by data integration</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1291-3">PDF</a>
</li>
  <li>–2016–Yu-An Huang–BMC–<strong>Sequence-based prediction of protein-protein interactions using weighted sparse representation model combined with global encoding</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1035-4">PDF</a>
</li>
  <li>–2016–Chan-Seok Jeong–BMC–<strong>Structure-based Markov random field model for representing evolutionary constraints on functional sites</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0948-2">PDF</a>
</li>
  <li>–2016–Xi Hang Cao–BMC–<strong>A robust data scaling algorithm to improve classification accuracies in biomedical data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1236-x">PDF</a>
</li>
  <li>–2016–Fern–BMC–<strong>Efficient and automated large-scale detection of structural relationships in proteins with a flexible aligner</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0866-8">PDF</a>
</li>
  <li>–2016–Timothy DeFreitas–BMC–<strong>GEMINI: a computationally-efficient search engine for large gene expression datasets</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0934-8">PDF</a>
</li>
  <li>–2016–David M. Budden–BMC–<strong>Distributed gene expression modelling for exploring variability in epigenetic function</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1313-1">PDF</a>
</li>
  <li>–2016–Aaron Y. Lee–BMC–<strong>Scalable metagenomics alignment research tool (SMART): a scalable, rapid, and complete search heuristic for the classification of metagenomic sequences from complex sequence populations</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1159-6">PDF</a>
</li>
  <li>–2016–Chaitanya R. Acharya–BMC–<strong>Exploiting expression patterns across multiple tissues to map expression quantitative trait loci</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1123-5">PDF</a>
    <h2 id="2015">
<a class="anchor" href="#2015" aria-hidden="true"><span class="octicon octicon-link"></span></a>2015</h2>
  </li>
  <li>–2015–Ward C Wheele–BMC–<strong>Phylogenetic network analysis as a parsimony optimization problem</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0675-0">PDF</a>
</li>
  <li>–2015–Xiao-Fei Zhang–BMC–<strong>Determining minimum set of driver nodes in protein-protein interaction networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0591-3">PDF</a>
</li>
  <li>–2015–Faraz Hussain–BMC–<strong>Automated parameter estimation for biological models using Bayesian statistical model checking</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S17-S8">PDF</a>
</li>
  <li>–2015–Naresh Doni Jayavelu–BMC–<strong>Iterative sub-network component analysis enables reconstruction of large scale genetic networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0768-9">PDF</a>
</li>
  <li>–2015–Ernst C Wit–BMC–<strong>Inferring slowly-changing dynamic gene-regulatory networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S6-S5">PDF</a>
</li>
  <li>–2015–Shun Yao–BMC–<strong>Prior knowledge driven Granger causality analysis on gene regulatory network discovery</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0710-1">PDF</a>
</li>
  <li>–2015–Roozbeh Dehghannasiri–BMC–<strong>Efficient experimental design for uncertainty reduction in gene regulatory networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S13-S2">PDF</a>
</li>
  <li>–2015–Aurélie Pirayre–BMC–<strong>BRANE Cut: biologically-related a priori network enhancement with graph cuts for gene regulatory network inference</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0754-2">PDF</a>
</li>
  <li>–2015–Zina M Ibrahim–BMC–<strong>The relative vertex clustering value - a new criterion for the fast discovery of functional modules in protein interaction networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S4-S3">PDF</a>
</li>
  <li>–2015–Akito Taneda–BMC–<strong>Multi-objective optimization for RNA design with multiple target secondary structures</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0706-x">PDF</a>
</li>
  <li>–2015–Wuming Gong–BMC–<strong>Inferring dynamic gene regulatory networks in cardiac differentiation through the integration of multi-dimensional data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0460-0">PDF</a>
</li>
  <li>–2015–Ahmet Ay–BMC–<strong>Hierarchical decomposition of dynamically evolving regulatory networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0529-9">PDF</a>
</li>
  <li>–2015–Jakob Dohrmann–BMC–<strong>Global multiple protein-protein interaction network alignment by combining pairwise network alignments</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S13-S11">PDF</a>
</li>
  <li>–2015–Wenhui Wang–BMC–<strong>Network tuned multiple rank aggregation and applications to gene ranking</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S1-S6">PDF</a>
</li>
  <li>–2015–Lun Hu–BMC–<strong>A density-based clustering approach for identifying overlapping protein complexes with functional preferences</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0583-3">PDF</a>
</li>
  <li>–2015–Anirban Bhar–BMC–<strong>Multiobjective triclustering of time-series transcriptome data reveals key genes of biological processes</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0635-8">PDF</a>
</li>
  <li>–2015–Hossam M Ashtawy–BMC–<strong>BgN-Score and BsN-Score: Bagging and boosting based ensemble neural networks scoring functions for accurate binding affinity prediction of protein-ligand complexes</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S4-S8">PDF</a>
</li>
  <li>–2015–Syed Murtuza Baker–BMC–<strong>A unified framework for estimating parameters of kinetic biological models</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0500-9">PDF</a>
</li>
  <li>–2015–Peter Bajcsy–BMC–<strong>Survey statistics of automated segmentations applied to optical imaging of mammalian cells</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0762-2">PDF</a>
</li>
  <li>–2015–Sven Thiele–BMC–<strong>Extended notions of sign consistency to relate experimental data to signaling and regulatory network topologies</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0733-7">PDF</a>
</li>
  <li>–2015–Wei Cheng–BMC–<strong>Fast and robust group-wise eQTL mapping using sparse graphical models</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0421-z">PDF</a>
</li>
  <li>–2015–Suyu Mei–BMC–<strong>Multi-label multi-instance transfer learning for simultaneous reconstruction and cross-talk modeling of multiple human signaling pathways</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0841-4">PDF</a>
</li>
  <li>–2015–Vasily Sachnev–BMC–<strong>Multi-class BCGA-ELM based classifier that identifies biomarkers associated with hallmarks of cancer</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0565-5">PDF</a>
</li>
  <li>–2015–Mark-Anthony Bray–BMC–<strong>CellProfiler Tracer: exploring and validating high-throughput, time-lapse microscopy image data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0759-x">PDF</a>
</li>
  <li>–2015–Thomas Provoost–BMC–<strong>Semi-supervised Learning for the BioNLP Gene Regulation Network</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S10-S4">PDF</a>
</li>
  <li>–2015–Amirhossein Manzourolajdad–BMC–<strong>Secondary structural entropy in RNA switch (Riboswitch) identification</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0523-2">PDF</a>
</li>
  <li>–2015–Wen Zhang–BMC–<strong>Predicting drug side effects by multi-label learning and ensemble learning</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0774-y">PDF</a>
</li>
  <li>–2015–Pietro Pinoli–BMC–<strong>Computational algorithms to predict Gene Ontology annotations</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S6-S4">PDF</a>
</li>
  <li>–2015–Tianmin Wang–BMC–<strong>DomSign: a top-down annotation pipeline to enlarge enzyme space in the protein universe</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0499-y">PDF</a>
</li>
  <li>–2015–Kevin M Livingston–BMC–<strong>KaBOB: ontology-based semantic integration of biomedical databases</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0559-3">PDF</a>
</li>
  <li>–2015–Michaela Bayerlová–BMC–<strong>Comparative study on gene set and pathway topology-based enrichment methods</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0751-5">PDF</a>
</li>
  <li>–2015–Felix Schönenberger–BMC–<strong>Discrimination of cell cycle phases in PCNA-immunolabeled cells</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0618-9">PDF</a>
</li>
  <li>–2015–Stefano Parodi–BMC–<strong>Differential diagnosis of pleural mesothelioma using Logic Learning Machine</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S9-S3">PDF</a>
</li>
  <li>–2015–Elsa Bernard–BMC–<strong>A convex formulation for joint RNA isoform detection and quantification from multiple RNA-seq samples</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0695-9">PDF</a>
</li>
  <li>–2015–Watcharaporn Tanchotsrinon–BMC–<strong>A high performance prediction of HPV genotypes by Chaos game representation and singular value decomposition</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0493-4">PDF</a>
</li>
  <li>–2015–Maureen Stolzer–BMC–<strong>Event inference in multidomain families with phylogenetic reconciliation</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S14-S8">PDF</a>
</li>
  <li>–2015–George Tsatsaronis–BMC–<strong>An overview of the BIOASQ large-scale biomedical semantic indexing and question answering competition</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0564-6">PDF</a>
</li>
  <li>–2015–Abdulkadir Elmas–BMC–<strong>Reconstruction of novel transcription factor regulons through inference of their binding sites</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0685-y">PDF</a>
</li>
  <li>–2015–Giulio Caravagna–BMC–<strong>Automatising the analysis of stochastic biochemical time-series</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S9-S8">PDF</a>
</li>
  <li>–2015–Catalina V Anghel–BMC–<strong>ISOpureR: an R implementation of a computational purification algorithm of mixed tumour profiles</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0597-x">PDF</a>
</li>
  <li>–2015–Tamara Vasylenko–BMC–<strong>SCMPSP: Prediction and characterization of photosynthetic proteins based on a scoring card method</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S1-S8">PDF</a>
</li>
  <li>–2015–Joan Maynou–BMC–<strong>Sequence information gain based motif analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0811-x">PDF</a>
</li>
  <li>–2015–Itziar Frades–BMC–<strong>Comparison of phosphorylation patterns across eukaryotes by discriminative N-gram analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0657-2">PDF</a>
</li>
  <li>–2015–Bui Ngoc Thang–BMC–<strong>A semi–supervised tensor regression model for siRNA efficacy prediction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0495-2">PDF</a>
</li>
  <li>–2015–Josef Laimer–BMC–<strong>MAESTRO - multi agent stability prediction upon point mutations</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0548-6">PDF</a>
</li>
  <li>–2015–Florin Chelaru–BMC–<strong>Epiviz: a view inside the design of an integrated visual analysis software for genomics</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S11-S4">PDF</a>
</li>
  <li>–2015–Dániel Kozma–BMC–<strong>TMFoldRec: a statistical potential-based transmembrane protein fold recognition tool</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0638-5">PDF</a>
</li>
  <li>–2015–Talha Bin Masood–BMC–<strong>CHEXVIS: a tool for molecular channel extraction and visualization</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0545-9">PDF</a>
</li>
  <li>–2015–Robert J Flassig–BMC–<strong>Rational selection of experimental readout and intervention sites for reducing uncertainties in computational model predictions</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0436-5">PDF</a>
</li>
  <li>–2015–Federico Comoglio–BMC–<strong>Sensitive and highly resolved identification of RNA-protein interaction sites in PAR-CLIP data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0470-y">PDF</a>
</li>
  <li>–2015–Ralf Eggeling–BMC–<strong>Inferring intra-motif dependencies of DNA binding sites from ChIP-seq data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0797-4">PDF</a>
</li>
  <li>–2015–Xizeng Mao–BMC–<strong>Revisiting operons: an analysis of the landscape of transcriptional units in E. coli</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0805-8">PDF</a>
</li>
  <li>–2015–Michaël Vyverman–BMC–<strong>A Long Fragment Aligner called ALFALFA</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0533-0">PDF</a>
</li>
  <li>–2015–Angela Serra–BMC–<strong>MVDA: a multi-view genomic data integration methodology</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0680-3">PDF</a>
</li>
  <li>–2015–Mohammed AlQuraishi–BMC–<strong>An affinity-structure database of helix-turn-helix: DNA complexes with a universal coordinate system</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0819-2">PDF</a>
</li>
  <li>–2015–Agnieszka Rybarczyk–BMC–<strong>New in silico approach to assessing RNA secondary structures with non-canonical base pairs</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0718-6">PDF</a>
</li>
  <li>–2015–Caroline König–BMC–<strong>Label noise in subtype discrimination of class C G protein-coupled receptors: A systematic approach to the analysis of classification errors</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0731-9">PDF</a>
</li>
  <li>–2015–Gongning Luo–BMC–<strong>Neuron anatomy structure reconstruction based on a sliding filter</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0780-0">PDF</a>
    <h2 id="2014">
<a class="anchor" href="#2014" aria-hidden="true"><span class="octicon octicon-link"></span></a>2014</h2>
  </li>
  <li>–2014–Yu-Ting Hsiao–BMC–<strong>Reverse engineering gene regulatory networks: Coupling an optimization algorithm with a parameter identification technique</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S15-S8">PDF</a>
</li>
  <li>–2014–Zhimin Deng–BMC–<strong>A continuous optimization approach for inferring parameters in mathematical models of regulatory networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-256">PDF</a>
</li>
  <li>–2014–Enzo Acerbi–BMC–<strong>Gene network inference using continuous time Bayesian networks: a comparative study and application to Th17 cell differentiation</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0387-x">PDF</a>
</li>
  <li>–2014–Young Hwan Chang–BMC–<strong>Exact reconstruction of gene regulatory networks using compressive sensing</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0400-4">PDF</a>
</li>
  <li>–2014–Florian Martin–BMC–<strong>Quantification of biological network perturbations for mechanistic insight and diagnostics using two-layer causal models</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-238">PDF</a>
</li>
  <li>–2014–Zeeshan Gillani–BMC–<strong>CompareSVM: supervised, Support Vector Machine (SVM) inference of gene regularity networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0395-x">PDF</a>
</li>
  <li>–2014–Jose A Egea–BMC–<strong>MEIGO: an open-source software suite based on metaheuristics for global optimization in systems biology and bioinformatics</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-136">PDF</a>
</li>
  <li>–2014–Jun Zhong Ji–BMC–<strong>MAE-FMD: Multi-agent evolutionary method for functional module detection in protein-protein interaction networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-325">PDF</a>
</li>
  <li>–2014–Alberto Noronha–BMC–<strong>An integrated network visualization framework towards metabolic engineering applications</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0420-0">PDF</a>
</li>
  <li>–2014–William C Ray–BMC–<strong>Addressing the unmet need for visualizing conditional random fields in biological data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-202">PDF</a>
</li>
  <li>–2014–Gianvito Pio–BMC–<strong>Integrating microRNA target predictions for the discovery of gene regulatory networks: a semi-supervised ensemble learning approach</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S1-S4">PDF</a>
</li>
  <li>–2014–Jun Sun–BMC–<strong>Biochemical systems identification by a random drift particle swarm optimization approach</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S6-S1">PDF</a>
</li>
  <li>–2014–Le Ou-Yang–BMC–<strong>Detecting temporal protein complexes from dynamic protein-protein interaction networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-335">PDF</a>
</li>
  <li>–2014–Narsis A Kiani–BMC–<strong>Dynamic probabilistic threshold networks to infer signaling pathways from time-course perturbation data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-250">PDF</a>
</li>
  <li>–2014–Víctor Martínez–BMC–<strong>ProphNet: A generic prioritization method through propagation of information</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S1-S5">PDF</a>
</li>
  <li>–2014–Abiel Roche-Lima–BMC–<strong>Metabolic network prediction through pairwise rational kernels</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-318">PDF</a>
</li>
  <li>–2014–Kasper Dinkla–BMC–<strong>eXamine: Exploring annotated modules in networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-201">PDF</a>
</li>
  <li>–2014–Elisabetta Fersini–BMC–<strong>A p-Median approach for predicting drug response in tumour cells</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0353-7">PDF</a>
</li>
  <li>–2014–Marianna Budnikova–BMC–<strong>Design of a flexible component gathering algorithm for converting cell-based models to graph representations for use in evolutionary search</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-178">PDF</a>
</li>
  <li>–2014–Yun Zhang–BMC–<strong>On finding bicliques in bipartite graphs: a novel algorithm and its application to the integration of diverse biological data types</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-110">PDF</a>
</li>
  <li>–2014–Jose A Gomez–BMC–<strong>DFBAlab: a fast and reliable MATLAB code for dynamic flux balance analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0409-8">PDF</a>
</li>
  <li>–2014–Mohammad Ali Moni–BMC–<strong>Network-based analysis of comorbidities risk during an infection: SARS and HIV case studies</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-333">PDF</a>
</li>
  <li>–2014–Yiyi Liu–BMC–<strong>A network-assisted co-clustering algorithm to discover cancer subtypes based on gene expression</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-37">PDF</a>
</li>
  <li>–2014–Claudio Saccà–BMC–<strong>Improved multi-level protein–protein interaction prediction with semantic-based regularization</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-103">PDF</a>
</li>
  <li>–2014–Qianqian Wu–BMC–<strong>Approximate Bayesian computation schemes for parameter inference of discrete stochastic models using simulated likelihood density</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S12-S3">PDF</a>
</li>
  <li>–2014–Lingjian Yang–BMC–<strong>Pathway activity inference for multiclass disease classification through a mathematical programming optimisation framework</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0390-2">PDF</a>
</li>
  <li>–2014–Xiao-Fei Zhang–BMC–<strong>Detecting overlapping protein complexes based on a generative model with functional and topological properties</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-186">PDF</a>
</li>
  <li>–2014–Chun-Hou Zheng–BMC–<strong>Gene differential coexpression analysis based on biweight correlation and maximum clique</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S15-S3">PDF</a>
</li>
  <li>–2014–Junhua Zhang–BMC–<strong>Discovery of co-occurring driver pathways in cancer</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-271">PDF</a>
</li>
  <li>–2014–Evrim Acar–BMC–<strong>Structure-revealing data fusion</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-239">PDF</a>
</li>
  <li>–2014–Andrew Schoenrock–BMC–<strong>Efficient prediction of human protein-protein interactions at a global scale</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0383-1">PDF</a>
</li>
  <li>–2014–Zhi-Ping Liu–BMC–<strong>Systematic identification of transcriptional and post-transcriptional regulations in human respiratory epithelial cells during influenza A virus infection</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-336">PDF</a>
</li>
  <li>–2014–Anirban Mukhopadhyay–BMC–<strong>Incorporating the type and direction information in predicting novel regulatory interactions between HIV-1 and human proteins using a biclustering approach</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-26">PDF</a>
</li>
  <li>–2014–Hamid Mohamadlou–BMC–<strong>Maximizing Kolmogorov Complexity for accurate and robust bright field cell segmentation</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-32">PDF</a>
</li>
  <li>–2014–Minta Thomas–BMC–<strong>Predicting breast cancer using an expression values weighted clinical classifier</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0411-1">PDF</a>
</li>
  <li>–2014–Leo van Iersel–BMC–<strong>A practical approximation algorithm for solving massive instances of hybridization number for binary and nonbinary trees</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-127">PDF</a>
</li>
  <li>–2014–Chun-Pei Cheng–BMC–<strong>MiningABs: mining associated biomarkers across multi-connected gene expression datasets</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-173">PDF</a>
</li>
  <li>–2014–Calem J Bendell–BMC–<strong>Transient protein-protein interface prediction: datasets, features, algorithms, and the RAD-T predictor</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-82">PDF</a>
</li>
  <li>–2014–Anthony Santella–BMC–<strong>A semi-local neighborhood-based framework for probabilistic cell lineage tracing</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-217">PDF</a>
</li>
  <li>–2014–Andrei Kramer–BMC–<strong>Hamiltonian Monte Carlo methods for efficient parameter estimation in steady state dynamical systems</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-253">PDF</a>
</li>
  <li>–2014–Chen Meng–BMC–<strong>A multivariate approach to the integration of multi-omics datasets</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-162">PDF</a>
</li>
  <li>–2014–Frauke Kracke–BMC–<strong>Identifying target processes for microbial electrosynthesis by elementary mode analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0410-2">PDF</a>
</li>
  <li>–2014–Xiaoli Lin–BMC–<strong>Protein structure prediction with local adjust tabu search algorithm</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S15-S1">PDF</a>
</li>
  <li>–2014–Maad Shatnawi–BMC–<strong>Protein inter-domain linker prediction using Random Forest and amino acid physiochemical properties</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S16-S8">PDF</a>
</li>
  <li>–2014–Yi-Fan Liou–BMC–<strong>SCMHBP: prediction and analysis of heme binding proteins using propensity scores of dipeptides</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S16-S4">PDF</a>
</li>
  <li>–2014–Narjeskhatoon Habibi–BMC–<strong>A review of machine learning methods to predict the solubility of overexpressed recombinant proteins in Escherichia coli</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-134">PDF</a>
</li>
  <li>–2014–Edward WJ Curry–BMC–<strong>A framework for generalized subspace pattern mining in high-dimensional datasets</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0355-5">PDF</a>
</li>
  <li>–2014–Zhu-Hong You–BMC–<strong>Prediction of protein-protein interactions from amino acid sequences using a novel multi-scale continuous and discontinuous feature set</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S15-S9">PDF</a>
</li>
  <li>–2014–Rui Henriques–BMC–<strong>BicSPAM: flexible biclustering using sequential patterns</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-130">PDF</a>
</li>
  <li>–2014–Daniel Arend–BMC–<strong>e!DAL - a framework to store, share and publish research data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-214">PDF</a>
</li>
  <li>–2014–Syed Saiden Abbas–BMC–<strong>A comparative study of cell classifiers for image-based high-throughput screening</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-342">PDF</a>
</li>
  <li>–2014–Koen Van der Borght–BMC–<strong>Multi-model inference using mixed effects from a linear regression based genetic algorithm</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-88">PDF</a>
</li>
  <li>–2014–Ping Ge–BMC–<strong>ProbeAlign: incorporating high-throughput sequencing-based structure probing information into ncRNA homology search</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S9-S15">PDF</a>
</li>
  <li>–2014–Jason M Knight–BMC–<strong>MCMC implementation of the optimal Bayesian classifier for non-Gaussian models: model-based RNA-Seq classification</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0401-3">PDF</a>
</li>
  <li>–2014–Marc Streit–BMC–<strong>Furby: fuzzy force-directed bicluster visualization</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S6-S4">PDF</a>
</li>
  <li>–2014–Prabina Kumar Meher–BMC–<strong>A statistical approach for 5′ splice site prediction using short sequence motifs and without encoding sequence data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0362-6">PDF</a>
</li>
  <li>–2014–Rittika Shamsuddin–BMC–<strong>Computational prediction of hinge axes in proteins</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S8-S2">PDF</a>
</li>
  <li>–2014–Benjamin Drinkwater–BMC–<strong>Introducing TreeCollapse: a novel greedy algorithm to solve the cophylogeny reconstruction problem</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S16-S14">PDF</a>
</li>
  <li>–2014–Bart KM Jacobs–BMC–<strong>Impact of variance components on reliability of absolute quantification using digital PCR</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-283">PDF</a>
</li>
  <li>–2014–Romeo Rizzi–BMC–<strong>On the complexity of Minimum Path Cover with Subpath Constraints for multi-assembly</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S9-S5">PDF</a>
</li>
  <li>–2014–Steffen Möller–BMC–<strong>Community-driven development for computational biology at Sprints, Hackathons and Codefests</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S14-S7">PDF</a>
</li>
  <li>–2014–Li Song–BMC–<strong>nDNA-prot: identification of DNA-binding proteins based on unbalanced classification</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-298">PDF</a>
</li>
  <li>–2014–Shawn M Waldon–BMC–<strong>SketchBio: a scientist’s 3D interface for molecular modeling and animation</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-334">PDF</a>
</li>
  <li>–2014–Zhijie Dong–BMC–<strong>CRF-based models of protein surfaces improve protein-protein interaction site predictions</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-277">PDF</a>
</li>
  <li>–2014–Solon P Pissi–BMC–<strong>MoTeX-II: structured MoTif eXtraction from large-scale datasets</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-235">PDF</a>
</li>
  <li>–2014–Prashant Shingate–BMC–<strong>ECMIS: computational approach for the identification of hotspots at protein-protein interfaces</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-303">PDF</a>
</li>
  <li>–2014–Arthur Gilly–BMC–<strong>TE-Tracker: systematic identification of transposition events through whole-genome resequencing</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0377-z">PDF</a>
</li>
  <li>–2014–Tyler Weirick–BMC–<strong>LacSubPred: predicting subtypes of Laccases, an important lignin metabolism-related enzyme class, using in silico approaches</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S11-S15">PDF</a>
</li>
  <li>–2014–Ruifeng Liu–BMC–<strong>Exploiting large-scale drug-protein interaction information for computational drug repurposing</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-210">PDF</a>
</li>
  <li>–2014–Maciej Rybinski–BMC–<strong>Calculating semantic relatedness for biomedical use in a knowledge-poor environment</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S14-S2">PDF</a>
</li>
  <li>–2014–Tuan Hue Thi–BMC–<strong>Recognizing flu-like symptoms from videos</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-300">PDF</a>
</li>
  <li>–2014–Jintao Meng–BMC–<strong>SWAP-Assembler: scalable and efficient genome assembly towards thousands of cores</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S9-S2">PDF</a>
    <h2 id="2013">
<a class="anchor" href="#2013" aria-hidden="true"><span class="octicon octicon-link"></span></a>2013</h2>
  </li>
  <li>–2013–Yijie Wang–BMC–<strong>A novel subgradient-based optimization algorithm for blockmodel functional module identification</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S2-S23">PDF</a>
</li>
  <li>–2013–Jialiang Yang–BMC–<strong>BinAligner: a heuristic method to align biological networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S14-S8">PDF</a>
</li>
  <li>–2013–Giuseppe Facchetti–BMC–<strong>Partial inhibition and bilevel optimization in flux balance analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-344">PDF</a>
</li>
  <li>–2013–Ahsan Raja Chowdhury–BMC–<strong>Incorporating time-delays in S-System model for reverse engineering genetic networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-196">PDF</a>
</li>
  <li>–2013–Giuseppe Agapito–BMC–<strong>Visualization of protein interaction networks: problems and solutions</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S1-S1">PDF</a>
</li>
  <li>–2013–Yuekai Sun–BMC–<strong>Robust flux balance analysis of multiscale biochemical reaction networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-240">PDF</a>
</li>
  <li>–2013–Yi Yang–BMC–<strong>Differential reconstructed gene interaction networks for deriving toxicity threshold in chemical risk assessment</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S14-S3">PDF</a>
</li>
  <li>–2013–Ilana Lichtenstein–BMC–<strong>Active subnetwork recovery with a mechanism-dependent scoring function; with application to angiogenesis and organogenesis studies</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-59">PDF</a>
</li>
  <li>–2013–Nikolaos Berntenis–BMC–<strong>Detection of attractors of large Boolean networks via exhaustive enumeration of appropriate subspaces of the state space</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-361">PDF</a>
</li>
  <li>–2013–Daniela Stojanova–BMC–<strong>Using PPI network autocorrelation in hierarchical multi-label classification trees for gene function prediction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-285">PDF</a>
</li>
  <li>–2013–Afshin Sadeghi–BMC–<strong>Steiner tree methods for optimal sub-network identification: an empirical study</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-144">PDF</a>
</li>
  <li>–2013–Fabrizio F Borelli–BMC–<strong>Gene regulatory networks inference using a multi-GPU exhaustive search algorithm</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S18-S5">PDF</a>
</li>
  <li>–2013–Shaogang Ren–BMC–<strong>Adaptive bi-level programming for optimal gene knockouts for targeted overproduction under phenotypic constraints</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S2-S17">PDF</a>
</li>
  <li>–2013–Steven Eker–BMC–<strong>Computing minimal nutrient sets from metabolic networks via linear constraint solving</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-114">PDF</a>
</li>
  <li>–2013–Ivan Merelli–BMC–<strong>SNPranker 2.0: a gene-centric data mining tool for diseases associated SNP prioritization in GWAS</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S1-S9">PDF</a>
</li>
  <li>–2013–Xian Zhang–BMC–<strong>A novel phenotypic dissimilarity method for image-based high-throughput screens</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-336">PDF</a>
</li>
  <li>–2013–Mohammed Alshalalfa–BMC–<strong>Using context-specific effect of miRNAs to identify functional associations between miRNAs and gene signatures</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S12-S1">PDF</a>
</li>
  <li>–2013–Piyush B Madhamshettiwar–BMC–<strong>RMaNI: Regulatory Module Network Inference framework</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S16-S14">PDF</a>
</li>
  <li>–2013–Ben Niu–BMC–<strong>Biomimicry of quorum sensing using bacterial lifecycle model</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S8-S8">PDF</a>
</li>
  <li>–2013–Artem Sokolov–BMC–<strong>Combining heterogeneous data sources for accurate functional annotation of proteins</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S3-S10">PDF</a>
</li>
  <li>–2013–Dongdong Lin–BMC–<strong>Group sparse canonical correlation analysis for genomic data integration</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-245">PDF</a>
</li>
  <li>–2013–Vladimir Sarpe–BMC–<strong>Simulating the decentralized processes of the human immune system in a virtual anatomy model</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S6-S2">PDF</a>
</li>
  <li>–2013–Karan Uppal–BMC–<strong>xMSanalyzer: automated pipeline for improved feature detection and downstream analysis of large-scale, non-targeted metabolomics data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-15">PDF</a>
</li>
  <li>–2013–Noah Berlow–BMC–<strong>A new approach for prediction of tumor sensitivity to targeted drugs based on functional data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-239">PDF</a>
</li>
  <li>–2013–Liang Lan–BMC–<strong>MS-k NN: protein function prediction by integrating multiple data sources</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S3-S8">PDF</a>
</li>
  <li>–2013–Jin-Xing Liu–BMC–<strong>Robust PCA based method for discovering differentially expressed genes</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S8-S3">PDF</a>
</li>
  <li>–2013–Jorge Alberto Jaramillo-Garzón–BMC–<strong>Predictability of gene ontology slim-terms from primary structure information in Embryophyta plant proteins</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-68">PDF</a>
</li>
  <li>–2013–John C Earls–BMC–<strong>AUREA: an open-source software system for accurate and user-friendly identification of relative expression molecular signatures</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-78">PDF</a>
</li>
  <li>–2013–John O Woods–BMC–<strong>Prediction of gene-phenotype associations in humans, mice, and plants using phenologs</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-203">PDF</a>
</li>
  <li>–2013–Tobias Hamp–BMC–<strong>Homology-based inference sets the bar high for protein function prediction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S3-S7">PDF</a>
</li>
  <li>–2013–Takanori Ueda–BMC–<strong>Estimate hidden dynamic profiles of siRNA effect on apoptosis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-97">PDF</a>
</li>
  <li>–2013–Sepideh Babaei–BMC–<strong>Detecting recurrent gene mutation in interaction network context using multi-scale graph diffusion</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-29">PDF</a>
</li>
  <li>–2013–Brian J Schmidt–BMC–<strong>Alternate virtual populations elucidate the type I interferon signature predictive of the response to rituximab in rheumatoid arthritis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-221">PDF</a>
</li>
  <li>–2013–Alex–BMC–<strong>A novel min-cost flow method for estimating transcript expression with RNA-Seq</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S5-S15">PDF</a>
</li>
  <li>–2013–Nima Aghaeepour–BMC–<strong>Ensemble-based prediction of RNA secondary structures</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-139">PDF</a>
</li>
  <li>–2013–Salma Jamal–BMC–<strong>Cheminformatic models based on machine learning for pyruvate kinase inhibitors of Leishmania mexicana</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-329">PDF</a>
</li>
  <li>–2013–Bing Wang–BMC–<strong>Prediction of peptide drift time in ion mobility mass spectrometry from sequence-based features</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S8-S9">PDF</a>
</li>
  <li>–2013–Heinz Koeppl–BMC–<strong>Mapping behavioral specifications to model parameters in synthetic biology</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S10-S9">PDF</a>
</li>
  <li>–2013–Bhanu Rekapalli–BMC–<strong>PoPLAR: Portal for Petascale Lifescience Applications and Research</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S9-S3">PDF</a>
</li>
  <li>–2013–Qian Sun–BMC–<strong>Image-level and group-level models for Drosophilagene expression pattern annotation</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-350">PDF</a>
</li>
  <li>–2013–Jacob Biesinger–BMC–<strong>Discovering and mapping chromatin states using a tree hidden Markov model</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S5-S4">PDF</a>
</li>
  <li>–2013–Florent Baty–BMC–<strong>Exploring the transcription factor activity in high-throughput gene expression data using RLQ analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-178">PDF</a>
</li>
  <li>–2013–Yaping Fang–BMC–<strong>Identification of properties important to protein aggregation using feature selection</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-314">PDF</a>
</li>
  <li>–2013–Cameron Cotten–BMC–<strong>Mechanistic analysis of multi-omics datasets to generate kinetic parameters for constraint-based metabolic models</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-32">PDF</a>
</li>
  <li>–2013–Iain H Moal–BMC–<strong>The scoring of poses in protein-protein docking: current capabilities and future directions</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-286">PDF</a>
</li>
  <li>–2013–Ying-Tsang Lo–BMC–<strong>Prediction of conformational epitopes with the use of a knowledge-based energy function and geometrically related neighboring residue characteristics</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S4-S3">PDF</a>
</li>
  <li>–2013–Sushmita Paul–BMC–<strong>μHEM for identification of differentially expressed miRNAs using hypercuboid equivalence partition matrix</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-266">PDF</a>
</li>
  <li>–2013–François Van Lishout–BMC–<strong>An efficient algorithm to perform multiple testing in epistasis screening</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-138">PDF</a>
</li>
  <li>–2013–James WJ Anderson–BMC–<strong>Quantifying variances in comparative RNA secondary structure prediction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-149">PDF</a>
</li>
  <li>–2013–Wen Zou–BMC–<strong>Data mining tools for Salmonella characterization: application to gel-based fingerprinting analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S14-S15">PDF</a>
</li>
  <li>–2013–Malgorzata Kotulska–BMC–<strong>On the amyloid datasets used for training PAFIG ­ how (not) to extend the experimental dataset of hexapeptides</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-351">PDF</a>
    <h2 id="2012">
<a class="anchor" href="#2012" aria-hidden="true"><span class="octicon octicon-link"></span></a>2012</h2>
  </li>
  <li>–2012–Nguyen Xuan Vinh–BMC–<strong>Gene regulatory network modeling via global optimization of high-order dynamic Bayesian network</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-131">PDF</a>
</li>
  <li>–2012–Anton Miró–BMC–<strong>Deterministic global optimization algorithm based on outer approximation for the parameter estimation of nonlinear dynamic biological systems</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-90">PDF</a>
</li>
  <li>–2012–Boon-Siew Seah–BMC–<strong>FUSE: a profit maximization approach for functional summarization of biological networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S3-S10">PDF</a>
</li>
  <li>–2012–Yu-Keng Shih–BMC–<strong>Scalable global alignment for multiple biological networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S3-S11">PDF</a>
</li>
  <li>–2012–Wen-Chieh Chang–BMC–<strong>Exploring biological interaction networks with tailored weighted quasi-bicliques</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S10-S16">PDF</a>
</li>
  <li>–2012–Lauri Eronen–BMC–<strong>Biomine: predicting links between biological entities using network models of heterogeneous databases</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-119">PDF</a>
</li>
  <li>–2012–Yu-Ting Hsiao–BMC–<strong>Inferring robust gene networks from expression data by a sensitivity-based incremental evolution method</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S7-S8">PDF</a>
</li>
  <li>–2012–Ye Deng–BMC–<strong>Molecular ecological network analyses</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-113">PDF</a>
</li>
  <li>–2012–Bradley M Broom–BMC–<strong>Model averaging strategies for structure learning in Bayesian networks with limited data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S13-S10">PDF</a>
</li>
  <li>–2012–Günhan Gülsoy–BMC–<strong>HIDEN: Hierarchical decomposition of regulatory networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-250">PDF</a>
</li>
  <li>–2012–Mark K Transtrum–BMC–<strong>Optimal experiment selection for parameter estimation in biological differential equation models</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-181">PDF</a>
</li>
  <li>–2012–Steven M Hill–BMC–<strong>Integrating biological knowledge into variable selection: an empirical Bayes approach with an application in cancer biology</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-94">PDF</a>
</li>
  <li>–2012–Abdelhalim Larhlimi–BMC–<strong>F2C2: a fast tool for the computation of flux coupling in genome-scale metabolic networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-57">PDF</a>
</li>
  <li>–2012–Ying-Ke Lei–BMC–<strong>Assessing and predicting protein interactions by combining manifold embedding with multiple information integration</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S7-S3">PDF</a>
</li>
  <li>–2012–Atanas Kamburov–BMC–<strong>Cluster-based assessment of protein-protein interaction confidence</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-262">PDF</a>
</li>
  <li>–2012–Benjamin A Logsdon–BMC–<strong>Mouse obesity network reconstruction with a variational Bayes algorithm to employ aggressive false positive control</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-53">PDF</a>
</li>
  <li>–2012–Yong Fuga Li–BMC–<strong>Computational approaches to protein inference in shotgun proteomics</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S16-S4">PDF</a>
</li>
  <li>–2012–Xi-Jun Liang–BMC–<strong>Inference of gene regulatory subnetworks from time course gene expression data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S9-S3">PDF</a>
</li>
  <li>–2012–Jeremy J Jay–BMC–<strong>A systematic comparison of genome-scale clustering algorithms</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S10-S7">PDF</a>
</li>
  <li>–2012–Tianlei Xu–BMC–<strong>Quantitatively integrating molecular structure and bioactivity profile evidence into drug-target relationship analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-75">PDF</a>
</li>
  <li>–2012–Ruoting Yang–BMC–<strong>Core module biomarker identification with network exploration for breast cancer metastasis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-12">PDF</a>
</li>
  <li>–2012–Duojiao Wu–BMC–<strong>Cancer bioinformatics: A new approach to systems clinical medicine</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-71">PDF</a>
</li>
  <li>–2012–H Robert Frost–BMC–<strong>Markov Chain Ontology Analysis (MCOA)</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-23">PDF</a>
</li>
  <li>–2012–Jonathan Q Jiang–BMC–<strong>Predicting multiplex subcellular localization of proteins using protein-protein interaction network: a comparative study</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S10-S20">PDF</a>
</li>
  <li>–2012–Jun Gao–BMC–<strong>Integrated QSAR study for inhibitors of hedgehog signal pathway against multiple cell lines:a collaborative filtering method</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-186">PDF</a>
</li>
  <li>–2012–Abdelaali Briache–BMC–<strong>Transparent mediation-based access to multiple yeast data sources using an ontology driven interface</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S1-S7">PDF</a>
</li>
  <li>–2012–Muskan Kukreja–BMC–<strong>Comparative study of classification algorithms for immunosignaturing data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-139">PDF</a>
</li>
  <li>–2012–Andrew T Magis–BMC–<strong>The top-scoring ‘N’ algorithm: a generalized relative expression classification method from small numbers of biomolecules</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-227">PDF</a>
</li>
  <li>–2012–John D Eblen–BMC–<strong>The maximum clique enumeration problem: algorithms, applications, and implementations</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S10-S5">PDF</a>
</li>
  <li>–2012–Je-Gun Joung–BMC–<strong>A probabilistic coevolutionary biclustering algorithm for discovering coherent patterns in gene expression dataset</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S17-S12">PDF</a>
</li>
  <li>–2012–Shu-Lin Wang–BMC–<strong>Finding minimum gene subsets with heuristic breadth-first search algorithm for robust tumor classification</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-178">PDF</a>
</li>
  <li>–2012–Andrew F Neuwald–BMC–<strong>Automated hierarchical classification of protein domain subfamilies based on functionally-divergent residue signatures</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-144">PDF</a>
</li>
  <li>–2012–Carlo Bianca–BMC–<strong>Mathematical modeling of the immune system recognition to mammary carcinoma antigen</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S17-S21">PDF</a>
</li>
  <li>–2012–Jeffrey T Leek–BMC–<strong>A statistical approach to selecting and confirming validation targets in -omics experiments</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-150">PDF</a>
</li>
  <li>–2012–Susanna Zucca–BMC–<strong>Characterization of an inducible promoter in different DNA copy number conditions</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S4-S11">PDF</a>
</li>
  <li>–2012–Hua Tan–BMC–<strong>A Computational model for compressed sensing RNAi cellular screening</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-337">PDF</a>
</li>
  <li>–2012–Bernie J Daigle Jr–BMC–<strong>Accelerated maximum likelihood parameter estimation for stochastic biochemical systems</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-68">PDF</a>
</li>
  <li>–2012–Sean O’Callaghan–BMC–<strong>PyMS: a Python toolkit for processing of gas chromatography-mass spectrometry (GC-MS) data. Application and comparative study of selected tools</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-115">PDF</a>
</li>
  <li>–2012–Lakshmi Manohar Akella–BMC–<strong>NetiNeti: discovery of scientific names from text using machine learning methods</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-211">PDF</a>
</li>
  <li>–2012–Piotr Dittwald–BMC–<strong>Inferring serum proteolytic activity from LC-MS/MS data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S5-S7">PDF</a>
</li>
  <li>–2012–Lei Yuan–BMC–<strong>Learning Sparse Representations for Fruit-Fly Gene Expression Pattern Image Annotation and Retrieval</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-107">PDF</a>
</li>
  <li>–2012–Ruihua Fang–BMC–<strong>Automatic categorization of diverse experimental information in the bioscience literature</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-16">PDF</a>
</li>
  <li>–2012–Irene Sánchez-Linares–BMC–<strong>High-Throughput parallel blind Virtual Screening using BINDSURF</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S14-S13">PDF</a>
</li>
  <li>–2012–Rasna R Walia–BMC–<strong>Protein-RNA interface residue prediction using machine learning: an assessment of the state of the art</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-89">PDF</a>
</li>
  <li>–2012–Andrea Cornero–BMC–<strong>Design of a multi-signature ensemble classifier predicting neuroblastoma patients’ outcome</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S4-S13">PDF</a>
</li>
  <li>–2012–Alex–BMC–<strong>PHYLOViZ: phylogenetic inference and data visualization for sequence based typing methods</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-87">PDF</a>
</li>
  <li>–2012–Howard J Feldman–BMC–<strong>Identifying structural domains of proteins using clustering</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-286">PDF</a>
</li>
  <li>–2012–Lukasz P Kozlowski–BMC–<strong>MetaDisorder: a meta-server for the prediction of intrinsic disorder in proteins</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-111">PDF</a>
</li>
  <li>–2012–Riccardo Bellazzi–BMC–<strong>Clinical Bioinformatics: challenges and opportunities</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S14-S1">PDF</a>
</li>
  <li>–2012–Geet Duggal–BMC–<strong>Graph rigidity reveals well-constrained regions of chromosome conformation embeddings</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-241">PDF</a>
</li>
  <li>–2012–Akhil Kumar–BMC–<strong>MetRxn: a knowledgebase of metabolites and reactions spanning metabolic models and databases</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-6">PDF</a>
</li>
  <li>–2012–Arjun Ray–BMC–<strong>Improved model quality assessment using ProQ2</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-224">PDF</a>
</li>
  <li>–2012–B Jayaram–BMC–<strong>Sanjeevini: a freely accessible web-server for target directed lead molecule discovery</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S17-S7">PDF</a>
</li>
  <li>–2012–Yanni Sun–BMC–<strong>Genome-scale NCRNA homology search using a Hamming distance-based filtration strategy</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S3-S12">PDF</a>
</li>
  <li>–2012–Axel Skarman–BMC–<strong>A Bayesian variable selection procedure to rank overlapping gene sets</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-73">PDF</a>
</li>
  <li>–2012–Gad Abraham–BMC–<strong>SparSNP: Fast and memory-efficient analysis of all SNPs for phenotype prediction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-88">PDF</a>
</li>
  <li>–2012–Daniel S St–BMC–<strong>ParsEval: parallel comparison and analysis of gene structure annotations</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-187">PDF</a>
</li>
  <li>–2012–Mahmoud ElHefnawi–BMC–<strong>Multiple virtual screening approaches for finding new Hepatitis c virus RNA-dependent RNA polymerase inhibitors: Structure-based screens and molecular dynamics for the pursue of new poly pharmacological inhibitors</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S17-S5">PDF</a>
</li>
  <li>–2012–Steven Vercruysse–BMC–<strong>OLSVis: an animated, interactive visual browser for bio-ontologies</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-116">PDF</a>
</li>
  <li>–2012–Mohamed El-Kalioby–BMC–<strong>Personalized cloud-based bioinformatics services for research and education: use cases and the elasticHPC package</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S17-S22">PDF</a>
</li>
  <li>–2012–Alej–BMC–<strong>Federated ontology-based queries over cancer data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S1-S9">PDF</a>
</li>
  <li>–2012–Michele Pinelli–BMC–<strong>Simulating gene-gene and gene-environment interactions in complex diseases: Gene-Environment iNteraction Simulator 2</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-132">PDF</a>
</li>
  <li>–2012–Mohameth-François Sy–BMC–<strong>User centered and ontology based information retrieval system for life sciences</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S1-S4">PDF</a>
</li>
  <li>–2012–Ching-Wei Wang–BMC–<strong>Fast automatic quantitative cell replication with fluorescent live cell imaging</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-21">PDF</a>
</li>
  <li>–2012–Bin Li–BMC–<strong>Protein docking prediction using predicted protein-protein interface</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-7">PDF</a>
    <h2 id="2011">
<a class="anchor" href="#2011" aria-hidden="true"><span class="octicon octicon-link"></span></a>2011</h2>
  </li>
  <li>–2011–Byung-Jun Yoon–BMC–<strong>Enhanced stochastic optimization algorithm for finding effective multi-target therapeutics</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-S1-S18">PDF</a>
</li>
  <li>–2011–Peter Sykacek–BMC–<strong>The impact of quantitative optimization of hybridization conditions on gene expression analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-73">PDF</a>
</li>
  <li>–2011–Mi-Youn K Brusniak–BMC–<strong>ATAQS: A computational software tool for high throughput transition optimization and validation for selected reaction monitoring mass spectrometry</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-78">PDF</a>
</li>
  <li>–2011–Kyle C Chipman–BMC–<strong>Using Stochastic Causal Trees to Augment Bayesian Networks for Modeling eQTL Datasets</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-7">PDF</a>
</li>
  <li>–2011–Sang-Kyu Jung–BMC–<strong>Visual gene developer: a fully programmable bioinformatics software for synthetic gene optimization</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-340">PDF</a>
</li>
  <li>–2011–Cristian A Gallo–BMC–<strong>Discovering time-lagged rules from microarray data using gene profile classifiers</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-123">PDF</a>
</li>
  <li>–2011–Christian Bender–BMC–<strong>Inferring signalling networks from longitudinal data using sampling based approaches in the R-package ‘ddepn’</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-291">PDF</a>
</li>
  <li>–2011–Kahn Rhrissorrakrai–BMC–<strong>MINE: Module Identification in Networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-192">PDF</a>
</li>
  <li>–2011–Xiaofei Nan–BMC–<strong>Leveraging domain information to restructure biological prediction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-S10-S22">PDF</a>
</li>
  <li>–2011–Zafer Aydin–BMC–<strong>Learning sparse models for a dynamic Bayesian network classifier of protein secondary structure</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-154">PDF</a>
</li>
  <li>–2011–Giuseppe Gallone–BMC–<strong>Bio::Homology::InterologWalk - A Perl module to build putative protein-protein interaction networks through interolog mapping</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-289">PDF</a>
</li>
  <li>–2011–Weichen Wang–BMC–<strong>Network-based group variable selection for detecting expression quantitative trait loci (eQTL)</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-269">PDF</a>
</li>
  <li>–2011–Yao Fu–BMC–<strong>Reconstructing genome-wide regulatory network of E. coli using transcriptome data and predicted transcription factor activities</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-233">PDF</a>
</li>
  <li>–2011–Kai Wang–BMC–<strong>CASCADE_SCAN: mining signal transduction network from high-throughput data based on steepest descent method</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-164">PDF</a>
</li>
  <li>–2011–Kim-Anh Lê Cao–BMC–<strong>Sparse PLS discriminant analysis: biologically relevant feature selection and graphical displays for multiclass problems</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-253">PDF</a>
</li>
  <li>–2011–Hadas Birin–BMC–<strong>Efficient algorithms for reconstructing gene content by co-evolution</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-S9-S12">PDF</a>
</li>
  <li>–2011–Yongjin Park–BMC–<strong>Resolving the structure of interactomes with hierarchical agglomerative clustering</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-S1-S44">PDF</a>
</li>
  <li>–2011–Elon Correa–BMC–<strong>A genetic algorithm-Bayesian network approach for the analysis of metabolomics and spectroscopic data: application to the rapid identification of Bacillus spores and classification of Bacillus species</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-33">PDF</a>
</li>
  <li>–2011–Rosemary Braun–BMC–<strong>Partition decoupling for multi-gene analysis of gene expression profiling data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-497">PDF</a>
</li>
  <li>–2011–Xianwen Ren–BMC–<strong>Improving accuracy of protein-protein interaction prediction by considering the converse problem for sequence representation</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-409">PDF</a>
</li>
  <li>–2011–Domenico Fraccalvieri–BMC–<strong>Conformational and functional analysis of molecular dynamics trajectories by Self-Organising Maps</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-158">PDF</a>
</li>
  <li>–2011–Jiamin Xiao–BMC–<strong>Identification of microRNA precursors based on random forest with network-level representation method of stem-loop structure</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-165">PDF</a>
</li>
  <li>–2011–Gustavo Arellano–BMC–<strong>“Antelope”: a hybrid-logic model checker for branching-time Boolean GRN analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-490">PDF</a>
</li>
  <li>–2011–Edouard Pauwels–BMC–<strong>Predicting drug side-effect profiles: a chemical fragment-based approach</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-169">PDF</a>
</li>
  <li>–2011–Sangaalofa T Clark–BMC–<strong>A systems approach to identifying correlated gene targets for the loss of colour pigmentation in plants</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-343">PDF</a>
</li>
  <li>–2011–Ozgur Demir-Kavuk–BMC–<strong>Prediction using step-wise L1, L2 regularization and feature selection for small data sets with large number of features</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-412">PDF</a>
</li>
  <li>–2011–Daniel Restrepo-Montoya–BMC–<strong>NClassG+: A classifier for non-classically secreted Gram-positive bacterial proteins</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-21">PDF</a>
</li>
  <li>–2011–Jan Hasenauer–BMC–<strong>Identification of models of heterogeneous cell populations from population snapshot data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-125">PDF</a>
</li>
  <li>–2011–Patrick W Sheppard–BMC–<strong>Quantitative characterization and analysis of the dynamic NF-κB response in microglia</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-276">PDF</a>
</li>
  <li>–2011–Martin Krallinger–BMC–<strong>The Protein-Protein Interaction tasks of BioCreative III: classification/ranking of articles and linking bio-ontology concepts to full text</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-S8-S3">PDF</a>
</li>
  <li>–2011–Bailing Zhang–BMC–<strong>Phenotype Recognition with Combined Features and Random Subspace Classifier Ensemble</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-128">PDF</a>
</li>
  <li>–2011–George D Kritikos–BMC–<strong>Noise reduction in protein-protein interaction graphs by the implementation of a novel weighting scheme</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-239">PDF</a>
</li>
  <li>–2011–Alfredo Benso–BMC–<strong>Building gene expression profile classifiers with a simple and efficient rejection option in R</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-S13-S3">PDF</a>
</li>
  <li>–2011–Marcin Cieślik–BMC–<strong>A lightweight, flow-based toolkit for parallel and distributed bioinformatics pipelines</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-61">PDF</a>
</li>
  <li>–2011–Eduard Schreiner–BMC–<strong>Stereochemical errors and their implications for molecular dynamics simulations</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-190">PDF</a>
</li>
  <li>–2011–Usha K Muppirala–BMC–<strong>Predicting RNA-Protein Interactions Using Only Sequence Information</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-489">PDF</a>
</li>
  <li>–2011–Olga Domanova–BMC–<strong>Toponomics method for the automated quantification of membrane protein translocation</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-370">PDF</a>
</li>
  <li>–2011–Sorayya Malek–BMC–<strong>Assessment of predictive models for chlorophyll-a concentration of a tropical lake</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-S13-S12">PDF</a>
</li>
  <li>–2011–Arturo H Ariño–BMC–<strong>The Biodiversity Informatics Potential Index</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-S15-S4">PDF</a>
</li>
  <li>–2011–Ubbo Visser–BMC–<strong>BioAssay Ontology (BAO): a semantic description of bioassays and high-throughput screening results</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-257">PDF</a>
</li>
  <li>–2011–Ping Shi–BMC–<strong>Top scoring pairs for feature selection in machine learning and applications to cancer outcome prediction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-375">PDF</a>
</li>
  <li>–2011–Mark Needham–BMC–<strong>Hierarchical Parallelization of Gene Differential Association Analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-374">PDF</a>
</li>
  <li>–2011–Ivica Kopriva–BMC–<strong>A mixture model with a reference-based automatic selection of components for disease classification from protein and/or gene expression levels</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-496">PDF</a>
</li>
  <li>–2011–Richard Tzong-Han Tsai–BMC–<strong>Dynamic programming re-ranking for PPI interactor and pair extraction in full-text articles</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-60">PDF</a>
</li>
  <li>–2011–Alfredo A Kalaitzis–BMC–<strong>A Simple Approach to Ranking Differentially Expressed Gene Expression Time Courses through Gaussian Process Regression</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-180">PDF</a>
</li>
  <li>–2011–Hui Zhao–BMC–<strong>Query-based biclustering of gene expression data using Probabilistic Relational Models</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-S1-S37">PDF</a>
</li>
  <li>–2011–Fei Xia–BMC–<strong>FPGA accelerator for protein secondary structure prediction based on the GOR algorithm</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-S1-S5">PDF</a>
</li>
  <li>–2011–Suyu Mei–BMC–<strong>Gene ontology based transfer learning for protein subcellular localization</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-44">PDF</a>
</li>
  <li>–2011–Varun Khanna–BMC–<strong>In silico approach to screen compounds active against parasitic nematodes of major socio-economic importance</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-S13-S25">PDF</a>
</li>
  <li>–2011–Henry Wirth–BMC–<strong>Expression cartography of human tissues using self organizing maps</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-306">PDF</a>
</li>
  <li>–2011–Kailin Tang–BMC–<strong>Discrimination of approved drugs from experimental drugs by learning methods</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-157">PDF</a>
</li>
  <li>–2011–Hiroki Takahashi–BMC–<strong>AMDORAP: Non-targeted metabolic profiling based on high-resolution LC-MS</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-259">PDF</a>
</li>
  <li>–2011–Paul Thompson–BMC–<strong>The BioLexicon: a large-scale terminological resource for biomedical text mining</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-397">PDF</a>
</li>
  <li>–2011–Junliang Shang–BMC–<strong>Performance analysis of novel methods for detecting epistasis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-475">PDF</a>
</li>
  <li>–2011–Kathleen M Doherty–BMC–<strong>A multifaceted analysis of HIV-1 protease multidrug resistance phenotypes</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-477">PDF</a>
</li>
  <li>–2011–Yulong Wang–BMC–<strong>Determinants of antigenicity and specificity in immune response for protein sequences</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-251">PDF</a>
</li>
  <li>–2011–Jason Rudy–BMC–<strong>Empirical comparison of cross-platform normalization methods for gene expression data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-467">PDF</a>
</li>
  <li>–2011–Stephan Gade–BMC–<strong>Graph based fusion of miRNA and mRNA expression data improves clinical outcome prediction in prostate cancer</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-488">PDF</a>
</li>
  <li>–2011–Sabry Razick–BMC–<strong>iRefScape. A Cytoscape plug-in for visualization and data mining of protein interaction data from iRefIndex</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-388">PDF</a>
</li>
  <li>–2011–Yves-Pol Deniélou–BMC–<strong>Bacterial syntenies: an exact approach with gene quorum</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-193">PDF</a>
</li>
  <li>–2011–Andre J Faure–BMC–<strong>Investigating the effect of paralogs on microarray gene-set analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-29">PDF</a>
</li>
  <li>–2011–Natalia Becker–BMC–<strong>Elastic SCAD as a novel penalization method for SVM classification tasks in high-dimensional data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-138">PDF</a>
</li>
  <li>–2011–Ruoying Chen–BMC–<strong>Rigorous assessment and integration of the sequence and structure based features to predict hot spots</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-311">PDF</a>
</li>
  <li>–2011–Masaaki Kotera–BMC–<strong>MUCHA: multiple chemical alignment algorithm to identify building block substructures of orphan secondary metabolites</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-S14-S1">PDF</a>
</li>
  <li>–2011–Satya S Sahoo–BMC–<strong>A unified framework for managing provenance information in translational research</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-461">PDF</a>
</li>
  <li>–2011–Marcus Lechner–BMC–<strong>Proteinortho: Detection of (Co-)orthologs in large-scale analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-124">PDF</a>
</li>
  <li>–2011–Ernesto Iacucci–BMC–<strong>Predicting receptor-ligand pairs through kernel learning</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-336">PDF</a>
</li>
  <li>–2011–Ren-Xiang Yan–BMC–<strong>Outer membrane proteins can be simply identified using secondary structure element alignment</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-76">PDF</a>
</li>
  <li>–2011–Samuel C Flores–BMC–<strong>Predicting protein ligand binding motions with the conformation explorer</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-417">PDF</a>
</li>
  <li>–2011–Irina Tuszynska–BMC–<strong>DARS-RNP and QUASI-RNP: New statistical potentials for protein-RNA docking</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-348">PDF</a>
</li>
  <li>–2011–Paweł Daniluk–BMC–<strong>A novel method to compare protein structures using local descriptors</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-344">PDF</a>
</li>
  <li>–2011–Yoshiharu Sato–BMC–<strong>Meta-analytic approach to the accurate prediction of secreted virulence effectors in gram-negative bacteria</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-442">PDF</a>
</li>
  <li>–2011–Makiko Yoshida–BMC–<strong>SNPInterForest: A new method for detecting epistatic interactions</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-469">PDF</a>
</li>
  <li>–2011–Jonathon T Hill–BMC–<strong>Novel computational analysis of protein binding array data identifies direct targets of Nkx2.2 in the pancreas</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-62">PDF</a>
</li>
  <li>–2011–Xiaosheng Wang–BMC–<strong>Microarray-based cancer prediction using single genes</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-391">PDF</a>
</li>
  <li>–2011–Jacek Blazewicz–BMC–<strong>Protein alignment algorithms with an efficient backtracking routine on multiple GPUs</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-181">PDF</a>
    <h2 id="2010">
<a class="anchor" href="#2010" aria-hidden="true"><span class="octicon octicon-link"></span></a>2010</h2>
  </li>
  <li>–2010–Miha Štajdohar–BMC–<strong>FragViz: visualization of fragmented networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-475">PDF</a>
</li>
  <li>–2010–Martin T Swain–BMC–<strong>Comparative study of three commonly used continuous deterministic methods for modeling gene regulation networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-459">PDF</a>
</li>
  <li>–2010–Sheng He–BMC–<strong>LucidDraw: Efficiently visualizing complex biochemical networks within MATLAB</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-31">PDF</a>
</li>
  <li>–2010–Ioannis A Maraziotis–BMC–<strong>Gene regulatory networks modelling using a dynamic evolutionary hybrid</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-140">PDF</a>
</li>
  <li>–2010–Mara L Hartsperger–BMC–<strong>Structuring heterogeneous biological information using fuzzy clustering of k-partite graphs</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-522">PDF</a>
</li>
  <li>–2010–Kyoung Ae Kim–BMC–<strong>Systematic calibration of a cell signaling network model</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-202">PDF</a>
</li>
  <li>–2010–Kaname Kojima–BMC–<strong>An efficient biological pathway layout algorithm combining grid-layout and spring embedder for complicated cellular location information</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-335">PDF</a>
</li>
  <li>–2010–Linh M Tran–BMC–<strong>Trimming of mammalian transcriptional networks using network component analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-511">PDF</a>
</li>
  <li>–2010–Jongkwang Kim–BMC–<strong>Discover Protein Complexes in Protein-Protein Interaction Networks Using Parametric Local Modularity</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-521">PDF</a>
</li>
  <li>–2010–Adel Dayarian–BMC–<strong>SOPRA: Scaffolding algorithm for paired reads via statistical optimization</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-345">PDF</a>
</li>
  <li>–2010–Chen Wang–BMC–<strong>Knowledge-guided gene ranking by coordinative component analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-162">PDF</a>
</li>
  <li>–2010–Min Wu–BMC–<strong>Integrating diverse biological and computational sources for reliable protein-protein interactions</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-S7-S8">PDF</a>
</li>
  <li>–2010–Mitra Kabir–BMC–<strong>Reverse engineering gene regulatory network from microarray data using linear time-variant model</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-S1-S56">PDF</a>
</li>
  <li>–2010–Alina Sîrbu–BMC–<strong>Comparison of evolutionary algorithms in gene regulatory network model inference</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-59">PDF</a>
</li>
  <li>–2010–Alex–BMC–<strong>BisoGenet: a new tool for gene network building, visualization and analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-91">PDF</a>
</li>
  <li>–2010–S Yahya Anvar–BMC–<strong>The identification of informative genes from multiple datasets with increasing complexity</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-32">PDF</a>
</li>
  <li>–2010–Suresh Kumar Poovathingal–BMC–<strong>Global parameter estimation methods for stochastic biochemical systems</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-414">PDF</a>
</li>
  <li>–2010–Pingzhao Hu–BMC–<strong>Predicting protein functions by relaxation labelling protein interaction network</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-S1-S64">PDF</a>
</li>
  <li>–2010–Anup Parikh–BMC–<strong>New components of the Dictyostelium PKA pathway revealed by Bayesian analysis of expression data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-163">PDF</a>
</li>
  <li>–2010–Corban G Rivera–BMC–<strong>NeMo: Network Module identification in Cytoscape</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-S1-S61">PDF</a>
</li>
  <li>–2010–Subhadip Basu–BMC–<strong>AMS 3.0: prediction of post-translational modifications</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-210">PDF</a>
</li>
  <li>–2010–Steinn Gudmundsson–BMC–<strong>Computationally efficient flux variability analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-489">PDF</a>
</li>
  <li>–2010–Raffaele Di Natale–BMC–<strong>SING: Subgraph search In Non-homogeneous Graphs</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-96">PDF</a>
</li>
  <li>–2010–Shi Yu–BMC–<strong>Gene prioritization and clustering by multi-view text mining</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-28">PDF</a>
</li>
  <li>–2010–Anil Aswani–BMC–<strong>Nonparametric identification of regulatory interactions from spatial and temporal gene expression data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-413">PDF</a>
</li>
  <li>–2010–Jens Keilwagen–BMC–<strong>Apples and oranges: avoiding different priors in Bayesian DNA sequence analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-149">PDF</a>
</li>
  <li>–2010–Zhong-Ru Xie–BMC–<strong>An interaction-motif-based scoring function for protein-ligand docking</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-298">PDF</a>
</li>
  <li>–2010–Yu-Qing Qiu–BMC–<strong>Detecting disease associated modules and prioritizing active genes based on high throughput data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-26">PDF</a>
</li>
  <li>–2010–Martin Klammer–BMC–<strong>Identifying differentially regulated subnetworks from phosphoproteomic data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-351">PDF</a>
</li>
  <li>–2010–Junbai Wang–BMC–<strong>Quantitative model for inferring dynamic regulation of the tumour suppressor gene p53</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-36">PDF</a>
</li>
  <li>–2010–Benjamin Misselwitz–BMC–<strong>Enhanced CellClassifier: a multi-class classification tool for microscopy images</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-30">PDF</a>
</li>
  <li>–2010–Tianwei Y–BMC–<strong>An exploratory data analysis method to reveal modular latent structures in high-throughput data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-440">PDF</a>
</li>
  <li>–2010–Vinod Ch–BMC–<strong>MTar: a computational microRNA target prediction architecture for human transcriptome</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-S1-S2">PDF</a>
</li>
  <li>–2010–Shirley Hui–BMC–<strong>Proteome scanning to predict PDZ domain interactions using support vector machines</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-507">PDF</a>
</li>
  <li>–2010–Olof Görnerup–BMC–<strong>A model-independent approach to infer hierarchical codon substitution dynamics</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-201">PDF</a>
</li>
  <li>–2010–Ioannis K Valavanis–BMC–<strong>A multifactorial analysis of obesity as CVD risk factor: Use of neural network based methods in a nutrigenetics context</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-453">PDF</a>
</li>
  <li>–2010–Susmita Datta–BMC–<strong>An adaptive optimal ensemble classifier via bagging and rank aggregation with applications to high dimensional data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-427">PDF</a>
</li>
  <li>–2010–Shi Yu–BMC–<strong>L2-norm multiple kernel learning and its application to biomedical data fusion</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-309">PDF</a>
</li>
  <li>–2010–Garrett Jenkinson–BMC–<strong>Thermodynamically consistent Bayesian analysis of closed biochemical reaction systems</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-547">PDF</a>
</li>
  <li>–2010–Florian Battke–BMC–<strong>Mayday - integrative analytics for expression data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-121">PDF</a>
</li>
  <li>–2010–Jérôme Gracy–BMC–<strong>Optimizing structural modeling for a specific protein scaffold: knottins or inhibitor cystine knots</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-535">PDF</a>
</li>
  <li>–2010–Melissa J Morine–BMC–<strong>Bi-directional gene set enrichment and canonical correlation analysis identify key diet-sensitive pathways and biomarkers of metabolic syndrome</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-499">PDF</a>
</li>
  <li>–2010–Nobuyuki Uchikoga–BMC–<strong>Analysis of protein-protein docking decoys using interaction fingerprints: application to the reconstruction of CaM-ligand complexes</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-236">PDF</a>
</li>
  <li>–2010–Xiaojiang Xu–BMC–<strong>Application of machine learning methods to histone methylation ChIP-Seq data reveals H4R3me2 globally represses gene expression</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-396">PDF</a>
</li>
  <li>–2010–Daren Lee–BMC–<strong>MBAT: A scalable informatics system for unifying digital atlasing workflows</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-608">PDF</a>
</li>
  <li>–2010–Yosuke Ozawa–BMC–<strong>Protein complex prediction via verifying and reconstructing the topology of domain-domain interactions</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-350">PDF</a>
</li>
  <li>–2010–Ilenia Giangreco–BMC–<strong>Analysis of X-ray Structures of Matrix Metalloproteinases via Chaotic Map Clustering</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-500">PDF</a>
</li>
  <li>–2010–Marzio Pennisi–BMC–<strong>Modeling the competition between lung metastases and the immune system using agents</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-S7-S13">PDF</a>
</li>
  <li>–2010–Pekka Ruusuvuori–BMC–<strong>Evaluation of methods for detection of fluorescence labeled subcellular objects in microscope images</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-248">PDF</a>
</li>
  <li>–2010–Jan Schellenberger–BMC–<strong>BiGG: a Biochemical Genetic and Genomic knowledgebase of large scale metabolic reconstructions</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-213">PDF</a>
</li>
  <li>–2010–Lisbeth Carstensen–BMC–<strong>Multivariate Hawkes process models of the occurrence of regulatory elements</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-456">PDF</a>
</li>
  <li>–2010–Kenny Daily–BMC–<strong>Data structures and compression algorithms for high-throughput sequencing technologies</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-514">PDF</a>
</li>
  <li>–2010–Feng-Jie Sun–BMC–<strong>The ancient history of the structure of ribonuclease P and the early origins of Archaea</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-153">PDF</a>
</li>
  <li>–2010–Darby Tien-Hao Chang–BMC–<strong>Predicting the protein-protein interactions using primary structures with predicted protein surface</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-S1-S3">PDF</a>
</li>
  <li>–2010–István P Sugár–BMC–<strong>Misty Mountain clustering: application to fast unsupervised flow cytometry gating</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-502">PDF</a>
</li>
  <li>–2010–Prashanti M–BMC–<strong>GOModeler- A tool for hypothesis-testing of functional genomics datasets</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-S6-S29">PDF</a>
</li>
  <li>–2010–Jens Keilwagen–BMC–<strong>Unifying generative and discriminative learning principles</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-98">PDF</a>
</li>
  <li>–2010–Mirva Piippo–BMC–<strong>Pripper: prediction of caspase cleavage sites from whole proteomes</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-320">PDF</a>
</li>
  <li>–2010–SPT Krishnan–BMC–<strong>Towards high performance computing for molecular structure prediction using IBM Cell Broadband Engine - an implementation perspective</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-S1-S36">PDF</a>
</li>
  <li>–2010–Renaud Gaujoux–BMC–<strong>A flexible R package for nonnegative matrix factorization</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-367">PDF</a>
</li>
  <li>–2010–Jia Zeng–BMC–<strong>Multiconstrained gene clustering based on generalized projections</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-164">PDF</a>
</li>
  <li>–2010–Ernst Althaus–BMC–<strong>Computing H/D-Exchange rates of single residues from data of proteolytic fragments</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-424">PDF</a>
</li>
  <li>–2010–Matias Piipari–BMC–<strong>Metamotifs - a generative model for building families of nucleotide position weight matrices</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-348">PDF</a>
</li>
  <li>–2010–Petr Novák–BMC–<strong>Graph-based clustering and characterization of repetitive sequences in next-generation sequencing data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-378">PDF</a>
</li>
  <li>–2010–Jose M Duarte–BMC–<strong>Optimal contact definition for reconstruction of Contact Maps</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-283">PDF</a>
</li>
  <li>–2010–Pengyi Yang–BMC–<strong>A genetic ensemble approach for gene-gene interaction identification</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-524">PDF</a>
</li>
  <li>–2010–Patricia Francis-Lyon–BMC–<strong>Sampling the conformation of protein surface residues for flexible protein docking</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-575">PDF</a>
</li>
  <li>–2010–Aless–BMC–<strong>Structural alphabets derived from attractors in conformational space</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-97">PDF</a>
</li>
  <li>–2010–Alex D Stivala–BMC–<strong>Fast and accurate protein substructure searching with simulated annealing and GPUs</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-446">PDF</a>
</li>
  <li>–2010–Sebastien Tempel–BMC–<strong>ModuleOrganizer: detecting modules in families of transposable elements</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-474">PDF</a>
</li>
  <li>–2010–David-Olivier D Azulay–BMC–<strong>Visualisation tool for peptide fractionation data in proteomics: application to OFFGEL isoelectric focussing</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-371">PDF</a>
</li>
  <li>–2010–Russell L Zaretzki–BMC–<strong>Bias correction and Bayesian analysis of aggregate counts in SAGE libraries</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-72">PDF</a>
</li>
  <li>–2010–Zhanchao Li–BMC–<strong>Classification of G-protein coupled receptors based on support vector machine with maximum relevance minimum redundancy and genetic algorithm</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-325">PDF</a>
</li>
  <li>–2010–Christoph Bartenhagen–BMC–<strong>Comparative study of unsupervised dimension reduction techniques for the visualization of microarray gene expression data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-567">PDF</a>
</li>
  <li>–2010–Russell Schwartz–BMC–<strong>Applying unmixing to gene expression data for tumor phylogeny inference</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-42">PDF</a>
</li>
  <li>–2010–Cheng-Hong Yang–BMC–<strong>Confronting two-pair primer design for enzyme-free SNP genotyping based on a genetic algorithm</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-509">PDF</a>
</li>
  <li>–2010–Márton Münz–BMC–<strong>Dynamics based alignment of proteins: an alternative approach to quantify dynamic similarity</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-188">PDF</a>
</li>
  <li>–2010–Martin Mechelke–BMC–<strong>Robust probabilistic superposition and comparison of protein structures</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-363">PDF</a>
</li>
  <li>–2010–John C Dittmar–BMC–<strong>ScreenMill: A freely available software suite for growth measurement, analysis and visualization of high-throughput screen data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-353">PDF</a>
</li>
  <li>–2010–Andrew K Rider–BMC–<strong>A statistical approach to finding overlooked genetic associations</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-526">PDF</a>
</li>
  <li>–2010–Ashis Kumer Biswas–BMC–<strong>Machine learning approach to predict protein phosphorylation sites by incorporating evolutionary information</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-273">PDF</a>
</li>
  <li>–2010–Karel Zimmermann–BMC–<strong>Amino acid “little Big Bang”: Representing amino acid substitution matrices as dot products of Euclidian vectors</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-4">PDF</a>
</li>
  <li>–2010–Mariusz Popenda–BMC–<strong>RNA FRABASE 2.0: an advanced web-accessible database with the capacity to search the three-dimensional fragments within RNA structures</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-231">PDF</a>
    <h2 id="2009">
<a class="anchor" href="#2009" aria-hidden="true"><span class="octicon octicon-link"></span></a>2009</h2>
  </li>
  <li>–2009–Steffen Brasch–BMC–<strong>VANLO - Interactive visual exploration of aligned biological networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-327">PDF</a>
</li>
  <li>–2009–Falk Schreiber–BMC–<strong>A generic algorithm for layout of biological networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-375">PDF</a>
</li>
  <li>–2009–Franck Picard–BMC–<strong>Deciphering the connectivity structure of biological networks using MixNet</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S6-S17">PDF</a>
</li>
  <li>–2009–Tatsunori B Hashimoto–BMC–<strong>BFL: a node and edge betweenness based fast layout algorithm for large scale networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-19">PDF</a>
</li>
  <li>–2009–James Anderson–BMC–<strong>On validation and invalidation of biological models</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-132">PDF</a>
</li>
  <li>–2009–Tomáš Hirmajer–BMC–<strong>DOTcvpSB, a software toolbox for dynamic optimization in systems biology</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-199">PDF</a>
</li>
  <li>–2009–Xinan Yang–BMC–<strong>Mechanism-anchored profiling derived from epigenetic networks predicts outcome in acute lymphoblastic leukemia</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S9-S6">PDF</a>
</li>
  <li>–2009–David J Klinke II–BMC–<strong>An empirical Bayesian approach for model-based inference of cellular signaling networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-371">PDF</a>
</li>
  <li>–2009–George Zheng–BMC–<strong>Service-based analysis of biological pathways</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S10-S6">PDF</a>
</li>
  <li>–2009–Kim-Anh Lê Cao–BMC–<strong>Sparse canonical methods for biological data integration: application to a cross-platform study</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-34">PDF</a>
</li>
  <li>–2009–Feng Luo–BMC–<strong>Core and periphery structures in protein interaction networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S4-S8">PDF</a>
</li>
  <li>–2009–Gonzalo Guillén-Gosálbez–BMC–<strong>Identifying quantitative operation principles in metabolic pathways: a systematic method for searching feasible enzyme activity patterns leading to cellular adaptive responses</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-386">PDF</a>
</li>
  <li>–2009–Holger Fröhlich–BMC–<strong>Deterministic Effects Propagation Networks for reconstructing protein signaling networks from multiple interventions</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-322">PDF</a>
</li>
  <li>–2009–Rui-Sheng Wang–BMC–<strong>Modeling post-transcriptional regulation activity of small non-coding RNAs in Escherichia coli</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S4-S6">PDF</a>
</li>
  <li>–2009–Steve Pettifer–BMC–<strong>Visualising biological data: a semantic approach to tool and database integration</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S6-S19">PDF</a>
</li>
  <li>–2009–Johanna Mazur–BMC–<strong>Reconstructing nonlinear dynamic models of gene regulation using stochastic sampling</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-448">PDF</a>
</li>
  <li>–2009–Gunnar W Kla–BMC–<strong>A new graph-based method for pairwise global network alignment</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S1-S59">PDF</a>
</li>
  <li>–2009–Yanni Zhu–BMC–<strong>Network-based support vector machine for classification of microarray samples</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S1-S21">PDF</a>
</li>
  <li>–2009–James Vlasblom–BMC–<strong>Markov clustering versus affinity propagation for the partitioning of protein interaction graphs</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-99">PDF</a>
</li>
  <li>–2009–Ujjwal Maulik–BMC–<strong>Combining Pareto-optimal clusters using supervised learning for identifying co-expressed genes</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-27">PDF</a>
</li>
  <li>–2009–Michael Hecker–BMC–<strong>Integrative modeling of transcriptional regulation in response to antirheumatic therapy</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-262">PDF</a>
</li>
  <li>–2009–Luciano Milanesi–BMC–<strong>Trends in modeling Biomedical Complex Systems</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S12-I1">PDF</a>
</li>
  <li>–2009–Zixiang Xu–BMC–<strong>Genome-scale analysis to the impact of gene deletion on the metabolism of E. coli: constraint-based simulation approach</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S1-S62">PDF</a>
</li>
  <li>–2009–Yang Li–BMC–<strong>designGG: an R-package and web tool for the optimal design of genetical genomics experiments</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-188">PDF</a>
</li>
  <li>–2009–Ettore Mosca–BMC–<strong>Identification of functionally related genes using data mining and data integration: a breast cancer case study</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S12-S8">PDF</a>
</li>
  <li>–2009–Yiming Ying–BMC–<strong>Enhanced protein fold recognition through a novel data integration approach</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-267">PDF</a>
</li>
  <li>–2009–Atsushi Niida–BMC–<strong>Gene set-based module discovery in the breast cancer transcriptome</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-71">PDF</a>
</li>
  <li>–2009–Kevin Y Yip–BMC–<strong>Multi-level learning: improving the prediction of protein, domain and residue interactions by allowing information flow between levels</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-241">PDF</a>
</li>
  <li>–2009–Christopher W Maier–BMC–<strong>Ultra-Structure database design methodology for managing systems biology data and analyses</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-254">PDF</a>
</li>
  <li>–2009–Junwan Liu–BMC–<strong>Biclustering of microarray data with MOSPO based on crowding distance</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S4-S9">PDF</a>
</li>
  <li>–2009–Iftach Nachman–BMC–<strong>BRNI: Modular analysis of transcriptional regulatory programs</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-155">PDF</a>
</li>
  <li>–2009–Cuilan Gao–BMC–<strong>Graph ranking for exploratory gene data analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S11-S19">PDF</a>
</li>
  <li>–2009–Emanuel Schwarz–BMC–<strong>Clinical bioinformatics for complex disorders: a schizophrenia case study</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S12-S6">PDF</a>
</li>
  <li>–2009–Marcello Castellano–BMC–<strong>A bioinformatics knowledge discovery in text application for grid computing</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S6-S23">PDF</a>
</li>
  <li>–2009–Filippo Menolascina–BMC–<strong>Developing optimal input design strategies in cancer systems biology with applications to microfluidic device engineering</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S12-S4">PDF</a>
</li>
  <li>–2009–James R Green–BMC–<strong>PCI-SS: MISO dynamic nonlinear protein secondary structure prediction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-222">PDF</a>
</li>
  <li>–2009–Chengpeng Bi–BMC–<strong>DNA motif alignment by evolving a population of Markov chains</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S1-S13">PDF</a>
</li>
  <li>–2009–Nazar Zaki–BMC–<strong>Protein-protein interaction based on pairwise similarity</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-150">PDF</a>
</li>
  <li>–2009–Gregor Rot–BMC–<strong>dictyExpress: a Dictyostelium discoideum gene expression database with an explorative data analysis web-based interface</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-265">PDF</a>
</li>
  <li>–2009–Michalis Zervakis–BMC–<strong>Outcome prediction based on microarray analysis: a critical perspective on methods</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-53">PDF</a>
</li>
  <li>–2009–Michael Dondrup–BMC–<strong>EMMA 2 – A MAGE-compliant system for the collaborative analysis and integration of microarray data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-50">PDF</a>
</li>
  <li>–2009–Paola Bertolazzi–BMC–<strong>Learning to classify species with barcodes</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S14-S7">PDF</a>
</li>
  <li>–2009–Jignesh R Parikh–BMC–<strong>multiplierz: an extensible API based desktop environment for proteomics data analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-364">PDF</a>
</li>
  <li>–2009–Gernot Stocker–BMC–<strong>iLAP: a workflow-driven software for experimental protocol development, data acquisition and analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-390">PDF</a>
</li>
  <li>–2009–Enrico Glaab–BMC–<strong>ArrayMining: a modular web-application for microarray analysis combining ensemble and consensus methods with cross-study normalization</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-358">PDF</a>
</li>
  <li>–2009–Göran Sperber–BMC–<strong>RetroTector online, a rational tool for analysis of retroviral elements in small and medium size vertebrate genomic sequences</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S6-S4">PDF</a>
</li>
  <li>–2009–Helena Strömbergsson–BMC–<strong>A chemogenomics view on protein-ligand spaces</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S6-S13">PDF</a>
</li>
  <li>–2009–Erick Antezana–BMC–<strong>BioGateway: a semantic systems biology tool for the life sciences</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S10-S11">PDF</a>
</li>
  <li>–2009–Stefan R Maetschke–BMC–<strong>Exploiting structural and topological information to improve prediction of RNA-protein binding sites</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-341">PDF</a>
</li>
  <li>–2009–María del Mar Roldán-García–BMC–<strong>KA-SB: from data integration to large scale reasoning</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S10-S5">PDF</a>
</li>
  <li>–2009–Wei Guan–BMC–<strong>Ovarian cancer detection from metabolomic liquid chromatography/mass spectrometry data by support vector machines</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-259">PDF</a>
</li>
  <li>–2009–Yi Jia–BMC–<strong>Towards comprehensive structural motif mining for better fold annotation in the “twilight zone” of sequence dissimilarity</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S1-S46">PDF</a>
</li>
  <li>–2009–An-Min Zou–BMC–<strong>Quality assessment of tandem mass spectra using support vector machine (SVM)</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S1-S49">PDF</a>
</li>
  <li>–2009–Flavien Quintus–BMC–<strong>Ligand scaffold hopping combining 3D maximal substructure search and molecular similarity</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-245">PDF</a>
</li>
  <li>–2009–Monica Chiogna–BMC–<strong>A comparison on effects of normalisations in the detection of differentially expressed genes</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-61">PDF</a>
</li>
  <li>–2009–Aless–BMC–<strong>A Perl procedure for protein identification by Peptide Mass Fingerprinting</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S12-S11">PDF</a>
</li>
  <li>–2009–Bo Jin–BMC–<strong>Feature selection for fMRI-based deception detection</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S9-S15">PDF</a>
</li>
  <li>–2009–Fatemeh Zare-Mirakabad–BMC–<strong>New scoring schema for finding motifs in DNA Sequences</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-93">PDF</a>
</li>
  <li>–2009–Martin H van Vliet–BMC–<strong>Knowledge driven decomposition of tumor expression profiles</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S1-S20">PDF</a>
</li>
  <li>–2009–Lei Deng–BMC–<strong>Prediction of protein-protein interaction sites using an ensemble method</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-426">PDF</a>
</li>
  <li>–2009–Sung Hee Park–BMC–<strong>Prediction of protein-protein interaction types using association rule based classification</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-36">PDF</a>
</li>
  <li>–2009–Vitali Sintchenko–BMC–<strong>Towards bioinformatics assisted infectious disease control</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S2-S10">PDF</a>
</li>
  <li>–2009–Timothy Nugent–BMC–<strong>Transmembrane protein topology prediction using support vector machines</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-159">PDF</a>
</li>
  <li>–2009–Sebastian Schönherr–BMC–<strong>eCOMPAGT – efficient Combination and Management of Phenotypes and Genotypes for Genetic Epidemiology</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-139">PDF</a>
</li>
  <li>–2009–Frank PY Lin–BMC–<strong>In silico prioritisation of candidate genes for prokaryotic gene function discovery: an application of phylogenetic profiles</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-86">PDF</a>
</li>
  <li>–2009–Waranyu Wongseree–BMC–<strong>Detecting purely epistatic multi-locus interactions by an omnibus permutation test on ensembles of two-locus analyses</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-294">PDF</a>
</li>
  <li>–2009–Anna Korhonen–BMC–<strong>The first step in the development of text mining technology for cancer risk assessment: identifying and organizing scientific evidence in risk assessment literature</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-303">PDF</a>
</li>
  <li>–2009–Varun Khanna–BMC–<strong>Physicochemical property space distribution among human metabolites, drugs and toxins</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S15-S10">PDF</a>
</li>
  <li>–2009–Ren-Xiang Yan–BMC–<strong>DescFold: A web server for protein fold recognition</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-416">PDF</a>
</li>
  <li>–2009–Christian Schwager–BMC–<strong>TableButler – a Windows based tool for processing large data tables generated with high-throughput methods</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-235">PDF</a>
</li>
  <li>–2009–Richard D Pearson–BMC–<strong>puma: a Bioconductor package for propagating uncertainty in microarray analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-211">PDF</a>
</li>
  <li>–2009–Federica Chiappori–BMC–<strong>In silico saturation mutagenesis and docking screening for the analysis of protein-ligand interaction: the Endothelial Protein C Receptor case study</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S12-S3">PDF</a>
</li>
  <li>–2009–Antonis Koussounadis–BMC–<strong>Improving classification in protein structure databases using text mining</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-129">PDF</a>
    <h2 id="2008">
<a class="anchor" href="#2008" aria-hidden="true"><span class="octicon octicon-link"></span></a>2008</h2>
  </li>
  <li>–2008–Liang Sun–BMC–<strong>Adaptive diffusion kernel learning from biological networks for protein function prediction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-162">PDF</a>
</li>
  <li>–2008–Erwin P Gianch–BMC–<strong>Predicting biological system objectives de novo from internal state measurements</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-43">PDF</a>
</li>
  <li>–2008–Meng P Tan–BMC–<strong>Microarray data mining: A novel optimization-based approach to uncover biologically coherent structures</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-268">PDF</a>
</li>
  <li>–2008–Miguel Rocha–BMC–<strong>Natural computation meta-heuristics for the in silico optimization of microbial strains</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-499">PDF</a>
</li>
  <li>–2008–Iaroslav Ispolatov–BMC–<strong>Detection of the dominant direction of information flow and feedback links in densely interconnected regulatory networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-424">PDF</a>
</li>
  <li>–2008–Yuji Zhang–BMC–<strong>Network motif-based identification of transcription factor-target gene relationships by integrating multi-source biological data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-203">PDF</a>
</li>
  <li>–2008–Philippe Veber–BMC–<strong>Inferring the role of transcription factors in regulatory networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-228">PDF</a>
</li>
  <li>–2008–Nicola Zamboni–BMC–<strong>anNET: a tool for network-embedded thermodynamic analysis of quantitative metabolome data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-199">PDF</a>
</li>
  <li>–2008–Wolfram Gronwald–BMC–<strong>Evolutionary Pareto-optimization of stably folding peptides</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-109">PDF</a>
</li>
  <li>–2008–Cédric Auliac–BMC–<strong>Evolutionary approaches for the reverse-engineering of gene regulatory networks: A study on a biologically realistic dataset</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-91">PDF</a>
</li>
  <li>–2008–Alfredo Braunstein–BMC–<strong>Estimating the size of the solution space of metabolic networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-240">PDF</a>
</li>
  <li>–2008–Rileen Sinha–BMC–<strong>Improved identification of conserved cassette exons using Bayesian networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-477">PDF</a>
</li>
  <li>–2008–Cuong C To–BMC–<strong>Supervised inference of gene-regulatory networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-2">PDF</a>
</li>
  <li>–2008–Grace S Shieh–BMC–<strong>Inferring transcriptional compensation interactions in yeast via stepwise structure equation modeling</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-134">PDF</a>
</li>
  <li>–2008–Chen Wang–BMC–<strong>Motif-directed network component analysis for regulatory network inference</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S1-S21">PDF</a>
</li>
  <li>–2008–Rodrigo Santamaría–BMC–<strong>A visual analytics approach for understanding biclustering results from microarray data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-247">PDF</a>
</li>
  <li>–2008–Ari Rantanen–BMC–<strong>An analytic and systematic framework for estimating metabolic flux ratios from 13C tracer experiments</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-266">PDF</a>
</li>
  <li>–2008–Peter A DiMaggio Jr–BMC–<strong>Biclustering via optimal re-ordering of data matrices in systems biology: rigorous methods and comparative studies</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-458">PDF</a>
</li>
  <li>–2008–Harri T Kiiveri–BMC–<strong>A general approach to simultaneous model fitting and variable elimination in response models for biological data with many more variables than observations</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-195">PDF</a>
</li>
  <li>–2008–Aswath Manoharan–BMC–<strong>Optimizations for the EcoPod field identification tool</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-150">PDF</a>
</li>
  <li>–2008–Yinyin Yuan–BMC–<strong>Partial mixture model for tight clustering of gene expression time-course</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-287">PDF</a>
</li>
  <li>–2008–Katia S Guimarães–BMC–<strong>Interrogating domain-domain interactions with parsimony based approaches</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-171">PDF</a>
</li>
  <li>–2008–Irene Kouskoumvekaki–BMC–<strong>Identification of biomarkers for genotyping Aspergilli using non-linear methods for clustering and classification</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-59">PDF</a>
</li>
  <li>–2008–Marko Tscherepanow–BMC–<strong>An incremental approach to automated protein localisation</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-445">PDF</a>
</li>
  <li>–2008–Silvana Penco–BMC–<strong>New application of intelligent agents in sporadic amyotrophic lateral sclerosis identifies unexpected specific genetic background</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-254">PDF</a>
</li>
  <li>–2008–Yitan Zhu–BMC–<strong>caBIG™ VISDA: Modeling, visualization, and discovery for cluster analysis of genomic data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-383">PDF</a>
</li>
  <li>–2008–Ismael Navas-Delgado–BMC–<strong>AMMO-Prot: amine system project 3D-model finder</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S4-S5">PDF</a>
</li>
  <li>–2008–Cheng-Wei Cheng–BMC–<strong>Predicting RNA-binding sites of proteins using support vector machines and evolutionary information</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S12-S6">PDF</a>
</li>
  <li>–2008–Zhenqiu Liu–BMC–<strong>Gene and pathway identification with L
p
penalized Bayesian logistic regression</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-412">PDF</a>
</li>
  <li>–2008–Michael Gormley–BMC–<strong>Expression profiles of switch-like genes accurately classify tissue and infectious disease phenotypes in model-based classification</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-486">PDF</a>
</li>
  <li>–2008–Johannes Tuikkala–BMC–<strong>Missing value imputation improves clustering and interpretation of gene expression microarray data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-202">PDF</a>
</li>
  <li>–2008–Xiangchao Gan–BMC–<strong>Discovering biclusters in gene expression data based on high-dimensional linear geometries</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-209">PDF</a>
</li>
  <li>–2008–Zhenqiang Su–BMC–<strong>Very Important Pool (VIP) genes – an application for microarray-based molecular signatures</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S9-S9">PDF</a>
</li>
  <li>–2008–Wen-Lin Huang–BMC–<strong>ProLoc-GO: Utilizing informative Gene Ontology terms for sequence-based prediction of protein subcellular localization</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-80">PDF</a>
</li>
  <li>–2008–Celine Scornavacca–BMC–<strong>PhySIC_IST: cleaning source trees to infer more informative supertrees</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-413">PDF</a>
</li>
  <li>–2008–Chiara Parravicini–BMC–<strong>GPR17: Molecular modeling and dynamics studies of the 3-D structure and purinergic ligand binding features in comparison with P2Y receptors</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-263">PDF</a>
</li>
  <li>–2008–Rainer Merkl–BMC–<strong>H2r: Identification of evolutionary important residues by means of an entropy based analysis of multiple sequence alignments</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-151">PDF</a>
</li>
  <li>–2008–Markus Bundschus–BMC–<strong>Extraction of semantic biomedical relations from text using conditional random fields</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-207">PDF</a>
</li>
  <li>–2008–Paulo C Carvalho–BMC–<strong>PatternLab for proteomics: a tool for differential shotgun proteomics</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-316">PDF</a>
</li>
  <li>–2008–Yan Xin Huang–BMC–<strong>Pep-3D-Search: a method for B-cell epitope prediction based on mimotope analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-538">PDF</a>
</li>
  <li>–2008–Paul D Yoo–BMC–<strong>SiteSeek: Post-translational modification analysis using adaptive locality-effective kernel methods and new profiles</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-272">PDF</a>
</li>
  <li>–2008–Huiling Xiong–BMC–<strong>Using Generalized Procrustes Analysis (GPA) for normalization of cDNA microarray data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-25">PDF</a>
</li>
  <li>–2008–Kieran O’Neill–BMC–<strong>OntoDas – a tool for facilitating the construction of complex queries to the Gene Ontology</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-437">PDF</a>
</li>
  <li>–2008–Madhavi Ganapathiraju–BMC–<strong>Transmembrane helix prediction using amino acid property features and latent semantic analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S1-S4">PDF</a>
</li>
  <li>–2008–Pelin Armutlu–BMC–<strong>Classification of drug molecules considering their IC50 values using mixed-integer linear programming based hyper-boxes method</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-411">PDF</a>
</li>
  <li>–2008–Thierry Rème–BMC–<strong>A new method for class prediction based on signed-rank algorithms applied to Affymetrix® microarray experiments</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-16">PDF</a>
</li>
  <li>–2008–AKMA Baten–BMC–<strong>Fast splice site detection using information content and feature reduction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S12-S8">PDF</a>
</li>
  <li>–2008–Ce Zheng–BMC–<strong>Prediction of beta-turns at over 80% accuracy based on an ensemble of predicted secondary structures and multiple alignments</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-430">PDF</a>
</li>
  <li>–2008–Lianyi Han–BMC–<strong>Developing and validating predictive decision tree models from mining chemical structural fingerprints and high–throughput screening data in PubChem</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-401">PDF</a>
</li>
  <li>–2008–Gregory A Wilkin–BMC–<strong>A practical comparison of two K-Means clustering algorithms</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S6-S19">PDF</a>
</li>
  <li>–2008–Nan Li–BMC–<strong>Prediction of protein-protein binding site by using core interface residue and support vector machine</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-553">PDF</a>
</li>
  <li>–2008–Maris Lapins–BMC–<strong>Proteochemometric modeling of HIV protease susceptibility</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-181">PDF</a>
</li>
  <li>–2008–Rekin’s Janky–BMC–<strong>Evaluation of phylogenetic footprint discovery for predicting bacterial cis-regulatory elements and revealing their evolution</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-37">PDF</a>
</li>
  <li>–2008–Daniel Quest–BMC–<strong>MTAP: The Motif Tool Assessment Platform</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S9-S6">PDF</a>
</li>
  <li>–2008–Dadabhai T Singh–BMC–<strong>Comparative phyloinformatics of virus genes at micro and macro levels in a distributed computing environment</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S1-S23">PDF</a>
</li>
  <li>–2008–Emmanuel Perrodou–BMC–<strong>A new protein linear motif benchmark for multiple sequence alignment software</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-213">PDF</a>
</li>
  <li>–2008–Jonathan R Manning–BMC–<strong>The contrasting properties of conservation and correlated phylogeny in protein functional residue prediction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-51">PDF</a>
</li>
  <li>–2008–Hongya Zhao–BMC–<strong>Multivariate hierarchical Bayesian model for differential gene expression analysis in microarray experiments</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S1-S9">PDF</a>
</li>
  <li>–2008–Hiroyuki Mishima–BMC–<strong>Application of the Linux cluster for exhaustive window haplotype analysis using the FBAT and Unphased programs</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S6-S10">PDF</a>
</li>
  <li>–2008–Bernhard Gschloessl–BMC–<strong>HECTAR: A method to predict subcellular targeting in heterokonts</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-393">PDF</a>
</li>
  <li>–2008–Shen Jean Lim–BMC–<strong>The value of position-specific scoring matrices for assessment of protein allegenicity</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S12-S21">PDF</a>
</li>
  <li>–2008–Yuchen Yang–BMC–<strong>MiRTif: a support vector machine-based microRNA target interaction filter</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S12-S4">PDF</a>
</li>
  <li>–2008–Ramon Diaz-Uriarte–BMC–<strong>SignS: a parallelized, open-source, freely available, web-based tool for gene selection and molecular signatures for survival and censored data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-30">PDF</a>
</li>
  <li>–2008–Firoz Anwar–BMC–<strong>Pol II promoter prediction using characteristic 4-mer motifs: a machine learning approach</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-414">PDF</a>
</li>
  <li>–2008–Guo-Zheng Li–BMC–<strong>Asymmetric bagging and feature selection for activities prediction of drug molecules</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S6-S7">PDF</a>
</li>
  <li>–2008–Michael Richter–BMC–<strong>JCoast – A biologist-centric software tool for data mining and comparison of prokaryotic (meta)genomes</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-177">PDF</a>
</li>
  <li>–2008–Abhishek Tripathi–BMC–<strong>Simple integrative preprocessing preserves what is shared in data sources</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-111">PDF</a>
</li>
  <li>–2008–Catia Pesquita–BMC–<strong>Metrics for GO based protein semantic similarity: a systematic evaluation</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S5-S4">PDF</a>
</li>
  <li>–2008–Yumi Mizuno–BMC–<strong>Predicted mouse peroxisome-targeted proteins and their actual subcellular locations</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S12-S16">PDF</a>
</li>
  <li>–2008–Claudia Chica–BMC–<strong>A tree-based conservation scoring method for short linear motifs in multiple alignments of protein sequences</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-229">PDF</a>
</li>
  <li>–2008–Qingwu Yang–BMC–<strong>Predicting protein folding pathways at the mesoscopic level based on native interactions between secondary structure elements</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-320">PDF</a>
</li>
  <li>–2008–Sabry Razick–BMC–<strong>iRefIndex: A consolidated protein interaction database with provenance</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-405">PDF</a>
</li>
  <li>–2008–Luigi Buonaguro–BMC–<strong>Gene expression profile of peripheral blood mononuclear cells in response to HIV-VLPs stimulation</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S2-S5">PDF</a>
    <h2 id="2007">
<a class="anchor" href="#2007" aria-hidden="true"><span class="octicon octicon-link"></span></a>2007</h2>
  </li>
  <li>–2007–Reuben Thomas–BMC–<strong>A model-based optimization framework for the inference of regulatory interactions using time-course DNA microarray expression data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-228">PDF</a>
</li>
  <li>–2007–Jean-Philippe Vert–BMC–<strong>A new pairwise kernel for biological network inference with support vector machines</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S10-S8">PDF</a>
</li>
  <li>–2007–Kaname Kojima–BMC–<strong>An efficient grid layout algorithm for biological networks utilizing various biological attributes</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-76">PDF</a>
</li>
  <li>–2007–Anastasia Yartseva–BMC–<strong>Incremental and unifying modelling formalism for biological interaction networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-433">PDF</a>
</li>
  <li>–2007–Xiaohua Hu–BMC–<strong>Mining and state-space modeling and verification of sub-networks from large-scale biomolecular networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-324">PDF</a>
</li>
  <li>–2007–Mudita Singhal–BMC–<strong>A domain-based approach to predict protein-protein interactions</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-199">PDF</a>
</li>
  <li>–2007–Brian J Holden–BMC–<strong>An exploration of alternative visualisations of the basic helix-loop-helix protein interaction network</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-289">PDF</a>
</li>
  <li>–2007–Franck Rapaport–BMC–<strong>Classification of microarray data using gene networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-35">PDF</a>
</li>
  <li>–2007–Markus Bauer–BMC–<strong>Accurate multiple sequence-structure alignment of RNA sequences using combinatorial optimization</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-271">PDF</a>
</li>
  <li>–2007–Florian Markowetz–BMC–<strong>Inferring cellular networks – a review</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S6-S5">PDF</a>
</li>
  <li>–2007–Michael Weitzel–BMC–<strong>The topology of metabolic isotope labeling networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-315">PDF</a>
</li>
  <li>–2007–Jongrae Kim–BMC–<strong>Least-squares methods for identifying biochemical regulatory networks from noisy measurements</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-8">PDF</a>
</li>
  <li>–2007–Rainer Opgen-Rhein–BMC–<strong>Learning causal networks from systems biology time course data: an effective model selection procedure for the vector autoregressive process</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S2-S3">PDF</a>
</li>
  <li>–2007–Tom Michoel–BMC–<strong>Validating module network learning algorithms using simulated data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S2-S5">PDF</a>
</li>
  <li>–2007–Aless–BMC–<strong>Dynamic simulation of regulatory networks using SQUAD</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-462">PDF</a>
</li>
  <li>–2007–Marco Vilela–BMC–<strong>Automated smoother for the numerical decoupling of dynamics models</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-305">PDF</a>
</li>
  <li>–2007–Chang Sik Kim–BMC–<strong>Bayesian Orthogonal Least Squares (BOLS) algorithm for reverse engineering of gene regulatory networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-251">PDF</a>
</li>
  <li>–2007–Doheon Lee–BMC–<strong>BioCAD: an information fusion platform for bio-network inference and analysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S9-S2">PDF</a>
</li>
  <li>–2007–Suman Datta–BMC–<strong>Accelerated search for biomolecular network models to interpret high-throughput experimental data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-258">PDF</a>
</li>
  <li>–2007–Carlos Rodríguez-Penagos–BMC–<strong>Automatic reconstruction of a bacterial regulatory network using Natural Language Processing</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-293">PDF</a>
</li>
  <li>–2007–Blaise Gassend–BMC–<strong>Learning biophysically-motivated parameters for alpha helix prediction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S5-S3">PDF</a>
</li>
  <li>–2007–Ana P Teixeira–BMC–<strong>Hybrid elementary flux analysis/nonparametric modeling: application for bioprocess control</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-30">PDF</a>
</li>
  <li>–2007–Matthew Yeung–BMC–<strong>Estimation of the number of extreme pathways for metabolic networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-363">PDF</a>
</li>
  <li>–2007–Eunkyoung Jung–BMC–<strong>Artificial neural network models for prediction of intestinal permeability of oligopeptides</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-245">PDF</a>
</li>
  <li>–2007–Guillaume Launay–BMC–<strong>Recognizing protein–protein interfaces with empirical potentials and reduced amino acid alphabets</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-270">PDF</a>
</li>
  <li>–2007–Hon Nian Chua–BMC–<strong>Using indirect protein interactions for the prediction of Gene Ontology functions</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S4-S8">PDF</a>
</li>
  <li>–2007–Johannes J M–BMC–<strong>Modeling biochemical transformation processes and information processing with Narrator</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-103">PDF</a>
</li>
  <li>–2007–Markus Koschorreck–BMC–<strong>Reduced modeling of signal transduction – a modular approach</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-336">PDF</a>
</li>
  <li>–2007–Shawn Cokus–BMC–<strong>An improved method for identifying functionally linked proteins using phylogenetic profiles</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S4-S7">PDF</a>
</li>
  <li>–2007–Holger Froehlich–BMC–<strong>Large scale statistical inference of signaling pathways from RNAi and microarray data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-386">PDF</a>
</li>
  <li>–2007–Lars Kuepfer–BMC–<strong>Efficient classification of complete parameter regions based on semidefinite programming</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-12">PDF</a>
</li>
  <li>–2007–Kyoung-Jae Won–BMC–<strong>An evolutionary method for learning HMM structure: prediction of protein secondary structure</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-357">PDF</a>
</li>
  <li>–2007–Jingjing Ye–BMC–<strong>On the analysis of glycomics mass spectrometry data via the regularized area under the ROC curve</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-477">PDF</a>
</li>
  <li>–2007–Wataru Fujibuchi–BMC–<strong>Classification of heterogeneous microarray data by maximum entropy kernel</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-267">PDF</a>
</li>
  <li>–2007–Arijit Chakravarty–BMC–<strong>A novel ensemble learning method for de novo computational identification of DNA binding sites</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-249">PDF</a>
</li>
  <li>–2007–Andrew S Peek–BMC–<strong>Improving model predictions for RNA interference activities that use support vector machine regression by combining and filtering features</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-182">PDF</a>
</li>
  <li>–2007–Limin Fu–BMC–<strong>FLAME, a novel fuzzy clustering method for the analysis of DNA microarray data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-3">PDF</a>
</li>
  <li>–2007–Henning Redestig–BMC–<strong>Transcription factor target prediction using multiple short expression time series from Arabidopsis thaliana</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-454">PDF</a>
</li>
  <li>–2007–Lars Gidskehaug–BMC–<strong>A framework for significance analysis of gene expression data using dimension reduction methods</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-346">PDF</a>
</li>
  <li>–2007–Hyunsoo Kim–BMC–<strong>Extracting unrecognized gene relationships from the biomedical literature via matrix factorizations</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S9-S6">PDF</a>
</li>
  <li>–2007–David Talavera–BMC–<strong>A procedure for identifying homologous alternative splicing events</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-260">PDF</a>
</li>
  <li>–2007–Marc Strickert–BMC–<strong>Correlation-maximizing surrogate gene space for visual mining of gene expression patterns in developing barley endosperm tissue</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-165">PDF</a>
</li>
  <li>–2007–Tobias Wittkop–BMC–<strong>Large scale clustering of protein sequences with FORCE -A layout based heuristic for weighted cluster editing</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-396">PDF</a>
</li>
  <li>–2007–Susana Vinga–BMC–<strong>Local Renyi entropic profiles of DNA sequences</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-393">PDF</a>
</li>
  <li>–2007–Yanjun Qi–BMC–<strong>A mixture of feature experts approach for protein-protein interaction prediction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S10-S6">PDF</a>
</li>
  <li>–2007–Malik Yousef–BMC–<strong>Recursive Cluster Elimination (RCE) for classification and feature selection from gene expression data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-144">PDF</a>
</li>
  <li>–2007–Wasinee Rungsarityotin–BMC–<strong>Identifying protein complexes directly from high-throughput TAP data with Markov random fields</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-482">PDF</a>
</li>
  <li>–2007–Chris Thachuk–BMC–<strong>A replica exchange Monte Carlo algorithm for protein folding in the HP model</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-342">PDF</a>
</li>
  <li>–2007–Guangan Hu–BMC–<strong>Selection of long oligonucleotides for gene expression microarrays using weighted rank-sum strategy</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-350">PDF</a>
</li>
  <li>–2007–Sneh Lata–BMC–<strong>Analysis and prediction of antibacterial peptides</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-263">PDF</a>
</li>
  <li>–2007–Corinne Dahinden–BMC–<strong>Penalized likelihood for sparse contingency tables with an application to full-length cDNA libraries</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-476">PDF</a>
</li>
  <li>–2007–Kana Shimizu–BMC–<strong>Predicting mostly disordered proteins by using structure-unknown protein data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-78">PDF</a>
</li>
  <li>–2007–Modan K Das–BMC–<strong>A survey of DNA motif finding algorithms</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S7-S21">PDF</a>
</li>
  <li>–2007–Sören Sonnenburg–BMC–<strong>Accurate splice site prediction using support vector machines</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S10-S7">PDF</a>
</li>
  <li>–2007–Menaka Rajapakse–BMC–<strong>Predicting peptides binding to MHC class II molecules using multi-objective evolutionary algorithms</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-459">PDF</a>
</li>
  <li>–2007–Ryoko Morioka–BMC–<strong>Predicting state transitions in the transcriptome and metabolome using a linear dynamical system model</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-343">PDF</a>
</li>
  <li>–2007–Cornelia Caragea–BMC–<strong>Glycosylation site prediction using ensembles of Support Vector Machine classifiers</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-438">PDF</a>
</li>
  <li>–2007–Meng Wang–BMC–<strong>Context based mixture model for cell phase identification in automated fluorescence microscopy</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-32">PDF</a>
</li>
  <li>–2007–Andrea Splendiani–BMC–<strong>The Genopolis Microarray Database</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S1-S21">PDF</a>
</li>
  <li>–2007–Roger A Craig–BMC–<strong>Phylogenetic tree information aids supervised learning for predicting protein-protein interaction based on distance matrices</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-6">PDF</a>
</li>
  <li>–2007–Qiwen Dong–BMC–<strong>Exploiting residue-level and profile-level interface propensities for usage in binding sites prediction of proteins</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-147">PDF</a>
</li>
  <li>–2007–Gabriele A Trombetti–BMC–<strong>Data handling strategies for high throughput pyrosequencers</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S1-S22">PDF</a>
</li>
  <li>–2007–Florian Teichert–BMC–<strong>SABERTOOTH: protein structural alignment based on a vectorial structure representation</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-425">PDF</a>
</li>
  <li>–2007–Harmen J Bussemaker–BMC–<strong>Dissecting complex transcriptional responses using pathway-level scores based on prior information</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S6-S6">PDF</a>
</li>
  <li>–2007–Aki Vehtari–BMC–<strong>A novel Bayesian approach to quantify clinical variables and to determine their spectroscopic counterparts in 1H NMR metabonomic data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S2-S8">PDF</a>
</li>
  <li>–2007–Chrysi Konstantinou-Kirtay–BMC–<strong>Scoring functions and enrichment: a case study on Hsp90</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-27">PDF</a>
    <h2 id="2006">
<a class="anchor" href="#2006" aria-hidden="true"><span class="octicon octicon-link"></span></a>2006</h2>
  </li>
  <li>–2006–David J Reiss–BMC–<strong>Integrated biclustering of heterogeneous genome-wide datasets for the inference of global regulatory networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-280">PDF</a>
</li>
  <li>–2006–Yoshiya Matsubara–BMC–<strong>Parameter estimation for stiff equations of biosystems using radial basis function networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-230">PDF</a>
</li>
  <li>–2006–Maria Rodriguez-Fern–BMC–<strong>Novel metaheuristic for parameter estimation in nonlinear dynamic biological systems</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-483">PDF</a>
</li>
  <li>–2006–Bor-Sen Chen–BMC–<strong>On the attenuation and amplification of molecular noise in genetic regulatory networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-52">PDF</a>
</li>
  <li>–2006–Gunnar Rätsch–BMC–<strong>Learning Interpretable SVMs for Biological Sequence Classification</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S1-S9">PDF</a>
</li>
  <li>–2006–I Emrah Nikerel–BMC–<strong>A method for estimation of elasticities in metabolic networks using steady state and dynamic metabolomics data and linlog kinetics</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-540">PDF</a>
</li>
  <li>–2006–Adam A Margolin–BMC–<strong>ARACNE: An Algorithm for the Reconstruction of Gene Regulatory Networks in a Mammalian Cellular Context</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S1-S7">PDF</a>
</li>
  <li>–2006–Tsuyoshi Kato–BMC–<strong>Network-based de-noising improves prediction from microarray data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S1-S4">PDF</a>
</li>
  <li>–2006–Weichun Huang–BMC–<strong>Optimized mixed Markov models for motif identification</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-279">PDF</a>
</li>
  <li>–2006–Ethan G Cerami–BMC–<strong>cPath: open source software for collecting, storing, and querying biological pathways</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-497">PDF</a>
</li>
  <li>–2006–Jianghui Xiong–BMC–<strong>Genome wide prediction of protein function via a generic knowledge discovery approach based on evidence integration</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-268">PDF</a>
</li>
  <li>–2006–Sylvain Brohée–BMC–<strong>Evaluation of clustering algorithms for protein-protein interaction networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-488">PDF</a>
</li>
  <li>–2006–Kazuhito Shida–BMC–<strong>GibbsST: a Gibbs sampling method for motif discovery with enhanced resistance to local optima</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-486">PDF</a>
</li>
  <li>–2006–Chung-Yen Lin–BMC–<strong>Fly-DPI: database of protein interactomes for D. melanogaster in the approach of systems biology</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S5-S18">PDF</a>
</li>
  <li>–2006–Srinka Ghosh–BMC–<strong>Rank-statistics based enrichment-site prediction algorithm developed for chromatin immunoprecipitation on chip experiments</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-434">PDF</a>
</li>
  <li>–2006–Br–BMC–<strong>Spectral embedding finds meaningful (relevant) structure in image and microarray data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-74">PDF</a>
</li>
  <li>–2006–Jean-Marc Schwartz–BMC–<strong>Quantitative elementary mode analysis of metabolic pathways: the example of yeast glycolysis</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-186">PDF</a>
</li>
  <li>–2006–Rakesh Kaundal–BMC–<strong>Machine learning techniques in disease forecasting: a case study on rice blast prediction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-485">PDF</a>
</li>
  <li>–2006–Rainer König–BMC–<strong>Discovering functional gene expression patterns in the metabolic network of Escherichia coli with wavelets transforms</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-119">PDF</a>
</li>
  <li>–2006–Monica Chagoyen–BMC–<strong>Discovering semantic features in the literature: a foundation for building functional associations</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-41">PDF</a>
</li>
  <li>–2006–Ling Qin–BMC–<strong>An improved ant colony algorithm with diversified solutions based on the immune strategy</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S4-S3">PDF</a>
</li>
  <li>–2006–Alison A Motsinger–BMC–<strong>GPNN: Power studies and applications of a neural network method for detecting gene-gene interactions in studies of human disease</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-39">PDF</a>
</li>
  <li>–2006–Susmita Datta–BMC–<strong>Evaluation of clustering algorithms for gene expression data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S4-S17">PDF</a>
</li>
  <li>–2006–Chung-Tsai Su–BMC–<strong>Protein disorder prediction by condensed PSSM considering propensity for order or disorder</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-319">PDF</a>
</li>
  <li>–2006–Kenzie D MacIsaac–BMC–<strong>An improved map of conserved regulatory sites for Saccharomyces cerevisiae</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-113">PDF</a>
</li>
  <li>–2006–Hongbo Zhu–BMC–<strong>NOXclass: prediction of protein-protein interaction types</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-27">PDF</a>
</li>
  <li>–2006–Lawrence JK Wee–BMC–<strong>SVM-based prediction of caspase substrate cleavage sites</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S5-S14">PDF</a>
</li>
  <li>–2006–Ester Vilaprinyo–BMC–<strong>Use of physiological constraints to identify quantitative design principles for gene expression in yeast adaptation to heat shock</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-184">PDF</a>
</li>
  <li>–2006–AKMA Baten–BMC–<strong>Splice site identification using probabilistic parameters and SVM classification</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S5-S15">PDF</a>
</li>
  <li>–2006–Xiuzhen Huang–BMC–<strong>Maximum common subgraph: some upper bound and lower bound results</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S4-S6">PDF</a>
</li>
  <li>–2006–Yian A Chen–BMC–<strong>A multivariate prediction model for microarray cross-hybridization</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-101">PDF</a>
</li>
  <li>–2006–Bartek Wilczyński–BMC–<strong>Using local gene expression similarities to discover regulatory binding site modules</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-505">PDF</a>
</li>
  <li>–2006–Chaoyang Zhang–BMC–<strong>Parallelization of multicategory support vector machines (PMC-SVM) for classifying microarray data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S4-S15">PDF</a>
</li>
  <li>–2006–Kazuharu Arakawa–BMC–<strong>GEM System: automatic prototyping of cell-wide metabolic pathway models from genomes</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-168">PDF</a>
</li>
  <li>–2006–Mikael Bodén–BMC–<strong>Prediction of protein continuum secondary structure with probabilistic models based on NMR solved structures</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-68">PDF</a>
</li>
  <li>–2006–Stephen Winters-Hil–BMC–<strong>Nanopore Detector based analysis of single-molecule conformational kinetics and binding interactions</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S2-S21">PDF</a>
</li>
  <li>–2006–Alex–BMC–<strong>Genome BLAST distance phylogenies inferred from whole plastid and whole mitochondrion genome sequences</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-350">PDF</a>
</li>
  <li>–2006–Stefano Lise–BMC–<strong>Docking protein domains in contact space</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-310">PDF</a>
</li>
  <li>–2006–Natalia V Petrova–BMC–<strong>Prediction of catalytic residues using Support Vector Machine with selected protein sequence and structural properties</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-312">PDF</a>
</li>
  <li>–2006–Martin Hoffmann–BMC–<strong>Robust computational reconstitution – a new method for the comparative analysis of gene expression in tissues and isolated cell fractions</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-369">PDF</a>
</li>
  <li>–2006–Prabakaran Subramani–BMC–<strong>Feature selection using Haar wavelet power spectrum</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-432">PDF</a>
</li>
  <li>–2006–Youping Deng–BMC–<strong>Development of computations in bioscience and bioinformatics and its application: review of the Symposium of Computations in Bioinformatics and Bioscience (SCBB06)</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S4-S1">PDF</a>
</li>
  <li>–2006–Kang Peng–BMC–<strong>Length-dependent prediction of protein intrinsic disorder</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-208">PDF</a>
</li>
  <li>–2006–Yu Xue–BMC–<strong>PPSP: prediction of PK-specific phosphorylation site with Bayesian decision theory</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-163">PDF</a>
</li>
  <li>–2006–Pantelis G Bagos–BMC–<strong>Algorithms for incorporating prior topological information in HMMs: application to transmembrane proteins</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-189">PDF</a>
</li>
  <li>–2006–Maksim V Plikus–BMC–<strong>PubFocus: semantic MEDLINE/PubMed citations analytics through integration of controlled biomedical dictionaries and ranking algorithm</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-424">PDF</a>
</li>
  <li>–2006–Xian Wang–BMC–<strong>Missing value estimation for DNA microarray gene expression data by Support Vector Regression imputation and orthogonal coding scheme</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-32">PDF</a>
</li>
  <li>–2006–Jianjun Hu–BMC–<strong>EMD: an ensemble algorithm for discovering regulatory motifs in DNA sequences</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-342">PDF</a>
</li>
  <li>–2006–Teresa Haynes–BMC–<strong>A quantitative analysis of secondary RNA structure using domination based parameters on trees</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-108">PDF</a>
</li>
  <li>–2006–Manju R Mamtani–BMC–<strong>A simple method to combine multiple molecular biomarkers for dichotomous diagnostic classification</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-442">PDF</a>
</li>
  <li>–2006–Jonathan D Wren–BMC–<strong>Proceedings of the Third Annual Conference of the MidSouth Computational Biology and Bioinformatics Society</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S2-S1">PDF</a>
</li>
  <li>–2006–Denis C Bauer–BMC–<strong>STAR: predicting recombination sites from amino acid sequence</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-437">PDF</a>
</li>
  <li>–2006–Feng Li–BMC–<strong>Construction of a nasopharyngeal carcinoma 2D/MS repository with Open Source XML Database – Xindice</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-13">PDF</a>
</li>
  <li>–2006–Jesper Salomon–BMC–<strong>Predicting Class II MHC-Peptide binding: a kernel based approach using similarity scores</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-501">PDF</a>
</li>
  <li>–2006–Scott Montgomerie–BMC–<strong>Improving the accuracy of protein secondary structure prediction using structural alignment</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-301">PDF</a>
</li>
  <li>–2006–Jiangning Song–BMC–<strong>Predicting residue-wise contact orders in proteins by support vector regression</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-425">PDF</a>
</li>
  <li>–2006–Peter L Chang–BMC–<strong>Structure alignment based on coding of local geometric measures</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-346">PDF</a>
</li>
  <li>–2006–Stephan Waack–BMC–<strong>Score-based prediction of genomic islands in prokaryotic genomes using hidden Markov models</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-142">PDF</a>
</li>
  <li>–2006–Pufeng Du–BMC–<strong>Prediction of protein submitochondria locations by hybridizing pseudo-amino acid composition with various physicochemical features of segmented sequence</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-518">PDF</a>
</li>
  <li>–2006–Mehdi Pirooznia–BMC–<strong>SVM Classifier – a comprehensive java interface for support vector machine classification of microarray data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S4-S25">PDF</a>
</li>
  <li>–2006–Cheong Xin Chan–BMC–<strong>Detecting recombination in evolving nucleotide sequences</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-412">PDF</a>
</li>
  <li>–2006–Ingolf Sommer–BMC–<strong>Improving the quality of protein structure models by selecting from alignment alternatives</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-364">PDF</a>
    <h2 id="2005">
<a class="anchor" href="#2005" aria-hidden="true"><span class="octicon octicon-link"></span></a>2005</h2>
  </li>
  <li>–2005–Kapil G Gadkar–BMC–<strong>Iterative approach to model identification of biological networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-155">PDF</a>
</li>
  <li>–2005–Barbara Di Camillo–BMC–<strong>A quantization method based on threshold optimization for microarray short time series</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-S4-S11">PDF</a>
</li>
  <li>–2005–Dongxiao Zhu–BMC–<strong>Structural comparison of metabolic networks in selected single cell organisms</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-8">PDF</a>
</li>
  <li>–2005–Robert G Beiko–BMC–<strong>GANN: Genetic algorithm neural networks for the detection of conserved combinations of features in DNA</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-36">PDF</a>
</li>
  <li>–2005–AM Cohen–BMC–<strong>Using co-occurrence network structure to extract synonymous gene and protein names from MEDLINE abstracts</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-103">PDF</a>
</li>
  <li>–2005–Kiran Raosaheb Patil–BMC–<strong>Evolutionary programming as a platform for in silico metabolic engineering</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-308">PDF</a>
</li>
  <li>–2005–Xiaoxing Liu–BMC–<strong>An Entropy-based gene selection method for cancer classification using microarray data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-76">PDF</a>
</li>
  <li>–2005–Markey C McNutt–BMC–<strong>Human promoter genomic composition demonstrates non-random groupings that reflect general cellular function</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-259">PDF</a>
</li>
  <li>–2005–Nicola Zamboni–BMC–<strong>FiatFlux – a software for metabolic flux analysis from 13C-glucose experiments</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-209">PDF</a>
</li>
  <li>–2005–Chin-Jen Ku–BMC–<strong>The distance-profile representation and its application to detection of distantly related protein families</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-282">PDF</a>
</li>
  <li>–2005–Blaise TF Alako–BMC–<strong>CoPub Mapper: mining MEDLINE based on search term co-publication</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-51">PDF</a>
</li>
  <li>–2005–Michael Maurer–BMC–<strong>MARS: Microarray analysis, retrieval, and storage system</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-101">PDF</a>
</li>
  <li>–2005–Yunlong Liu–BMC–<strong>Principal component analysis for predicting transcription-factor binding motifs from array-derived data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-276">PDF</a>
</li>
  <li>–2005–Claudio Lottaz–BMC–<strong>stam – a Bioconductor compliant R package for structured analysis of microarray data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-211">PDF</a>
</li>
  <li>–2005–Giuliano Armano–BMC–<strong>A Hybrid Genetic-Neural System for Predicting Protein Secondary Structure</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-S4-S3">PDF</a>
</li>
  <li>–2005–Piero Fariselli–BMC–<strong>A new decoding algorithm for hidden Markov models improves the prediction of the topology of all-beta membrane proteins</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-S4-S12">PDF</a>
</li>
  <li>–2005–Jannick Dyrløv Bendtsen–BMC–<strong>Prediction of twin-arginine signal peptides</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-167">PDF</a>
    <h2 id="2004">
<a class="anchor" href="#2004" aria-hidden="true"><span class="octicon octicon-link"></span></a>2004</h2>
  </li>
  <li>–2004–Hao Chen–BMC–<strong>Content-rich biological network constructed by mining PubMed abstracts</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-147">PDF</a>
</li>
  <li>–2004–Christine Brun–BMC–<strong>Clustering proteins from interaction networks for the prediction of cellular functions</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-95">PDF</a>
</li>
  <li>–2004–David Adalsteinsson–BMC–<strong>Biochemical Network Stochastic Simulator (BioNetS): software for stochastic modeling of biochemical networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-24">PDF</a>
</li>
  <li>–2004–Bahrad A Sokhansanj–BMC–<strong>Linear fuzzy gene network models obtained from microarray data by exhaustive search</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-108">PDF</a>
</li>
  <li>–2004–Peter Meinicke–BMC–<strong>Oligo kernels for datamining on biological sequences: a case study on prokaryotic translation initiation sites</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-169">PDF</a>
</li>
  <li>–2004–John A Berger–BMC–<strong>Optimized LOWESS normalization parameter selection for DNA microarray data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-194">PDF</a>
</li>
  <li>–2004–Yasuyuki Tomita–BMC–<strong>Artificial neural network approach for selection of susceptible single nucleotide polymorphisms and construction of prediction model on childhood allergic asthma</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-120">PDF</a>
</li>
  <li>–2004–William H Majoros–BMC–<strong>An empirical analysis of training protocols for probabilistic gene finders</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-206">PDF</a>
</li>
  <li>–2004–Kayvan Najarian–BMC–<strong>A novel Mixture Model Method for identification of differentially expressed genes from DNA microarray data</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-201">PDF</a>
</li>
  <li>–2004–Yvan Saeys–BMC–<strong>Feature selection for splice site prediction: A new method using EDA-based feature ranking</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-64">PDF</a>
</li>
  <li>–2004–Gustavo Camps-Valls–BMC–<strong>Profiled support vector machines for antisense oligonucleotide efficacy prediction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-135">PDF</a>
</li>
  <li>–2004–Arunachalam Vinayagam–BMC–<strong>Applying Support Vector Machines for Gene ontology based gene function prediction</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-116">PDF</a>
</li>
  <li>–2004–Qinghua Cui–BMC–<strong>Esub8: A novel tool to predict protein subcellular localizations in eukaryotic organisms</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-66">PDF</a>
</li>
  <li>–2004–Drew Lett–BMC–<strong>Interaction profile-based protein classification of death domain</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-75">PDF</a>
</li>
  <li>–2004–Remko de Knikker–BMC–<strong>A web services choreography scenario for interoperating bioinformatics applications</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-25">PDF</a>
</li>
  <li>–2004–Rainer Merkl–BMC–<strong>SIGI: score-based identification of genomic islands</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-22">PDF</a>
    <h2 id="2003">
<a class="anchor" href="#2003" aria-hidden="true"><span class="octicon octicon-link"></span></a>2003</h2>
  </li>
  <li>–2003–Gary D Bader–BMC–<strong>An automated method for finding molecular complexes in large protein interaction networks</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-4-2">PDF</a>
</li>
  <li>–2003–Marylyn D Ritchie–BMC–<strong>Optimizationof neural network architecture using genetic programming improvesdetection and modeling of gene-gene interactions in studies of humandiseases</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-4-28">PDF</a>
</li>
  <li>–2003–Robert M Hubley–BMC–<strong>Evolutionary algorithms for the selection of single nucleotide polymorphisms</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-4-30">PDF</a>
</li>
  <li>–2003–Samuel Kaski–BMC–<strong>Trustworthiness and metrics in visualizing similarity of gene expression</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-4-48">PDF</a>
    <h2 id="2002">
<a class="anchor" href="#2002" aria-hidden="true"><span class="octicon octicon-link"></span></a>2002</h2>
  </li>
  <li>–2002–Lan Ma–BMC–<strong>Quantifying robustness of biochemical network models</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-3-38">PDF</a>
</li>
  <li>–2002–Pierre Dönnes–BMC–<strong>Prediction of MHC class I binding peptides, using SVMHC</strong>– <a href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-3-25">PDF</a>
</li>
</ul>

  </div><a class="u-url" href="/translations/deeplearning/2020/09/26/papers-BMC-optimization-for-biological-networks.html" hidden></a>
</article>
      </div>
    </main><footer class="site-footer h-card">
  <data class="u-url" href="/"></data>

  <div class="wrapper">

    <div class="footer-col-wrapper">
      <div class="footer-col">
        <p class="feed-subscribe">
          <a href="/feed.xml">
            <svg class="svg-icon orange">
              <use xlink:href="/assets/minima-social-icons.svg#rss"></use>
            </svg><span>Subscribe</span>
          </a>
        </p>
      </div>
      <div class="footer-col">
        <p>未来秃头协会成员</p>
      </div>
    </div>

    <div class="social-links"><ul class="social-media-list"><li><a rel="me" href="https://github.com/BjChacha" title="BjChacha"><svg class="svg-icon grey"><use xlink:href="/assets/minima-social-icons.svg#github"></use></svg></a></li></ul>
</div>

  </div>

</footer>
</body>

</html>
