---
title: "[Paper List]Optimization for Biological Networks"
description: "Only BMC Bioinformatics"
toc: true
hide: false
layout: post
categories: [translations, deeplearning]
author: BjChacha
---

# BMC Bioinformatics

关键字：optimization for biological networks

    ## 2020
    * --2020--Jane Hung--BMC--**Keras R-CNN: library for cell detection in biological images using deep neural networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03635-x)
    * --2020--Yuanke Zhong--BMC--**Twadn: an efficient alignment algorithm based on time warping for pairwise dynamic networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03672-6)
    * --2020--Ricardo Andrade--BMC--**MOMO - multi-objective metabolic mixed integer optimization: application to yeast strain engineering**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3377-1)
    * --2020--Xihui Lin--BMC--**Optimization and expansion of non-negative matrix factorization**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3312-5)
    * --2020--Yan Li--BMC--**Bayesian differential analysis of gene regulatory networks exploiting genetic perturbations**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3314-3)
    * --2020--Bruna Piereck--BMC--**LAITOR4HPC: A text mining pipeline based on HPC for building interaction networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03620-4)
    * --2020--Michael Sekula--BMC--**A sparse Bayesian factor model for the construction of gene co-expression networks from single-cell RNA sequencing count data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03707-y)
    * --2020--Tania Timmermann--BMC--**Reconstruction of a gene regulatory network of the induced systemic resistance defense response in Arabidopsis using boolean networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3472-3)
    * --2020--Luisa Cutillo--BMC--**OscoNet: inferring oscillatory gene networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03561-y)
    * --2020--Sudipta Acharya--BMC--**A consensus multi-view multi-objective gene selection approach for improved sample classification**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03681-5)
    * --2020--Bihai Zhao--BMC--**NPF:network propagation for protein function prediction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03663-7)
    * --2020--Chunyan Tang--BMC--**Drug-target interactions prediction using marginalized denoising model on heterogeneous networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03662-8)
    * --2020--Juncheng Guo--BMC--**An embedded gene selection method using knockoffs optimizing neural network**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03717-w)
    * --2020--Ana Cernea--BMC--**Robust pathway sampling in phenotype prediction. Application to triple negative breast cancer**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3356-6)
    * --2020--Mojtaba Tefagh--BMC--**SWIFTCORE: a tool for the context-specific reconstruction of genome-scale metabolic networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3440-y)
    * --2020--Anna M. Nia--BMC--**Efficient identification of multiple pathways: RNA-Seq analysis of livers from 56Fe ion irradiated mice**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3446-5)
    * --2020--Thevaa Ch--BMC--**Lag penalized weighted correlation for time series clustering**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3324-1)
    * --2020--Robert A. Dromms--BMC--**LK-DFBA: a linear programming-based modeling strategy for capturing dynamics and metabolite-dependent regulation in metabolism**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3422-0)
    * --2020--Yosef Masoudi-Sobhanzadeh--BMC--**Synthetic repurposing of drugs against hypertension: a datamining method based on association rules and a novel discrete algorithm**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03644-w)
    * --2020--Lvxing Zhu--BMC--**Biomedical event extraction with a novel combination strategy based on hybrid deep neural networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3376-2)
    * --2020--Wei Du--BMC--**CapsNet-SSP: multilane capsule network for predicting human saliva-secretory proteins**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03579-2)
    * --2020--Cheng Ye--BMC--**The corrected gene proximity map for analyzing the 3D genome organization using Hi-C data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03545-y)
    * --2020--Zhijing Li--BMC--**Bio-semantic relation extraction with attention-based external knowledge reinforcement**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3540-8)
    * --2020--Hynek Bachratý--BMC--**Applications of machine learning for simulations of red blood cells in microfluidic devices**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3357-5)
    * --2020--Isabella Mendolia--BMC--**Convolutional architectures for virtual screening**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03645-9)
    * --2020--Jan Klosa--BMC--**Seagull: lasso, group lasso and sparse-group lasso regularization for linear regression models via proximal gradient descent**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03725-w)
    * --2020--Jian Wang--BMC--**Biomedical document triage using a hierarchical attention-based capsule network**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03673-5)
    * --2020--Hasindu Gamaarachchi--BMC--**GPU accelerated adaptive banded event alignment for rapid comparative nanopore signal analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03697-x)
    * --2020--Melanie Fachet--BMC--**Reconstruction and analysis of a carbon-core metabolic network for Dunaliella salina**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3325-0)
    * --2020--Yahya Bokhari--BMC--**QuaDMutNetEx: a method for detecting cancer driver genes with low mutation frequency**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3449-2)
    * --2020--Fang Yang--BMC--**Graph-based prediction of Protein-protein interactions with attributed signed graph embedding**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03646-8)
    * --2020--Paolo Castagno--BMC--**A computational framework for modeling and studying pertussis epidemiology and vaccination**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03648-6)
    * --2020--Mikołaj Rybiński--BMC--**TopoFilter: a MATLAB package for mechanistic model identification in systems biology**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3343-y)
    * --2020--Marta B. Lopes--BMC--**Tracking intratumoral heterogeneity in glioblastoma via regularized classification of single-cell RNA-Seq data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3390-4)
    * --2020--Tiziana Castrignanò--BMC--**ELIXIR-IT HPC@CINECA: high performance computing resources for the bioinformatics community**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03565-8)
    * --2020--Eun Jeong Min--BMC--**Sparse multiple co-Inertia analysis with application to integrative analysis of multi -Omics data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3455-4)
    * --2020--Amina Lemsara--BMC--**PathME: pathway based multi-modal sparse autoencoders for clustering of patient-level multi-omics data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3465-2)
    * --2020--Ali Foroughi pour--BMC--**High dimensional model representation of log-likelihood ratio: binary classification with expression data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3486-x)
    * --2020--Pavlos Stephanos Bekiaris--BMC--**Automatic construction of metabolic models with enzyme constraints**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3329-9)
    * --2020--Henri Riihimäki--BMC--**A topological data analysis based classification method for multiple measurements**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03659-3)
    * --2020--Xin Guan--BMC--**Dynamic incorporation of prior knowledge from multiple domains in biomarker discovery**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3344-x)
    * --2020--Hesham ElAbd--BMC--**Amino acid encoding for deep learning applications**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03546-x)
    * --2020--Zhen Gao--BMC--**Graph regularized L2,1-nonnegative matrix factorization for miRNA-disease association prediction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3409-x)
    * --2020--Alessio Mancini--BMC--**Machine learning models predicting multidrug resistant urinary tract infections using “DsaaS”**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03566-7)
    * --2020--Aaron M. Smith--BMC--**Standard machine learning approaches outperform deep representation learning on phenotype prediction from transcriptomics data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3427-8)
    * --2020--Joungmin Choi--BMC--**methCancer-gen: a DNA methylome dataset generator for user-specified cancer type based on conditional variational autoencoder**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3516-8)
    * --2020--Joshua J. Levy--BMC--**MethylNet: an automated and modular deep learning approach for DNA methylation analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3443-8)
    * --2020--Kitsuchart Pasupa--BMC--**Discovery of significant porcine SNPs for swine breed identification by a hybrid of information gain, genetic algorithm, and frequency feature selection technique**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3471-4)
    * --2020--Yuliang Pan--BMC--**Computationally identifying hot spots in protein-DNA binding interfaces using an ensemble approach**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03675-3)
    * --2020--Rita T. Sousa--BMC--**Evolving knowledge graph similarity for supervised learning in complex biomedical domains**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3296-1)
    * --2020--Muaaz G. Awan--BMC--**ADEPT: a domain independent sequence alignment strategy for gpu architectures**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03720-1)
    * --2020--Attila Csala--BMC--**Multiset sparse partial least squares path modeling for high dimensional omics data analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3286-3)
    * --2020--Prasannavenkatesh Durai--BMC--**Evolutionary chemical binding similarity approach integrated with 3D-QSAR method for effective virtual screening**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03643-x)
    * --2020--Xingheng Yu--BMC--**Heterogeneous multiple kernel learning for breast cancer outcome evaluation**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3483-0)
    * --2020--Katja Ovchinnikova--BMC--**OffsampleAI: artificial intelligence approach to recognize off-sample mass spectrometry images**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3425-x)
    * --2020--Niraj Thapa--BMC--**DeepSuccinylSite: a deep learning based approach for protein succinylation site prediction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3342-z)
    * --2020--Lei Du--BMC--**Subcellular location prediction of apoptosis proteins using two novel feature extraction methods based on evolutionary information and LDA**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3539-1)
    * --2020--Julie Hurel--BMC--**DUGMO: tool for the detection of unknown genetically modified organisms with high-throughput sequencing data for pure bacterial samples**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03611-5)
    * --2020--Cheng Yan--BMC--**PESM: predicting the essentiality of miRNAs based on gradient boosting machines and sequences**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3426-9)
    * --2020--Lei Li--BMC--**Multi-task learning sparse group lasso: a method for quantifying antigenicity of influenza A(H1N1) virus using mutations and variations in glycosylation of Hemagglutinin**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3527-5)
    * --2020--Samuel E. Barnett--BMC--**Simulating metagenomic stable isotope probing datasets with MetaSIPSim**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3372-6)
    * --2020--Matthew D. Koslovsky--BMC--**MicroBVS: Dirichlet-tree multinomial regression models with Bayesian variable selection - an R package**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03640-0)
    * --2020--Mattia Chiesa--BMC--**GARS: Genetic Algorithm for the identification of a Robust Subset of features in high-dimensional datasets**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3400-6)
    * --2020--Lu Cao--BMC--**Automated image analysis system for studying cardiotoxicity in human pluripotent stem cell-Derived cardiomyocytes**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3466-1)
    * --2020--Behrend Heeren--BMC--**Statistical shape analysis of tap roots: a methodological case study on laser scanned sugar beets**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-03654-8)
    * --2020--Qiaoyue Liu--BMC--**Deep learning improves the ability of sgRNA off-target propensity prediction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3395-z)
    * --2020--Matthew Merski--BMC--**Self-analysis of repeat proteins reveals evolutionarily conserved patterns**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3493-y)
    * --2020--Feixiao Long--BMC--**Microscopy cell nuclei segmentation with enhanced U-Net**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3332-1)
    * --2020--Audrey Hulot--BMC--**Fast tree aggregation for consensus hierarchical clustering**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3453-6)
    * --2020--Sinead E Morris--BMC--**ushr: Understanding suppression of HIV in R**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-020-3389-x)
    ## 2019
    * --2019--Wei Zheng--BMC--**Control of multilayer biological networks and applied to target identification of complex diseases**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2841-2)
    * --2019--Sara Rahiminejad--BMC--**Topological and functional comparison of community detection algorithms in biological networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2746-0)
    * --2019--Hsiang-Yun Wu--BMC--**Metabopolis: scalable network layout for biological pathway diagrams in urban map style**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2779-4)
    * --2019--Chen Wang--BMC--**Efficient proximal gradient algorithm for inference of differential gene networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2749-x)
    * --2019--Florian Martin--BMC--**NPA: an R package for computing network perturbation amplitudes using gene expression data and two-layer networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3016-x)
    * --2019--Tom W. Hiscock--BMC--**Adapting machine-learning algorithms to design gene circuits**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2788-3)
    * --2019--Kipoong Kim--BMC--**Incorporating genetic networks into case-control association studies with high-dimensional DNA methylation data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3040-x)
    * --2019--Zolzaya Dashdorj--BMC--**An application of convolutional neural networks with salient features for relation classification**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2808-3)
    * --2019--Savvas Kinalis--BMC--**Deconvolution of autoencoders to learn biological regulatory modules from single cell mRNA sequencing data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2952-9)
    * --2019--Yifeng Cui--BMC--**Predicting protein-ligand binding residues with deep convolutional neural networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2672-1)
    * --2019--Yongtian Wang--BMC--**LncDisAP: a computation model for LncRNA-disease association prediction based on multiple biological datasets**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3081-1)
    * --2019--Tongle Ma--BMC--**Periodicity and dosage optimization of an RNAi model in eukaryotes cells**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2925-z)
    * --2019--Eun-Youn Kim--BMC--**Identification of critical connectors in the directed reaction-centric graphs of microbial metabolic networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2897-z)
    * --2019--Fei Han--BMC--**A hybrid gene selection method based on gene scoring strategy and improved particle swarm optimization**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2773-x)
    * --2019--Han Yu--BMC--**Integration of probabilistic regulatory networks into constraint-based models of metabolism with applications to Alzheimer’s disease**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2872-8)
    * --2019--Xiujuan Lei--BMC--**Protein complex detection based on flower pollination mechanism in multi-relation reconstructed dynamic protein networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2649-0)
    * --2019--Denise Thiel--BMC--**Identifying lncRNA-mediated regulatory modules via ChIA-PET network analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2900-8)
    * --2019--Min Luo--BMC--**Screening drug target combinations in disease-related molecular networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2730-8)
    * --2019--Ryohei Eguchi--BMC--**Classification of alkaloids according to the starting substances of their biosynthetic pathways using graph convolutional neural networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2963-6)
    * --2019--Mahroo Moridi--BMC--**The assessment of efficient representation of drug features using deep learning for drug repositioning**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3165-y)
    * --2019--Joseph Luttrell IV--BMC--**Predicting protein residue-residue contacts using random forests and deep networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2627-6)
    * --2019--Jake Alan Pitt--BMC--**Parameter estimation in models of biological oscillators: an automated regularised estimation approach**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2630-y)
    * --2019--Byung-Jun Yoon--BMC--**Selected research articles from the 2018 International Workshop on Computational Network Biology: Modeling, Analysis, and Control (CNB-MAC)**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2830-5)
    * --2019--Binh P. Nguyen--BMC--**iProDNA-CapsNet: identifying protein-DNA binding residues using capsule neural networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3295-2)
    * --2019--Zheng Gao--BMC--**edge2vec: Representation learning using edge semantics for biomedical knowledge discovery**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2914-2)
    * --2019--Bihai Zhao--BMC--**An iteration method for identifying yeast essential proteins from heterogeneous network**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2930-2)
    * --2019--Miriam Bern--BMC--**Network-based prediction of polygenic disease genes involved in cell motility**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2834-1)
    * --2019--Ali Navid--BMC--**System-level analysis of metabolic trade-offs during anaerobic photoheterotrophic growth in Rhodopseudomonas palustris**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2844-z)
    * --2019--Weizhong Lu--BMC--**Predicting RNA secondary structure via adaptive deep recurrent neural networks with energy-based filter**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3258-7)
    * --2019--Qiu Xiao--BMC--**CeModule: an integrative framework for discovering regulatory patterns from genomic data in cancer**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2654-3)
    * --2019--Huan Jin--BMC--**Moiety modeling framework for deriving moiety abundances from mass spectrometry measured isotopologues**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3096-7)
    * --2019--Da Zhang--BMC--**Multimodal deep representation learning for protein interaction identification and protein family classification**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3084-y)
    * --2019--Yuanjie Liu--BMC--**A general-purpose signal processing algorithm for biological profiles using only first-order derivative information**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3188-4)
    * --2019--Chieh Lo--BMC--**MetaNN: accurate classification of host phenotypes from metagenomic data using neural networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2833-2)
    * --2019--Lifan Liang--BMC--**Integrating data and knowledge to identify functional modules of genes: a multilayer approach**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2800-y)
    * --2019--Yahui Long--BMC--**WMGHMDA: a novel weighted meta-graph-based model for predicting human microbe-disease association on heterogeneous information network**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3066-0)
    * --2019--Felipe Kenji Nakano--BMC--**Machine learning for discovering missing or wrong protein function annotations**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3060-6)
    * --2019--Thomas A. Geddes--BMC--**Autoencoder-based cluster ensembles for single-cell RNA-seq data analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3179-5)
    * --2019--Chun-Mei Feng--BMC--**PCA via joint graph Laplacian and sparse constraint: Identification of differentially expressed genes and sample clustering on gene expression data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3229-z)
    * --2019--Jing Xu--BMC--**A hierarchical integration deep flexible neural forest framework for cancer subtype classification by integrating multi-omics data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3116-7)
    * --2019--Trevor Cickovski--BMC--**MATria: a unified centrality algorithm**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2820-7)
    * --2019--Hokuto Hirano--BMC--**Difficulty in inferring microbial community structure based on co-occurrence network approaches**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2915-1)
    * --2019----BMC--**Third international conference “Bioinformatics: from Algorithms to Applications” (BiATA 2019)**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3122-9)
    * --2019--Jiesi Luo--BMC--**Prediction of activity and specificity of CRISPR-Cpf1 using convolutional deep learning neural networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2939-6)
    * --2019--Hongda Bu--BMC--**DEEPSEN: a convolutional neural network based method for super-enhancer prediction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3180-z)
    * --2019--Jol--BMC--**A network-based approach for identifying suitable biomarkers for oral immunotherapy of food allergy**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2802-9)
    * --2019--Pascal Dolejsch--BMC--**Extensions of ℓ1 regularization increase detection specificity for cell-type specific parameters in dynamic models**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2976-1)
    * --2019--Fortunato Bianconi--BMC--**CRA toolbox: software package for conditional robustness analysis of cancer systems biology models in MATLAB**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2933-z)
    * --2019--Karsten Wüllems--BMC--**Detection and visualization of communities in mass spectrometry imaging data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2890-6)
    * --2019--Tongxin Wang--BMC--**Generalized gene co-expression analysis via subspace clustering using low-rank representation**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2733-5)
    * --2019--Ilaria Granata--BMC--**Integration of transcriptomic data in a genome-scale metabolic model to investigate the link between obesity and breast cancer**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2685-9)
    * --2019--Jie Zhao--BMC--**Detecting overlapping protein complexes in weighted PPI network based on overlay network chain in quotient space**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3256-9)
    * --2019--Vítor Vieira--BMC--**Comparison of pathway analysis and constraint-based methods for cell factory design**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2934-y)
    * --2019--Nikolay Martyushenko--BMC--**ModelExplorer - software for visual inspection and inconsistency correction of genome-scale metabolic reconstructions**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2615-x)
    * --2019--Leanne S. Whitmore--BMC--**RetSynth: determining all optimal and sub-optimal synthetic pathways that facilitate synthesis of target compounds in chassis organisms**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3025-9)
    * --2019--Marta B. Lopes--BMC--**Twiner: correlation-based regularization for identifying common cancer gene signatures**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2937-8)
    * --2019--Wei Li Wang--BMC--**Robustness of signal detection in cryo-electron microscopy via a bi-objective-function approach**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2714-8)
    * --2019--Zhen Cui--BMC--**The computational prediction of drug-disease interactions using the dual-network L2,1-CMF method**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2575-6)
    * --2019--Marie Postel--BMC--**A multiscale mathematical model of cell dynamics during neurogenesis in the mouse cerebral cortex**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3018-8)
    * --2019--Yang Guo--BMC--**Epi-GTBN: an approach of epistasis mining based on genetic Tabu algorithm and Bayesian network**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3022-z)
    * --2019--Xiaochuan Wang--BMC--**SIMLIN: a bioinformatics tool for prediction of S-sulphenylation in the human proteome based on multi-stage ensemble-learning models**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3178-6)
    * --2019--Dan Liu--BMC--**Predicting virus-host association by Kernelized logistic matrix factorization and similarity network fusion**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3082-0)
    * --2019--Martin Palazzo--BMC--**A pan-cancer somatic mutation embedding using autoencoders**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3298-z)
    * --2019--Qingxin Xiao--BMC--**Occurrence prediction of pests and diseases in cotton on the basis of weather factors by long short term memory network**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3262-y)
    * --2019--Roman Hornung--BMC--**Block Forests: random forests for blocks of clinical and omics covariate data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2942-y)
    * --2019--Chih-Hao Fang--BMC--**AIKYATAN: mapping distal regulatory elements using convolutional learning on GPU**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3049-1)
    * --2019--Tobias B. Alter--BMC--**Determination of growth-coupling strategies and their underlying principles**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2946-7)
    * --2019--Stefan Hammer--BMC--**Fixed-parameter tractable sampling for RNA design with multiple target structures**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2784-7)
    * --2019--Le--BMC--**Parametric and non-parametric gradient matching for network inference: a comparison**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2590-7)
    * --2019--Ying-Lian Gao--BMC--**NPCMF: Nearest Profile-based Collaborative Matrix Factorization method for predicting miRNA-disease associations**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2956-5)
    * --2019--Ruohan Wang--BMC--**SpliceFinder: ab initio prediction of splice sites using convolutional neural network**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3306-3)
    * --2019--Yongtian Wang--BMC--**Prioritizing candidate diseases-related metabolites based on literature and functional similarity**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3127-4)
    * --2019--Tianyi Zhao--BMC--**Identifying Alzheimer’s disease-related proteins by LRRGD**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3124-7)
    * --2019--Michael Mayers--BMC--**Time-resolved evaluation of compound repositioning predictions on a text-mined knowledge network**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3297-0)
    * --2019--Tomas Vicar--BMC--**Cell segmentation methods for label-free contrast microscopy: review and comprehensive comparison**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2880-8)
    * --2019--Yong Wang--BMC--**Systems biology intertwines with single cell and AI**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2731-7)
    * --2019--Stefano Teso--BMC--**Combining learning and constraints for genome-wide protein annotation**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2875-5)
    * --2019--Najaf A. Shah--BMC--**Variable cellular decision-making behavior in a constant synthetic network topology**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2866-6)
    * --2019--Kevin Michael Boehm--BMC--**Predicting peptide presentation by major histocompatibility complex class I: an improved machine learning approach to the immunopeptidome**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2561-z)
    * --2019--Ya-Wei Niu--BMC--**Integrating random walk and binary regression to identify novel miRNA-disease association**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2640-9)
    * --2019--Nguyen Quoc Khanh Le--BMC--**ET-GRU: using multi-layer gated recurrent units to identify electron transport proteins**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2972-5)
    * --2019--Ying Xiong--BMC--**An efficient gene selection method for microarray data based on LASSO and BPSO**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3228-0)
    * --2019--Chuang Li--BMC--**MCtandem: an efficient tool for large-scale peptide identification on many integrated core (MIC) architecture**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2980-5)
    * --2019--Thomas Gatter--BMC--**Ryūtō: network-flow based transcriptome reconstruction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2786-5)
    * --2019--Aditya Sai--BMC--**Exploring the information transmission properties of noise-induced dynamics: application to glioma differentiation**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2970-7)
    * --2019--Amarin Jettakul--BMC--**Relation extraction between bacteria and biotopes from biomedical texts with attention mechanisms and domain-specific contextual representations**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3217-3)
    * --2019--Rudi Alberts--BMC--**ZDOG: zooming in on dominating genes with mutations in cancer pathways**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3326-z)
    * --2019--Luis Torada--BMC--**ImaGene: a convolutional neural network to quantify natural selection from genomic data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2927-x)
    * --2019--Marina Esteban-Medina--BMC--**Exploring the druggable space around the Fanconi anemia pathway using machine learning and mechanistic models**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2969-0)
    * --2019--Ye Wang--BMC--**Semi-supervised prediction of protein interaction sites from unlabeled sample information**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3274-7)
    * --2019--Thorsten Will--BMC--**Differential analysis of combinatorial protein complexes with CompleXChange**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2852-z)
    * --2019--Hyang-Mi Lee--BMC--**Computational determination of hERG-related cardiotoxicity of drug candidates**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2814-5)
    * --2019--Rahul Shaw--BMC--**A mass and charge balanced metabolic model of Setaria viridis revealed mechanisms of proton balancing in C4 plants**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2941-z)
    * --2019--Yosef Masoudi-Sobhanzadeh--BMC--**FeatureSelect: a software for feature selection based on machine learning approaches**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2754-0)
    * --2019--Ine Melckenbeeck--BMC--**Optimising orbit counting of arbitrary order by equation selection**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2483-9)
    * --2019--Yiran Zhou--BMC--**NmSEER V2.0: a prediction tool for 2′-O-methylation sites based on random forest and multi-encoding combination**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3265-8)
    * --2019--Alex--BMC--**Walking pathways with positive feedback loops reveal DNA methylation biomarkers of colorectal cancer**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2687-7)
    * --2019--Heinrich Roder--BMC--**Robust identification of molecular phenotypes using semi-supervised learning**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2885-3)
    * --2019--Menghan Wang--BMC--**A probabilistic method for leveraging functional annotations to enhance estimation of the temporal order of pathway mutations during carcinogenesis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3218-2)
    * --2019--Xinxing Yang--BMC--**Additional Neural Matrix Factorization model for computational drug repositioning**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2983-2)
    * --2019--Jin Yao--BMC--**Predicting clinically promising therapeutic hypotheses using tensor factorization**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2664-1)
    * --2019--Tianjun Li--BMC--**Quality control of imbalanced mass spectra from isotopic labeling experiments**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3170-1)
    * --2019--Alex--BMC--**Modelling TERT regulation across 19 different cancer types based on the MIPRIP 2.0 gene regulatory network approach**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3323-2)
    * --2019--Debangana Chakravorty--BMC--**In silico modeling of phosphorylation dependent and independent c-Myc degradation**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2846-x)
    * --2019--Yunyun Dong--BMC--**MLW-gcForest: a multi-weighted gcForest model towards the staging of lung adenocarcinoma based on multi-modal genetic data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3172-z)
    * --2019--Xusheng Li--BMC--**A hybrid deep learning framework for bacterial named entity recognition with domain features**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3071-3)
    * --2019--Lei Cai--BMC--**DeepSV: accurate calling of genomic deletions from high-throughput sequencing data using deep convolutional neural network**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3299-y)
    * --2019--Hiroki Konishi--BMC--**Capturing the differences between humoral immunity in the normal and tumor environments from repertoire-seq of B-cell receptors using supervised machine learning**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2853-y)
    * --2019--Christian Carsten Sachs--BMC--**mycelyso – high-throughput analysis of Streptomyces mycelium live cell imaging data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3004-1)
    * --2019--Yoann Dufresne--BMC--**SLIM: a flexible web application for the reproducible processing of environmental DNA metabarcoding data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2663-2)
    * --2019--Olalekan Agbolade--BMC--**3-Dimensional facial expression recognition in human using multi-points warping**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3153-2)
    * --2019--Marco Seel--BMC--**Image-based classification of plant genus and family for trained and untrained plant species**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2474-x)
    * --2019--Gregory M. Parkes--BMC--**Uncovering extensive post-translation regulation during human cell cycle progression by integrative multi-’omics analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3150-5)
    * --2019--Arezo Torang--BMC--**An elastic-net logistic regression approach to generate classifiers and gene signatures for types of immune cells and T helper cell subsets**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2994-z)
    * --2019--Zhen Cui--BMC--**L2,1-GRMF: an improved graph regularized matrix factorization method to predict drug-target interactions**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2768-7)
    * --2019--Francesco Pappalardo--BMC--**Toward computational modelling on immune system function**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3239-x)
    * --2019--Minghan Chen--BMC--**Analysis and remedy of negativity problem in hybrid stochastic simulation algorithm and its application**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2836-z)
    * --2019--Qiang Yan--BMC--**Metabolic characterization of the chitinolytic bacterium Serratia marcescens using a genome-scale metabolic model**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2826-1)
    * --2019--Paul Müller--BMC--**nanite: using machine learning to assess the quality of atomic force microscopy-enabled nano-indentation data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3010-3)
    * --2019--Haohan Wang--BMC--**Deep mixed model for marginal epistasis detection and population stratification correction in genome-wide association studies**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3300-9)
    * --2019--N. Özlem ÖZCAN ŞİMŞEK--BMC--**Statistical representation models for mutation information within genomic data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2868-4)
    * --2019--Damiano Verda--BMC--**Analyzing gene expression data for pediatric and adult cancer diagnosis using logic learning machine and standard supervised methods**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2953-8)
    * --2019--Yi Zheng--BMC--**Inverse similarity and reliable negative samples for drug side-effect prediction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2563-x)
    * --2019--Zhen Cui--BMC--**RCMF: a robust collaborative matrix factorization method to predict miRNA-disease associations**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3260-0)
    * --2019--Licheng Wang--BMC--**DDmap: a MATLAB package for the double digest problem using multiple genetic operators**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2862-x)
    * --2019--Beatriz Peñalver Bernabé--BMC--**Dynamic genome-scale cell-specific metabolic models reveal novel inter-cellular and intra-cellular metabolic communications during ovarian follicle development**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2825-2)
    * --2019--Daphne Ezer--BMC--**NITPicker: selecting time points for follow-up experiments**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2717-5)
    * --2019--Chuanyan Wu--BMC--**PTPD: predicting therapeutic peptides by deep learning and word2vec**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3006-z)
    * --2019--Beiji Zou--BMC--**A novel glaucomatous representation method based on Radon and wavelet transform**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3267-6)
    * --2019--Elies Ramon--BMC--**HIV drug resistance prediction with weighted categorical kernel functions**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2991-2)
    * --2019--Zhongwang Yu--BMC--**ANDIS: an atomic angle- and distance-dependent statistical potential for protein structure quality assessment**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2898-y)
    * --2019--Christine P’ng--BMC--**BPG: Seamless, automated and interactive visualization of scientific data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2610-2)
    * --2019--Saugato Rahman Dhruba--BMC--**Recursive model for dose-time responses in pharmacological studies**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2831-4)
    * --2019--Kohei Amano--BMC--**Metabolic disassembler for understanding and predicting the biosynthetic units of natural products**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3183-9)
    * --2019--Sarah Machado--BMC--**LimeSeg: a coarse-grained lipid membrane simulation for 3D image segmentation**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2471-0)
    * --2019--Grzegorz M. Boratyn--BMC--**Magic-BLAST, an accurate RNA-seq aligner for long and short reads**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2996-x)
    * --2019--Yannick Martin--BMC--**WellInverter: a web application for the analysis of fluorescent reporter gene data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-2920-4)
    * --2019--Josefa Díaz Álvarez--BMC--**An application of machine learning with feature selection to improve diagnosis and classification of neurodegenerative disorders**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-019-3027-7)
    ## 2018
    * --2018--Peizhuo Wang--BMC--**Feature related multi-view nonnegative matrix factorization for identifying conserved functional modules in multiple biological networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2434-5)
    * --2018--Marek Ostaszewski--BMC--**Clustering approaches for visual knowledge exploration in molecular interaction networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2314-z)
    * --2018--Yuanfang Ren--BMC--**ProMotE: an efficient algorithm for counting independent motifs in uncertain network topologies**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2236-9)
    * --2018--Bo Xu--BMC--**Protein complexes identification based on go attributed network embedding**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2555-x)
    * --2018--Tong Liu--BMC--**Reconstructing high-resolution chromosome three-dimensional structures by Hi-C complex networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2464-z)
    * --2018--Yong He--BMC--**A new insight into underlying disease mechanism through semi-parametric latent differential network model**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2461-2)
    * --2018--Victor A. O. Carmelo--BMC--**WISH-R– a fast and efficient tool for construction of epistatic networks for complex traits and diseases**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2291-2)
    * --2018--Jie Hao--BMC--**PASNet: pathway-associated sparse deep neural network for prognosis prediction from high-throughput data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2500-z)
    * --2018--Christopher I Cooper--BMC--**Valection: design optimization for validation and verification studies**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2391-z)
    * --2018--Francesco Bardozzo--BMC--**A study on multi-omic oscillations in Escherichia coli metabolic networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2175-5)
    * --2018--Marco Fornili--BMC--**Modeling the covariates effects on the hazard function by piecewise exponential artificial neural networks: an application to a controlled clinical trial on renal carcinoma**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2179-1)
    * --2018--Helen N. Catanese--BMC--**A nearest-neighbors network model for sequence data reveals new insight into genotype distribution of a pathogen**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2453-2)
    * --2018--Lei Huang--BMC--**Completing sparse and disconnected protein-protein network by deep learning**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2112-7)
    * --2018--Justin M. Wozniak--BMC--**CANDLE/Supervisor: a workflow framework for machine learning applied to cancer research**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2508-4)
    * --2018--Cong Pian--BMC--**Discovering the ‘Dark matters’ in expression data of miRNA based on the miRNA-mRNA and miRNA-lncRNA networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2410-0)
    * --2018--Pau Farré--BMC--**Dense neural networks for predicting chromatin conformation**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2286-z)
    * --2018--Mina Khoshdeli--BMC--**Fusion of encoder-decoder deep networks improves delineation of multiple nuclear phenotypes**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2285-0)
    * --2018--Shuonan Chen--BMC--**Evaluating methods of inferring gene regulatory networks highlights their lack of performance for single cell gene expression data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2217-z)
    * --2018--Kazunori D. Yamada--BMC--**De novo profile generation based on sequence context specificity with the long short-term memory network**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2284-1)
    * --2018--Katarina Elez--BMC--**Distinguishing crystallographic from biological interfaces in protein complexes: role of intermolecular contacts and energetics for classification**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2414-9)
    * --2018--Diego Fioravanti--BMC--**Phylogenetic convolutional neural networks in metagenomics**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2033-5)
    * --2018--Gamal Crichton--BMC--**Neural networks for link prediction in realistic biomedical graphs: a multi-dimensional evaluation of graph embedding-based approaches**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2163-9)
    * --2018--Yoichi Takenaka--BMC--**Automated transition analysis of activated gene regulation during diauxic nutrient shift in Escherichia coli and adipocyte differentiation in mouse cells**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2072-y)
    * --2018--Faliu Yi--BMC--**Microvessel prediction in H&E Stained Pathology Images using fully convolutional neural networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2055-z)
    * --2018--Lokmane Chebouba--BMC--**Discriminate the response of Acute Myeloid Leukemia patients to treatment by using proteomics data and Answer Set Programming**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2034-4)
    * --2018--Phan Nguyen--BMC--**Time-lagged Ordered Lasso for network inference**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2558-7)
    * --2018--Christine Sinoque--BMC--**A method combining a random forest-based technique with the modeling of linkage disequilibrium through latent variables, to run multilocus genome-wide association studies**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2054-0)
    * --2018--Yaogong Zhang--BMC--**Prioritizing disease genes with an improved dual label propagation framework**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2040-6)
    * --2018--Régis Garnier--BMC--**Comparison of metaheuristics to measure gene effects on phylogenetic supports and topologies**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2172-8)
    * --2018--Makbule Guclin Ozsoy--BMC--**Realizing drug repositioning by adapting a recommendation system to handle the process**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2142-1)
    * --2018--Guido Zampieri--BMC--**Scuba: scalable kernel-based gene prioritization**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2025-5)
    * --2018--Shunyun Yang--BMC--**cmFSM: a scalable CPU-MIC coordinated drug-finding tool by frequent subgraph mining**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2071-z)
    * --2018--Yiqian Zhang--BMC--**DeepM6ASeq: prediction and characterization of m6A-containing sequences using deep learning**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2516-4)
    * --2018--Jonathan Ozik--BMC--**High-throughput cancer hypothesis testing with an integrated PhysiCell-EMEWS workflow**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2510-x)
    * --2018--Guifeng Tang--BMC--**Sequence-based bacterial small RNAs prediction using ensemble learning strategies**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2535-1)
    * --2018--Jian-Yu Shi--BMC--**BMCMDA: a novel model for predicting human microbe-disease associations via binary matrix completion**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2274-3)
    * --2018--Wen Zhang--BMC--**Predicting drug-disease associations by using similarity constrained matrix factorization**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2220-4)
    * --2018--Reza Rezazadegan--BMC--**Degeneracy and genetic assimilation in RNA evolution**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2497-3)
    * --2018--Abhik Seal--BMC--**Netpredictor: R and Shiny package to perform drug-target network analysis and prediction of missing links**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2254-7)
    * --2018--Jianing Xi--BMC--**Discovering mutated driver genes through a robust and sparse co-regularized matrix factorization framework with prior information from mRNA expression patterns and interaction network**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2218-y)
    * --2018--Chi Xiao--BMC--**Effective automated pipeline for 3D reconstruction of synapses based on deep learning**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2232-0)
    * --2018--Nhat Tran--BMC--**MicroRNA dysregulational synergistic network: discovering microRNA dysregulatory modules across subtypes in non-small cell lung cancers**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2536-0)
    * --2018--Debsindhu Bhowmik--BMC--**Deep clustering of protein folding simulations**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2507-5)
    * --2018--Simone Dinarelli--BMC--**FC_analysis: a tool for investigating atomic force microscopy maps of force curves**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2265-4)
    * --2018--Yözen Hernández--BMC--**BpWrapper: BioPerl-based sequence and tree utilities for rapid prototyping of bioinformatics pipelines**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2074-9)
    * --2018--Maciej Antczak--BMC--**RNAfitme: a webserver for modeling nucleobase and nucleoside residue conformation in fixed-backbone RNA structures**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2317-9)
    * --2018--Riccardo Colombo--BMC--**Emerging ensembles of kinetic parameters to characterize observed metabolic phenotypes**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2181-7)
    * --2018--Peiying Ruan--BMC--**Improving prediction of heterodimeric protein complexes using combination with pairwise kernel**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2017-5)
    * --2018--Senthil B. Girimurugan--BMC--**iSeg: an efficient algorithm for segmentation of genomic and epigenomic data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2140-3)
    * --2018--Buzhong Zhang--BMC--**Prediction of 8-state protein secondary structures by a novel deep learning architecture**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2280-5)
    * --2018--Alperen Dalkiran--BMC--**ECPred: a tool for the prediction of the enzymatic functions of protein sequences based on the EC nomenclature**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2368-y)
    * --2018--Michail Tsagris--BMC--**Feature selection for high-dimensional temporal data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2023-7)
    * --2018--Lacramioara Dranca--BMC--**Using Kinect to classify Parkinson’s disease stages related to severity of gait impairment**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2488-4)
    * --2018--Yujuan Gao--BMC--**RaptorX-Angle: real-value prediction of protein backbone dihedral angles through a hybrid method of clustering and deep learning**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2065-x)
    * --2018--Yifeng Li--BMC--**Genome-wide prediction of cis-regulatory regions using supervised deep learning methods**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2187-1)
    * --2018--Yousef Al-Kofahi--BMC--**A deep learning-based algorithm for 2-D cell segmentation in microscopy images**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2375-z)
    * --2018--Xiaoming Liu--BMC--**The International Conference on Intelligent Biology and Medicine (ICIBM) 2018: bioinformatics towards translational applications**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2460-3)
    * --2018--Ali Foroughi pour--BMC--**Heuristic algorithms for feature selection under Bayesian models with block-diagonal covariance structure**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2059-8)
    * --2018--Viachaslau Tsyvina--BMC--**Fast estimation of genetic relatedness between members of heterogeneous populations of closely related genomic variants**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2333-9)
    * --2018--Sebastian Daberdaku--BMC--**Exploring the potential of 3D Zernike descriptors and SVM for protein–protein interface prediction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2043-3)
    * --2018--Konstantina Dimitrakopoulou--BMC--**Deblender: a semi−/unsupervised multi-operational computational method for complete deconvolution of expression data from heterogeneous samples**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2442-5)
    * --2018--Roman Prytuliak--BMC--**SLALOM, a flexible method for the identification and statistical analysis of overlapping continuous sequence elements in sequence- and time-series data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2020-x)
    * --2018--Xianchao Zhu--BMC--**Nonlinear expression and visualization of nonmetric relationships in genetic diseases and microbiome data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2537-z)
    * --2018--Xingang Jia--BMC--**Analyzing the similarity of samples and genes by MG-PCC algorithm, t-SNE-SS and t-SNE-SG maps**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2495-5)
    * --2018--Raza-Ur Rahman--BMC--**Oasis 2: improved online analysis of small RNA-seq data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2047-z)
    * --2018--Sinan Abo Alchamlat--BMC--**Aggregation of experts: an application in the field of “interactomics” (detection of interactions on the basis of genomic data)**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2447-0)
    * --2018--Armen Abnousi--BMC--**Alignment-free clustering of large data sets of unannotated protein conserved regions using minhashing**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2080-y)
    * --2018--Yanhua Qiao--BMC--**Protein-protein interface hot spots prediction based on a hybrid feature selection strategy**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2009-5)
    * --2018--Skylar W. Marvel--BMC--**ToxPi Graphical User Interface 2.0: Dynamic exploration, visualization, and sharing of integrated data models**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2089-2)
    * --2018--Muxin Gu--BMC--**Semi-supervised machine learning for automated species identification by collagen peptide mass fingerprinting**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2221-3)
    * --2018--Florian Burger--BMC--**PCRdrive: the largest qPCR assay archive to date and endless potential for lab workflow revitalization**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2452-3)
    ## 2017
    * --2017--Jing Liu--BMC--**A time series driven decomposed evolutionary optimization approach for reconstructing large-scale gene regulatory networks based on fuzzy cognitive maps**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1657-1)
    * --2017--Shahin Boluki--BMC--**Incorporating biological prior knowledge for Bayesian learning via maximal knowledge-driven information priors**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1893-4)
    * --2017--Yiming Zuo--BMC--**Incorporating prior biological knowledge for network-based differential gene expression analysis using differentially weighted graphical LASSO**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1515-1)
    * --2017--Yusuke Azuma--BMC--**Biologically constrained optimization based cell membrane segmentation in C. elegans embryos**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1717-6)
    * --2017--Tianyu Kang--BMC--**A biological network-based regularized artificial neural network model for robust phenotype prediction from gene expression data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1984-2)
    * --2017--Ziyi Li--BMC--**Incorporating biological information in sparse principal component analysis with application to genomic data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1740-7)
    * --2017--Shirin Taghipour--BMC--**Improving protein complex prediction by reconstructing a high-confidence protein-protein interaction network of Escherichia coli from different physical interaction data sources**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1422-x)
    * --2017--Chuanchao Zhang--BMC--**Comparative network stratification analysis for identifying functional interpretable network biomarkers**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1462-x)
    * --2017--Le Ou-Yang--BMC--**A multi-network clustering method for detecting protein complexes from multiple heterogeneous networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1877-4)
    * --2017--Hyundoo Jeong--BMC--**CUFID-query: accurate network querying through random walk based network flow estimation**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1899-y)
    * --2017--Marianna Milano--BMC--**An extensive assessment of network alignment algorithms for comparison of brain connectomes**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1635-7)
    * --2017--Wen Zhang--BMC--**Predicting potential drug-drug interactions by integrating chemical, biological, phenotypic and network data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1415-9)
    * --2017--Shupeng Gui--BMC--**A scalable algorithm for structure identification of complex gene regulatory network from temporal expression data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1489-z)
    * --2017--Xu Min--BMC--**Predicting enhancers with deep convolutional neural networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1878-3)
    * --2017--Li Ma--BMC--**CURE-SMOTE algorithm and hybrid algorithm for feature selection and parameter optimization based on random forests**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1578-z)
    * --2017--Kazuhiro Takemoto--BMC--**Limitations of a metabolic network-based reverse ecology method for inferring host–pathogen interactions**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1696-7)
    * --2017--Hao He--BMC--**Comparison of statistical methods for subnetwork detection in the integration of gene expression and protein interaction network**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1567-2)
    * --2017--Byung-Jun Yoon--BMC--**Selected research articles from the 2016 International Workshop on Computational Network Biology: Modeling, Analysis, and Control (CNB-MAC)**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1521-3)
    * --2017--Kang K. Yan--BMC--**A comparison of graph- and kernel-based –omics data integration algorithms for classifying complex traits**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1982-4)
    * --2017--Jie Tan--BMC--**ADAGE signature analysis: differential expression analysis with data-defined gene sets**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1905-4)
    * --2017--Kevin Matlock--BMC--**Combination therapy design for maximizing sensitivity and minimizing toxicity**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1523-1)
    * --2017--Xiaoyong Pan--BMC--**RNA-protein binding motifs mining with a new hybrid deep learning based cross-domain knowledge integration approach**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1561-8)
    * --2017--Garrett W. Birkel--BMC--**The JBEI quantitative metabolic modeling library (jQMM): a python library for modeling microbial metabolism**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1615-y)
    * --2017--Lisa Katharina Blaß--BMC--**Network design and analysis for multi-enzyme biocatalysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1773-y)
    * --2017--David R. Penas--BMC--**Parameter estimation in large-scale systems biology models: a parallel and self-adaptive cooperative strategy**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1452-4)
    * --2017--Lei Deng--BMC--**A sparse autoencoder-based deep neural network for protein solvent accessibility and contact number prediction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1971-7)
    * --2017--Atif Khan--BMC--**Meta-analysis of cell- specific transcriptomic data using fuzzy c-means clustering discovers versatile viral responsive genes**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1669-x)
    * --2017--James C. Wilgenbusch--BMC--**Visualizing phylogenetic tree landscapes**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1479-1)
    * --2017--Alex M. Ascensión--BMC--**NaviSE: superenhancer navigator integrating epigenomics signal algebra**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1698-5)
    * --2017--Jonathan D. Young--BMC--**Unsupervised deep learning reveals prognostically relevant subtypes of glioblastoma**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1798-2)
    * --2017--Zhi-Ping Liu--BMC--**Structure alignment-based classification of RNA-binding pockets reveals regional RNA recognition motifs on protein surfaces**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1410-1)
    * --2017--Nicola Lazzarini--BMC--**RGIFE: a ranked guided iterative feature elimination heuristic for the identification of biomarkers**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1729-2)
    * --2017--Zhen Zhang--BMC--**An integrated enhancement and reconstruction strategy for the quantitative extraction of actin stress fibers from fluorescence micrographs**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1684-y)
    * --2017--Sunny Raj--BMC--**A theorem proving approach for automatically synthesizing visualizations of flow cytometry data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1662-4)
    * --2017--Clarence White--BMC--**CNN-BLPred: a Convolutional neural network based predictor for β-Lactamases (BL) and their classes**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1972-6)
    * --2017--Yanan Zhu--BMC--**A deep convolutional neural network approach to single-particle recognition in cryo-electron microscopy**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1757-y)
    * --2017--Michael I. Klein--BMC--**GRAPE: a pathway template method to characterize tissue-specific functionality from gene expression profiles**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1711-z)
    * --2017--Wei Zheng--BMC--**An attention-based effective neural model for drug-drug interactions extraction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1855-x)
    * --2017--Yahya Bokhari--BMC--**QuaDMutEx: quadratic driver mutation explorer**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1869-4)
    * --2017--Biswanath Chowdhury--BMC--**An optimized approach for annotation of large eukaryotic genomic sequences using genetic algorithm**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1874-7)
    * --2017--Bo Wen--BMC--**metaX: a flexible and comprehensive software for processing metabolomics data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1579-y)
    * --2017--Herty Liany--BMC--**MultiDCoX: Multi-factor analysis of differential co-expression**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1963-7)
    * --2017--Andrea Tangherloni--BMC--**LASSIE: simulating large-scale models of biochemical systems on GPUs**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1666-0)
    * --2017--Sinan Abo Alchamlat--BMC--**KNN-MDR: a learning approach for improving interactions mapping performances in genome wide association studies**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1599-7)
    * --2017--Pratyaydipta Rudra--BMC--**Model based heritability scores for high-throughput sequencing data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1539-6)
    * --2017--Alex--BMC--**Vermont: a multi-perspective visual interactive platform for mutational analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1789-3)
    * --2017--Tao Huang--BMC--**MOST: most-similar ligand based approach to target prediction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1586-z)
    * --2017--Peter Bajcsy--BMC--**Modeling, validation and verification of three-dimensional cell-scaffold contacts from terabyte-sized images**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1928-x)
    * --2017--Michael Kern--BMC--**Interactive visual exploration and refinement of cluster assignments**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1813-7)
    * --2017--W. Duncan Wadsworth--BMC--**An integrative Bayesian Dirichlet-multinomial regression model for the analysis of taxonomic abundances in microbiome data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1516-0)
    * --2017--José Cleydson F. Silva--BMC--**Fangorn Forest (F2): a machine learning approach to classify genes and genera in the family Geminiviridae**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1839-x)
    * --2017--Sophie Engle--BMC--**Unboxing cluster heatmaps**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1442-6)
    * --2017--Sugeerth Murugesan--BMC--**Multi-scale visual analysis of time-varying electrocorticography data via clustering of brain regions**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1633-9)
    * --2017--Kyoungyeul Lee--BMC--**Utilizing random Forest QSAR models with optimized parameters for target identification and its application to target-fishing server**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1960-x)
    * --2017--Juan C. Aledo--BMC--**A machine learning approach for predicting methionine oxidation sites**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1848-9)
    * --2017--Eleni Ioanna Delatola--BMC--**SegCorr a statistical procedure for the detection of genomic regions of correlated expression**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1742-5)
    * --2017--Santosh Philips--BMC--**Using machine learning algorithms to identify genes essential for cell survival**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1799-1)
    * --2017--Richard J. Cotton--BMC--**flippant–An R package for the automated analysis of fluorescence-based scramblase assays**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1542-y)
    * --2017--Onur Erbilgin--BMC--**Dynamic substrate preferences predict metabolic properties of a simple microbial consortium**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1478-2)
    * --2017--Kuang-Lim Chan--BMC--**Seqping: gene prediction pipeline for plant genomes using self-training gene models and transcriptomic data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1426-6)
    * --2017--Daniel Ian McSkimming--BMC--**Classifying kinase conformations using a machine learning approach**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1506-2)
    * --2017--Alvin Farrel--BMC--**An efficient algorithm for improving structure-based prediction of transcription factor binding sites**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1755-0)
    * --2017--Youyi Fong--BMC--**chngpt: threshold regression model estimation and inference**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1863-x)
    * --2017--Xue Jiang--BMC--**Identify Huntington’s disease associated genes based on restricted Boltzmann machine with RNA-seq data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1859-6)
    * --2017--Mengdi Zhao--BMC--**Segmentation and classification of two-channel C. elegans nucleus-labeled fluorescence images**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1817-3)
    * --2017--Linlin Gao--BMC--**Brain medical image diagnosis based on corners with importance-values**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1903-6)
    * --2017--Raquel Dias--BMC--**Improving the accuracy of high-throughput protein-protein affinity prediction may require better training data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-017-1533-z)
    ## 2016
    * --2016--Julien Dorier--BMC--**Boolean regulatory network reconstruction using literature based knowledge with a genetic algorithm optimization method**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1287-z)
    * --2016--Sapna Kumari--BMC--**Bottom-up GGM algorithm for constructing multilayered hierarchical gene regulatory networks that govern biological pathways or processes**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0981-1)
    * --2016--Francesco Iorio--BMC--**Efficient randomization of biological networks while preserving functional characterization of individual nodes**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1402-1)
    * --2016--Veronica Vinciotti--BMC--**Consistency of biological networks inferred from microarray and sequencing data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1136-0)
    * --2016--Xiao-Fei Zhang--BMC--**Comparative analysis of housekeeping and tissue-specific driver nodes in human protein interaction networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1233-0)
    * --2016--Joeri Ruyssinck--BMC--**Netter: re-ranking gene network inference predictions using structural network properties**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0913-0)
    * --2016--Matteo Bersanelli--BMC--**Methods for the integration of multi-omics data: mathematical aspects**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0857-9)
    * --2016--Ehsan Kazemi--BMC--**PROPER: global protein interaction network alignment through percolation matching**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1395-9)
    * --2016--Daniele Muraro--BMC--**An integrative analysis of gene expression and molecular interaction data to identify dys-regulated sub-networks in inflammatory bowel disease**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0886-z)
    * --2016--Meng-Yun Wu--BMC--**Regularized logistic regression with network-based pairwise interaction for biomarker identification in breast cancer**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0951-7)
    * --2016--Andrew J. Sedgewick--BMC--**Learning mixed graphical models with separate sparsity parameters and stability-based model selection**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1039-0)
    * --2016--André C. A. Nascimento--BMC--**A multiple kernel learning algorithm for drug-target interaction prediction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0890-3)
    * --2016--Chee Yee Lim--BMC--**BTR: training asynchronous Boolean models using single-cell expression data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1235-y)
    * --2016--Séverine Affeldt--BMC--**3off2: A network reconstruction algorithm based on 2-point and 3-point information statistics**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0856-x)
    * --2016--Sefa Kılıç--BMC--**Assessment of transfer methods for comparative genomics of regulatory networks in bacteria**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1113-7)
    * --2016--Vicente Acuña--BMC--**Deciphering transcriptional regulations coordinating the response to environmental changes**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0885-0)
    * --2016--Patrick K. O’Neill--BMC--**Parametric bootstrapping for biological sequence motifs**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1246-8)
    * --2016--Hussein A. Hejase--BMC--**A scalability study of phylogenetic network inference methods using empirical datasets and simulations involving a single reticulation**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1277-1)
    * --2016--Sumanta Ray--BMC--**A NMF based approach for integrating multiple data sources to predict HIV-1–human PPIs**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0952-6)
    * --2016--Hirotaka Matsumoto--BMC--**SCOUP: a probabilistic model based on the Ornstein–Uhlenbeck process to analyze single-cell expression data during differentiation**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1109-3)
    * --2016--Lujia Chen--BMC--**Learning a hierarchical representation of the yeast transcriptomic machinery using an autoencoder model**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0852-1)
    * --2016--André Veríssimo--BMC--**DegreeCox – a network-based regularization method for survival analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1310-4)
    * --2016--Le Ou-Yang--BMC--**Protein complex detection based on partially shared multi-view clustering**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1164-9)
    * --2016--Ricardo Cerri--BMC--**Reduction strategies for hierarchical multi-label classification in protein function prediction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1232-1)
    * --2016--Shun Guo--BMC--**Gene regulatory network inference using PLS-based methods**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1398-6)
    * --2016--Eric C. Rouchka--BMC--**Proceedings of the 15th Annual UT-KBRIN Bioinformatics Summit 2016**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1154-y)
    * --2016--Arnav Kapur--BMC--**Gene expression prediction using low-rank matrix completion**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1106-6)
    * --2016--Shibiao Wan--BMC--**Sparse regressions for predicting and interpreting subcellular localization of multi-label proteins**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0940-x)
    * --2016--Pi-Jing Wei--BMC--**LNDriver: identifying driver genes by integrating mutation and expression data based on gene-gene interaction network**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1332-y)
    * --2016--Laurence Yang--BMC--**solveME: fast and reliable solution of nonlinear ME models**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1240-1)
    * --2016--Yijie Ding--BMC--**Predicting protein-protein interactions via multivariate mutual information of protein sequences**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1253-9)
    * --2016--Neel Prabh--BMC--**Are orphan genes protein-coding, prediction artifacts, or non-coding RNAs?**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1102-x)
    * --2016--Le Ou-Yang--BMC--**A two-layer integration framework for protein complex detection**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0939-3)
    * --2016--Moritz von Stosch--BMC--**A principal components method constrained by elementary flux modes: analysis of flux data sets**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1063-0)
    * --2016--Mirela Domijan--BMC--**PeTTSy: a computational tool for perturbation analysis of complex systems biology models**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0972-2)
    * --2016--Ryan Eshleman--BMC--**Leveraging graph topology and semantic context for pharmacovigilance through twitter-streams**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1220-5)
    * --2016--Yang Liu--BMC--**PGen: large-scale genomic variations analysis workflow and browser in SoyKB**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1227-y)
    * --2016--Ariful Azad--BMC--**flowVS: channel-specific variance stabilization in flow cytometry**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1083-9)
    * --2016--Wei Cheng--BMC--**Sparse regression models for unraveling group and individual associations in eQTL mapping**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0986-9)
    * --2016--Yang Song--BMC--**Bioimage classification with subcategory discriminant transform of high dimensional visual descriptors**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1318-9)
    * --2016--Youzhong Liu--BMC--**MetICA: independent component analysis for high-resolution mass-spectrometry based non-targeted metabolomics**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0970-4)
    * --2016--Otte Heinävaara--BMC--**On the inconsistency of ℓ
    1-penalised sparse precision matrix estimation**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1309-x)
    * --2016--Said el Bouhaddani--BMC--**Evaluation of O2PLS in Omics data integration**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0854-z)
    * --2016--Junliang Shang--BMC--**CINOEDV: a co-information based method for detecting and visualizing n-order epistatic interactions**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1076-8)
    * --2016--David A. duVerle--BMC--**CellTree: an R/bioconductor package to infer the hierarchical structure of cell populations from single-cell RNA-seq data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1175-6)
    * --2016--Javier Pérez-Rodríguez--BMC--**Stepwise approach for combining many sources of evidence for site-recognition in genomic sequences**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0968-y)
    * --2016--Nicolas Rodriguez--BMC--**The systems biology format converter**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1000-2)
    * --2016--Chao Fan--BMC--**PredRSA: a gradient boosted regression trees approach for predicting protein solvent accessibility**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0851-2)
    * --2016--Yuan Chen--BMC--**Informative gene selection and the direct classification of tumors based on relative simplicity**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0893-0)
    * --2016--David Gomez-Cabrero--BMC--**From comorbidities of chronic obstructive pulmonary disease to identification of shared molecular mechanisms by data integration**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1291-3)
    * --2016--Yu-An Huang--BMC--**Sequence-based prediction of protein-protein interactions using weighted sparse representation model combined with global encoding**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1035-4)
    * --2016--Chan-Seok Jeong--BMC--**Structure-based Markov random field model for representing evolutionary constraints on functional sites**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0948-2)
    * --2016--Xi Hang Cao--BMC--**A robust data scaling algorithm to improve classification accuracies in biomedical data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1236-x)
    * --2016--Fern--BMC--**Efficient and automated large-scale detection of structural relationships in proteins with a flexible aligner**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0866-8)
    * --2016--Timothy DeFreitas--BMC--**GEMINI: a computationally-efficient search engine for large gene expression datasets**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-0934-8)
    * --2016--David M. Budden--BMC--**Distributed gene expression modelling for exploring variability in epigenetic function**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1313-1)
    * --2016--Aaron Y. Lee--BMC--**Scalable metagenomics alignment research tool (SMART): a scalable, rapid, and complete search heuristic for the classification of metagenomic sequences from complex sequence populations**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1159-6)
    * --2016--Chaitanya R. Acharya--BMC--**Exploiting expression patterns across multiple tissues to map expression quantitative trait loci**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-016-1123-5)
    ## 2015
    * --2015--Ward C Wheele--BMC--**Phylogenetic network analysis as a parsimony optimization problem**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0675-0)
    * --2015--Xiao-Fei Zhang--BMC--**Determining minimum set of driver nodes in protein-protein interaction networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0591-3)
    * --2015--Faraz Hussain--BMC--**Automated parameter estimation for biological models using Bayesian statistical model checking**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S17-S8)
    * --2015--Naresh Doni Jayavelu--BMC--**Iterative sub-network component analysis enables reconstruction of large scale genetic networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0768-9)
    * --2015--Ernst C Wit--BMC--**Inferring slowly-changing dynamic gene-regulatory networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S6-S5)
    * --2015--Shun Yao--BMC--**Prior knowledge driven Granger causality analysis on gene regulatory network discovery**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0710-1)
    * --2015--Roozbeh Dehghannasiri--BMC--**Efficient experimental design for uncertainty reduction in gene regulatory networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S13-S2)
    * --2015--Aurélie Pirayre--BMC--**BRANE Cut: biologically-related a priori network enhancement with graph cuts for gene regulatory network inference**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0754-2)
    * --2015--Zina M Ibrahim--BMC--**The relative vertex clustering value - a new criterion for the fast discovery of functional modules in protein interaction networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S4-S3)
    * --2015--Akito Taneda--BMC--**Multi-objective optimization for RNA design with multiple target secondary structures**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0706-x)
    * --2015--Wuming Gong--BMC--**Inferring dynamic gene regulatory networks in cardiac differentiation through the integration of multi-dimensional data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0460-0)
    * --2015--Ahmet Ay--BMC--**Hierarchical decomposition of dynamically evolving regulatory networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0529-9)
    * --2015--Jakob Dohrmann--BMC--**Global multiple protein-protein interaction network alignment by combining pairwise network alignments**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S13-S11)
    * --2015--Wenhui Wang--BMC--**Network tuned multiple rank aggregation and applications to gene ranking**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S1-S6)
    * --2015--Lun Hu--BMC--**A density-based clustering approach for identifying overlapping protein complexes with functional preferences**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0583-3)
    * --2015--Anirban Bhar--BMC--**Multiobjective triclustering of time-series transcriptome data reveals key genes of biological processes**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0635-8)
    * --2015--Hossam M Ashtawy--BMC--**BgN-Score and BsN-Score: Bagging and boosting based ensemble neural networks scoring functions for accurate binding affinity prediction of protein-ligand complexes**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S4-S8)
    * --2015--Syed Murtuza Baker--BMC--**A unified framework for estimating parameters of kinetic biological models**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0500-9)
    * --2015--Peter Bajcsy--BMC--**Survey statistics of automated segmentations applied to optical imaging of mammalian cells**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0762-2)
    * --2015--Sven Thiele--BMC--**Extended notions of sign consistency to relate experimental data to signaling and regulatory network topologies**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0733-7)
    * --2015--Wei Cheng--BMC--**Fast and robust group-wise eQTL mapping using sparse graphical models**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0421-z)
    * --2015--Suyu Mei--BMC--**Multi-label multi-instance transfer learning for simultaneous reconstruction and cross-talk modeling of multiple human signaling pathways**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0841-4)
    * --2015--Vasily Sachnev--BMC--**Multi-class BCGA-ELM based classifier that identifies biomarkers associated with hallmarks of cancer**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0565-5)
    * --2015--Mark-Anthony Bray--BMC--**CellProfiler Tracer: exploring and validating high-throughput, time-lapse microscopy image data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0759-x)
    * --2015--Thomas Provoost--BMC--**Semi-supervised Learning for the BioNLP Gene Regulation Network**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S10-S4)
    * --2015--Amirhossein Manzourolajdad--BMC--**Secondary structural entropy in RNA switch (Riboswitch) identification**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0523-2)
    * --2015--Wen Zhang--BMC--**Predicting drug side effects by multi-label learning and ensemble learning**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0774-y)
    * --2015--Pietro Pinoli--BMC--**Computational algorithms to predict Gene Ontology annotations**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S6-S4)
    * --2015--Tianmin Wang--BMC--**DomSign: a top-down annotation pipeline to enlarge enzyme space in the protein universe**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0499-y)
    * --2015--Kevin M Livingston--BMC--**KaBOB: ontology-based semantic integration of biomedical databases**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0559-3)
    * --2015--Michaela Bayerlová--BMC--**Comparative study on gene set and pathway topology-based enrichment methods**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0751-5)
    * --2015--Felix Schönenberger--BMC--**Discrimination of cell cycle phases in PCNA-immunolabeled cells**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0618-9)
    * --2015--Stefano Parodi--BMC--**Differential diagnosis of pleural mesothelioma using Logic Learning Machine**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S9-S3)
    * --2015--Elsa Bernard--BMC--**A convex formulation for joint RNA isoform detection and quantification from multiple RNA-seq samples**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0695-9)
    * --2015--Watcharaporn Tanchotsrinon--BMC--**A high performance prediction of HPV genotypes by Chaos game representation and singular value decomposition**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0493-4)
    * --2015--Maureen Stolzer--BMC--**Event inference in multidomain families with phylogenetic reconciliation**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S14-S8)
    * --2015--George Tsatsaronis--BMC--**An overview of the BIOASQ large-scale biomedical semantic indexing and question answering competition**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0564-6)
    * --2015--Abdulkadir Elmas--BMC--**Reconstruction of novel transcription factor regulons through inference of their binding sites**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0685-y)
    * --2015--Giulio Caravagna--BMC--**Automatising the analysis of stochastic biochemical time-series**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S9-S8)
    * --2015--Catalina V Anghel--BMC--**ISOpureR: an R implementation of a computational purification algorithm of mixed tumour profiles**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0597-x)
    * --2015--Tamara Vasylenko--BMC--**SCMPSP: Prediction and characterization of photosynthetic proteins based on a scoring card method**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S1-S8)
    * --2015--Joan Maynou--BMC--**Sequence information gain based motif analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0811-x)
    * --2015--Itziar Frades--BMC--**Comparison of phosphorylation patterns across eukaryotes by discriminative N-gram analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0657-2)
    * --2015--Bui Ngoc Thang--BMC--**A semi–supervised tensor regression model for siRNA efficacy prediction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0495-2)
    * --2015--Josef Laimer--BMC--**MAESTRO - multi agent stability prediction upon point mutations**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0548-6)
    * --2015--Florin Chelaru--BMC--**Epiviz: a view inside the design of an integrated visual analysis software for genomics**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-16-S11-S4)
    * --2015--Dániel Kozma--BMC--**TMFoldRec: a statistical potential-based transmembrane protein fold recognition tool**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0638-5)
    * --2015--Talha Bin Masood--BMC--**CHEXVIS: a tool for molecular channel extraction and visualization**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0545-9)
    * --2015--Robert J Flassig--BMC--**Rational selection of experimental readout and intervention sites for reducing uncertainties in computational model predictions**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0436-5)
    * --2015--Federico Comoglio--BMC--**Sensitive and highly resolved identification of RNA-protein interaction sites in PAR-CLIP data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0470-y)
    * --2015--Ralf Eggeling--BMC--**Inferring intra-motif dependencies of DNA binding sites from ChIP-seq data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0797-4)
    * --2015--Xizeng Mao--BMC--**Revisiting operons: an analysis of the landscape of transcriptional units in E. coli**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0805-8)
    * --2015--Michaël Vyverman--BMC--**A Long Fragment Aligner called ALFALFA**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0533-0)
    * --2015--Angela Serra--BMC--**MVDA: a multi-view genomic data integration methodology**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0680-3)
    * --2015--Mohammed AlQuraishi--BMC--**An affinity-structure database of helix-turn-helix: DNA complexes with a universal coordinate system**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0819-2)
    * --2015--Agnieszka Rybarczyk--BMC--**New in silico approach to assessing RNA secondary structures with non-canonical base pairs**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0718-6)
    * --2015--Caroline König--BMC--**Label noise in subtype discrimination of class C G protein-coupled receptors: A systematic approach to the analysis of classification errors**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0731-9)
    * --2015--Gongning Luo--BMC--**Neuron anatomy structure reconstruction based on a sliding filter**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-015-0780-0)
    ## 2014
    * --2014--Yu-Ting Hsiao--BMC--**Reverse engineering gene regulatory networks: Coupling an optimization algorithm with a parameter identification technique**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S15-S8)
    * --2014--Zhimin Deng--BMC--**A continuous optimization approach for inferring parameters in mathematical models of regulatory networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-256)
    * --2014--Enzo Acerbi--BMC--**Gene network inference using continuous time Bayesian networks: a comparative study and application to Th17 cell differentiation**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0387-x)
    * --2014--Young Hwan Chang--BMC--**Exact reconstruction of gene regulatory networks using compressive sensing**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0400-4)
    * --2014--Florian Martin--BMC--**Quantification of biological network perturbations for mechanistic insight and diagnostics using two-layer causal models**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-238)
    * --2014--Zeeshan Gillani--BMC--**CompareSVM: supervised, Support Vector Machine (SVM) inference of gene regularity networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0395-x)
    * --2014--Jose A Egea--BMC--**MEIGO: an open-source software suite based on metaheuristics for global optimization in systems biology and bioinformatics**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-136)
    * --2014--Jun Zhong Ji--BMC--**MAE-FMD: Multi-agent evolutionary method for functional module detection in protein-protein interaction networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-325)
    * --2014--Alberto Noronha--BMC--**An integrated network visualization framework towards metabolic engineering applications**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0420-0)
    * --2014--William C Ray--BMC--**Addressing the unmet need for visualizing conditional random fields in biological data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-202)
    * --2014--Gianvito Pio--BMC--**Integrating microRNA target predictions for the discovery of gene regulatory networks: a semi-supervised ensemble learning approach**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S1-S4)
    * --2014--Jun Sun--BMC--**Biochemical systems identification by a random drift particle swarm optimization approach**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S6-S1)
    * --2014--Le Ou-Yang--BMC--**Detecting temporal protein complexes from dynamic protein-protein interaction networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-335)
    * --2014--Narsis A Kiani--BMC--**Dynamic probabilistic threshold networks to infer signaling pathways from time-course perturbation data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-250)
    * --2014--Víctor Martínez--BMC--**ProphNet: A generic prioritization method through propagation of information**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S1-S5)
    * --2014--Abiel Roche-Lima--BMC--**Metabolic network prediction through pairwise rational kernels**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-318)
    * --2014--Kasper Dinkla--BMC--**eXamine: Exploring annotated modules in networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-201)
    * --2014--Elisabetta Fersini--BMC--**A p-Median approach for predicting drug response in tumour cells**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0353-7)
    * --2014--Marianna Budnikova--BMC--**Design of a flexible component gathering algorithm for converting cell-based models to graph representations for use in evolutionary search**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-178)
    * --2014--Yun Zhang--BMC--**On finding bicliques in bipartite graphs: a novel algorithm and its application to the integration of diverse biological data types**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-110)
    * --2014--Jose A Gomez--BMC--**DFBAlab: a fast and reliable MATLAB code for dynamic flux balance analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0409-8)
    * --2014--Mohammad Ali Moni--BMC--**Network-based analysis of comorbidities risk during an infection: SARS and HIV case studies**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-333)
    * --2014--Yiyi Liu--BMC--**A network-assisted co-clustering algorithm to discover cancer subtypes based on gene expression**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-37)
    * --2014--Claudio Saccà--BMC--**Improved multi-level protein–protein interaction prediction with semantic-based regularization**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-103)
    * --2014--Qianqian Wu--BMC--**Approximate Bayesian computation schemes for parameter inference of discrete stochastic models using simulated likelihood density**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S12-S3)
    * --2014--Lingjian Yang--BMC--**Pathway activity inference for multiclass disease classification through a mathematical programming optimisation framework**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0390-2)
    * --2014--Xiao-Fei Zhang--BMC--**Detecting overlapping protein complexes based on a generative model with functional and topological properties**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-186)
    * --2014--Chun-Hou Zheng--BMC--**Gene differential coexpression analysis based on biweight correlation and maximum clique**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S15-S3)
    * --2014--Junhua Zhang--BMC--**Discovery of co-occurring driver pathways in cancer**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-271)
    * --2014--Evrim Acar--BMC--**Structure-revealing data fusion**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-239)
    * --2014--Andrew Schoenrock--BMC--**Efficient prediction of human protein-protein interactions at a global scale**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0383-1)
    * --2014--Zhi-Ping Liu--BMC--**Systematic identification of transcriptional and post-transcriptional regulations in human respiratory epithelial cells during influenza A virus infection**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-336)
    * --2014--Anirban Mukhopadhyay--BMC--**Incorporating the type and direction information in predicting novel regulatory interactions between HIV-1 and human proteins using a biclustering approach**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-26)
    * --2014--Hamid Mohamadlou--BMC--**Maximizing Kolmogorov Complexity for accurate and robust bright field cell segmentation**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-32)
    * --2014--Minta Thomas--BMC--**Predicting breast cancer using an expression values weighted clinical classifier**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0411-1)
    * --2014--Leo van Iersel--BMC--**A practical approximation algorithm for solving massive instances of hybridization number for binary and nonbinary trees**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-127)
    * --2014--Chun-Pei Cheng--BMC--**MiningABs: mining associated biomarkers across multi-connected gene expression datasets**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-173)
    * --2014--Calem J Bendell--BMC--**Transient protein-protein interface prediction: datasets, features, algorithms, and the RAD-T predictor**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-82)
    * --2014--Anthony Santella--BMC--**A semi-local neighborhood-based framework for probabilistic cell lineage tracing**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-217)
    * --2014--Andrei Kramer--BMC--**Hamiltonian Monte Carlo methods for efficient parameter estimation in steady state dynamical systems**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-253)
    * --2014--Chen Meng--BMC--**A multivariate approach to the integration of multi-omics datasets**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-162)
    * --2014--Frauke Kracke--BMC--**Identifying target processes for microbial electrosynthesis by elementary mode analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0410-2)
    * --2014--Xiaoli Lin--BMC--**Protein structure prediction with local adjust tabu search algorithm**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S15-S1)
    * --2014--Maad Shatnawi--BMC--**Protein inter-domain linker prediction using Random Forest and amino acid physiochemical properties**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S16-S8)
    * --2014--Yi-Fan Liou--BMC--**SCMHBP: prediction and analysis of heme binding proteins using propensity scores of dipeptides**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S16-S4)
    * --2014--Narjeskhatoon Habibi--BMC--**A review of machine learning methods to predict the solubility of overexpressed recombinant proteins in Escherichia coli**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-134)
    * --2014--Edward WJ Curry--BMC--**A framework for generalized subspace pattern mining in high-dimensional datasets**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0355-5)
    * --2014--Zhu-Hong You--BMC--**Prediction of protein-protein interactions from amino acid sequences using a novel multi-scale continuous and discontinuous feature set**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S15-S9)
    * --2014--Rui Henriques--BMC--**BicSPAM: flexible biclustering using sequential patterns**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-130)
    * --2014--Daniel Arend--BMC--**e!DAL - a framework to store, share and publish research data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-214)
    * --2014--Syed Saiden Abbas--BMC--**A comparative study of cell classifiers for image-based high-throughput screening**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-342)
    * --2014--Koen Van der Borght--BMC--**Multi-model inference using mixed effects from a linear regression based genetic algorithm**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-88)
    * --2014--Ping Ge--BMC--**ProbeAlign: incorporating high-throughput sequencing-based structure probing information into ncRNA homology search**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S9-S15)
    * --2014--Jason M Knight--BMC--**MCMC implementation of the optimal Bayesian classifier for non-Gaussian models: model-based RNA-Seq classification**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0401-3)
    * --2014--Marc Streit--BMC--**Furby: fuzzy force-directed bicluster visualization**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S6-S4)
    * --2014--Prabina Kumar Meher--BMC--**A statistical approach for 5′ splice site prediction using short sequence motifs and without encoding sequence data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0362-6)
    * --2014--Rittika Shamsuddin--BMC--**Computational prediction of hinge axes in proteins**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S8-S2)
    * --2014--Benjamin Drinkwater--BMC--**Introducing TreeCollapse: a novel greedy algorithm to solve the cophylogeny reconstruction problem**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S16-S14)
    * --2014--Bart KM Jacobs--BMC--**Impact of variance components on reliability of absolute quantification using digital PCR**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-283)
    * --2014--Romeo Rizzi--BMC--**On the complexity of Minimum Path Cover with Subpath Constraints for multi-assembly**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S9-S5)
    * --2014--Steffen Möller--BMC--**Community-driven development for computational biology at Sprints, Hackathons and Codefests**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S14-S7)
    * --2014--Li Song--BMC--**nDNA-prot: identification of DNA-binding proteins based on unbalanced classification**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-298)
    * --2014--Shawn M Waldon--BMC--**SketchBio: a scientist’s 3D interface for molecular modeling and animation**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-334)
    * --2014--Zhijie Dong--BMC--**CRF-based models of protein surfaces improve protein-protein interaction site predictions**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-277)
    * --2014--Solon P Pissi--BMC--**MoTeX-II: structured MoTif eXtraction from large-scale datasets**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-235)
    * --2014--Prashant Shingate--BMC--**ECMIS: computational approach for the identification of hotspots at protein-protein interfaces**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-303)
    * --2014--Arthur Gilly--BMC--**TE-Tracker: systematic identification of transposition events through whole-genome resequencing**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-014-0377-z)
    * --2014--Tyler Weirick--BMC--**LacSubPred: predicting subtypes of Laccases, an important lignin metabolism-related enzyme class, using in silico approaches**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S11-S15)
    * --2014--Ruifeng Liu--BMC--**Exploiting large-scale drug-protein interaction information for computational drug repurposing**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-210)
    * --2014--Maciej Rybinski--BMC--**Calculating semantic relatedness for biomedical use in a knowledge-poor environment**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S14-S2)
    * --2014--Tuan Hue Thi--BMC--**Recognizing flu-like symptoms from videos**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-300)
    * --2014--Jintao Meng--BMC--**SWAP-Assembler: scalable and efficient genome assembly towards thousands of cores**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-15-S9-S2)
    ## 2013
    * --2013--Yijie Wang--BMC--**A novel subgradient-based optimization algorithm for blockmodel functional module identification**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S2-S23)
    * --2013--Jialiang Yang--BMC--**BinAligner: a heuristic method to align biological networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S14-S8)
    * --2013--Giuseppe Facchetti--BMC--**Partial inhibition and bilevel optimization in flux balance analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-344)
    * --2013--Ahsan Raja Chowdhury--BMC--**Incorporating time-delays in S-System model for reverse engineering genetic networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-196)
    * --2013--Giuseppe Agapito--BMC--**Visualization of protein interaction networks: problems and solutions**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S1-S1)
    * --2013--Yuekai Sun--BMC--**Robust flux balance analysis of multiscale biochemical reaction networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-240)
    * --2013--Yi Yang--BMC--**Differential reconstructed gene interaction networks for deriving toxicity threshold in chemical risk assessment**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S14-S3)
    * --2013--Ilana Lichtenstein--BMC--**Active subnetwork recovery with a mechanism-dependent scoring function; with application to angiogenesis and organogenesis studies**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-59)
    * --2013--Nikolaos Berntenis--BMC--**Detection of attractors of large Boolean networks via exhaustive enumeration of appropriate subspaces of the state space**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-361)
    * --2013--Daniela Stojanova--BMC--**Using PPI network autocorrelation in hierarchical multi-label classification trees for gene function prediction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-285)
    * --2013--Afshin Sadeghi--BMC--**Steiner tree methods for optimal sub-network identification: an empirical study**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-144)
    * --2013--Fabrizio F Borelli--BMC--**Gene regulatory networks inference using a multi-GPU exhaustive search algorithm**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S18-S5)
    * --2013--Shaogang Ren--BMC--**Adaptive bi-level programming for optimal gene knockouts for targeted overproduction under phenotypic constraints**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S2-S17)
    * --2013--Steven Eker--BMC--**Computing minimal nutrient sets from metabolic networks via linear constraint solving**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-114)
    * --2013--Ivan Merelli--BMC--**SNPranker 2.0: a gene-centric data mining tool for diseases associated SNP prioritization in GWAS**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S1-S9)
    * --2013--Xian Zhang--BMC--**A novel phenotypic dissimilarity method for image-based high-throughput screens**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-336)
    * --2013--Mohammed Alshalalfa--BMC--**Using context-specific effect of miRNAs to identify functional associations between miRNAs and gene signatures**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S12-S1)
    * --2013--Piyush B Madhamshettiwar--BMC--**RMaNI: Regulatory Module Network Inference framework**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S16-S14)
    * --2013--Ben Niu--BMC--**Biomimicry of quorum sensing using bacterial lifecycle model**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S8-S8)
    * --2013--Artem Sokolov--BMC--**Combining heterogeneous data sources for accurate functional annotation of proteins**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S3-S10)
    * --2013--Dongdong Lin--BMC--**Group sparse canonical correlation analysis for genomic data integration**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-245)
    * --2013--Vladimir Sarpe--BMC--**Simulating the decentralized processes of the human immune system in a virtual anatomy model**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S6-S2)
    * --2013--Karan Uppal--BMC--**xMSanalyzer: automated pipeline for improved feature detection and downstream analysis of large-scale, non-targeted metabolomics data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-15)
    * --2013--Noah Berlow--BMC--**A new approach for prediction of tumor sensitivity to targeted drugs based on functional data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-239)
    * --2013--Liang Lan--BMC--**MS-k NN: protein function prediction by integrating multiple data sources**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S3-S8)
    * --2013--Jin-Xing Liu--BMC--**Robust PCA based method for discovering differentially expressed genes**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S8-S3)
    * --2013--Jorge Alberto Jaramillo-Garzón--BMC--**Predictability of gene ontology slim-terms from primary structure information in Embryophyta plant proteins**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-68)
    * --2013--John C Earls--BMC--**AUREA: an open-source software system for accurate and user-friendly identification of relative expression molecular signatures**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-78)
    * --2013--John O Woods--BMC--**Prediction of gene-phenotype associations in humans, mice, and plants using phenologs**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-203)
    * --2013--Tobias Hamp--BMC--**Homology-based inference sets the bar high for protein function prediction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S3-S7)
    * --2013--Takanori Ueda--BMC--**Estimate hidden dynamic profiles of siRNA effect on apoptosis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-97)
    * --2013--Sepideh Babaei--BMC--**Detecting recurrent gene mutation in interaction network context using multi-scale graph diffusion**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-29)
    * --2013--Brian J Schmidt--BMC--**Alternate virtual populations elucidate the type I interferon signature predictive of the response to rituximab in rheumatoid arthritis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-221)
    * --2013--Alex--BMC--**A novel min-cost flow method for estimating transcript expression with RNA-Seq**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S5-S15)
    * --2013--Nima Aghaeepour--BMC--**Ensemble-based prediction of RNA secondary structures**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-139)
    * --2013--Salma Jamal--BMC--**Cheminformatic models based on machine learning for pyruvate kinase inhibitors of Leishmania mexicana**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-329)
    * --2013--Bing Wang--BMC--**Prediction of peptide drift time in ion mobility mass spectrometry from sequence-based features**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S8-S9)
    * --2013--Heinz Koeppl--BMC--**Mapping behavioral specifications to model parameters in synthetic biology**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S10-S9)
    * --2013--Bhanu Rekapalli--BMC--**PoPLAR: Portal for Petascale Lifescience Applications and Research**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S9-S3)
    * --2013--Qian Sun--BMC--**Image-level and group-level models for Drosophilagene expression pattern annotation**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-350)
    * --2013--Jacob Biesinger--BMC--**Discovering and mapping chromatin states using a tree hidden Markov model**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S5-S4)
    * --2013--Florent Baty--BMC--**Exploring the transcription factor activity in high-throughput gene expression data using RLQ analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-178)
    * --2013--Yaping Fang--BMC--**Identification of properties important to protein aggregation using feature selection**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-314)
    * --2013--Cameron Cotten--BMC--**Mechanistic analysis of multi-omics datasets to generate kinetic parameters for constraint-based metabolic models**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-32)
    * --2013--Iain H Moal--BMC--**The scoring of poses in protein-protein docking: current capabilities and future directions**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-286)
    * --2013--Ying-Tsang Lo--BMC--**Prediction of conformational epitopes with the use of a knowledge-based energy function and geometrically related neighboring residue characteristics**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S4-S3)
    * --2013--Sushmita Paul--BMC--**μHEM for identification of differentially expressed miRNAs using hypercuboid equivalence partition matrix**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-266)
    * --2013--François Van Lishout--BMC--**An efficient algorithm to perform multiple testing in epistasis screening**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-138)
    * --2013--James WJ Anderson--BMC--**Quantifying variances in comparative RNA secondary structure prediction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-149)
    * --2013--Wen Zou--BMC--**Data mining tools for Salmonella characterization: application to gel-based fingerprinting analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-S14-S15)
    * --2013--Malgorzata Kotulska--BMC--**On the amyloid datasets used for training PAFIG ­ how (not) to extend the experimental dataset of hexapeptides**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-14-351)
    ## 2012
    * --2012--Nguyen Xuan Vinh--BMC--**Gene regulatory network modeling via global optimization of high-order dynamic Bayesian network**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-131)
    * --2012--Anton Miró--BMC--**Deterministic global optimization algorithm based on outer approximation for the parameter estimation of nonlinear dynamic biological systems**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-90)
    * --2012--Boon-Siew Seah--BMC--**FUSE: a profit maximization approach for functional summarization of biological networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S3-S10)
    * --2012--Yu-Keng Shih--BMC--**Scalable global alignment for multiple biological networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S3-S11)
    * --2012--Wen-Chieh Chang--BMC--**Exploring biological interaction networks with tailored weighted quasi-bicliques**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S10-S16)
    * --2012--Lauri Eronen--BMC--**Biomine: predicting links between biological entities using network models of heterogeneous databases**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-119)
    * --2012--Yu-Ting Hsiao--BMC--**Inferring robust gene networks from expression data by a sensitivity-based incremental evolution method**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S7-S8)
    * --2012--Ye Deng--BMC--**Molecular ecological network analyses**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-113)
    * --2012--Bradley M Broom--BMC--**Model averaging strategies for structure learning in Bayesian networks with limited data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S13-S10)
    * --2012--Günhan Gülsoy--BMC--**HIDEN: Hierarchical decomposition of regulatory networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-250)
    * --2012--Mark K Transtrum--BMC--**Optimal experiment selection for parameter estimation in biological differential equation models**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-181)
    * --2012--Steven M Hill--BMC--**Integrating biological knowledge into variable selection: an empirical Bayes approach with an application in cancer biology**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-94)
    * --2012--Abdelhalim Larhlimi--BMC--**F2C2: a fast tool for the computation of flux coupling in genome-scale metabolic networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-57)
    * --2012--Ying-Ke Lei--BMC--**Assessing and predicting protein interactions by combining manifold embedding with multiple information integration**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S7-S3)
    * --2012--Atanas Kamburov--BMC--**Cluster-based assessment of protein-protein interaction confidence**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-262)
    * --2012--Benjamin A Logsdon--BMC--**Mouse obesity network reconstruction with a variational Bayes algorithm to employ aggressive false positive control**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-53)
    * --2012--Yong Fuga Li--BMC--**Computational approaches to protein inference in shotgun proteomics**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S16-S4)
    * --2012--Xi-Jun Liang--BMC--**Inference of gene regulatory subnetworks from time course gene expression data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S9-S3)
    * --2012--Jeremy J Jay--BMC--**A systematic comparison of genome-scale clustering algorithms**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S10-S7)
    * --2012--Tianlei Xu--BMC--**Quantitatively integrating molecular structure and bioactivity profile evidence into drug-target relationship analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-75)
    * --2012--Ruoting Yang--BMC--**Core module biomarker identification with network exploration for breast cancer metastasis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-12)
    * --2012--Duojiao Wu--BMC--**Cancer bioinformatics: A new approach to systems clinical medicine**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-71)
    * --2012--H Robert Frost--BMC--**Markov Chain Ontology Analysis (MCOA)**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-23)
    * --2012--Jonathan Q Jiang--BMC--**Predicting multiplex subcellular localization of proteins using protein-protein interaction network: a comparative study**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S10-S20)
    * --2012--Jun Gao--BMC--**Integrated QSAR study for inhibitors of hedgehog signal pathway against multiple cell lines:a collaborative filtering method**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-186)
    * --2012--Abdelaali Briache--BMC--**Transparent mediation-based access to multiple yeast data sources using an ontology driven interface**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S1-S7)
    * --2012--Muskan Kukreja--BMC--**Comparative study of classification algorithms for immunosignaturing data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-139)
    * --2012--Andrew T Magis--BMC--**The top-scoring ‘N’ algorithm: a generalized relative expression classification method from small numbers of biomolecules**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-227)
    * --2012--John D Eblen--BMC--**The maximum clique enumeration problem: algorithms, applications, and implementations**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S10-S5)
    * --2012--Je-Gun Joung--BMC--**A probabilistic coevolutionary biclustering algorithm for discovering coherent patterns in gene expression dataset**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S17-S12)
    * --2012--Shu-Lin Wang--BMC--**Finding minimum gene subsets with heuristic breadth-first search algorithm for robust tumor classification**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-178)
    * --2012--Andrew F Neuwald--BMC--**Automated hierarchical classification of protein domain subfamilies based on functionally-divergent residue signatures**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-144)
    * --2012--Carlo Bianca--BMC--**Mathematical modeling of the immune system recognition to mammary carcinoma antigen**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S17-S21)
    * --2012--Jeffrey T Leek--BMC--**A statistical approach to selecting and confirming validation targets in -omics experiments**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-150)
    * --2012--Susanna Zucca--BMC--**Characterization of an inducible promoter in different DNA copy number conditions**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S4-S11)
    * --2012--Hua Tan--BMC--**A Computational model for compressed sensing RNAi cellular screening**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-337)
    * --2012--Bernie J Daigle Jr--BMC--**Accelerated maximum likelihood parameter estimation for stochastic biochemical systems**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-68)
    * --2012--Sean O'Callaghan--BMC--**PyMS: a Python toolkit for processing of gas chromatography-mass spectrometry (GC-MS) data. Application and comparative study of selected tools**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-115)
    * --2012--Lakshmi Manohar Akella--BMC--**NetiNeti: discovery of scientific names from text using machine learning methods**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-211)
    * --2012--Piotr Dittwald--BMC--**Inferring serum proteolytic activity from LC-MS/MS data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S5-S7)
    * --2012--Lei Yuan--BMC--**Learning Sparse Representations for Fruit-Fly Gene Expression Pattern Image Annotation and Retrieval**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-107)
    * --2012--Ruihua Fang--BMC--**Automatic categorization of diverse experimental information in the bioscience literature**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-16)
    * --2012--Irene Sánchez-Linares--BMC--**High-Throughput parallel blind Virtual Screening using BINDSURF**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S14-S13)
    * --2012--Rasna R Walia--BMC--**Protein-RNA interface residue prediction using machine learning: an assessment of the state of the art**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-89)
    * --2012--Andrea Cornero--BMC--**Design of a multi-signature ensemble classifier predicting neuroblastoma patients' outcome**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S4-S13)
    * --2012--Alex--BMC--**PHYLOViZ: phylogenetic inference and data visualization for sequence based typing methods**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-87)
    * --2012--Howard J Feldman--BMC--**Identifying structural domains of proteins using clustering**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-286)
    * --2012--Lukasz P Kozlowski--BMC--**MetaDisorder: a meta-server for the prediction of intrinsic disorder in proteins**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-111)
    * --2012--Riccardo Bellazzi--BMC--**Clinical Bioinformatics: challenges and opportunities**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S14-S1)
    * --2012--Geet Duggal--BMC--**Graph rigidity reveals well-constrained regions of chromosome conformation embeddings**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-241)
    * --2012--Akhil Kumar--BMC--**MetRxn: a knowledgebase of metabolites and reactions spanning metabolic models and databases**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-6)
    * --2012--Arjun Ray--BMC--**Improved model quality assessment using ProQ2**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-224)
    * --2012--B Jayaram--BMC--**Sanjeevini: a freely accessible web-server for target directed lead molecule discovery**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S17-S7)
    * --2012--Yanni Sun--BMC--**Genome-scale NCRNA homology search using a Hamming distance-based filtration strategy**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S3-S12)
    * --2012--Axel Skarman--BMC--**A Bayesian variable selection procedure to rank overlapping gene sets**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-73)
    * --2012--Gad Abraham--BMC--**SparSNP: Fast and memory-efficient analysis of all SNPs for phenotype prediction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-88)
    * --2012--Daniel S St--BMC--**ParsEval: parallel comparison and analysis of gene structure annotations**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-187)
    * --2012--Mahmoud ElHefnawi--BMC--**Multiple virtual screening approaches for finding new Hepatitis c virus RNA-dependent RNA polymerase inhibitors: Structure-based screens and molecular dynamics for the pursue of new poly pharmacological inhibitors**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S17-S5)
    * --2012--Steven Vercruysse--BMC--**OLSVis: an animated, interactive visual browser for bio-ontologies**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-116)
    * --2012--Mohamed El-Kalioby--BMC--**Personalized cloud-based bioinformatics services for research and education: use cases and the elasticHPC package**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S17-S22)
    * --2012--Alej--BMC--**Federated ontology-based queries over cancer data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S1-S9)
    * --2012--Michele Pinelli--BMC--**Simulating gene-gene and gene-environment interactions in complex diseases: Gene-Environment iNteraction Simulator 2**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-132)
    * --2012--Mohameth-François Sy--BMC--**User centered and ontology based information retrieval system for life sciences**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-S1-S4)
    * --2012--Ching-Wei Wang--BMC--**Fast automatic quantitative cell replication with fluorescent live cell imaging**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-21)
    * --2012--Bin Li--BMC--**Protein docking prediction using predicted protein-protein interface**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-7)
    ## 2011
    * --2011--Byung-Jun Yoon--BMC--**Enhanced stochastic optimization algorithm for finding effective multi-target therapeutics**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-S1-S18)
    * --2011--Peter Sykacek--BMC--**The impact of quantitative optimization of hybridization conditions on gene expression analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-73)
    * --2011--Mi-Youn K Brusniak--BMC--**ATAQS: A computational software tool for high throughput transition optimization and validation for selected reaction monitoring mass spectrometry**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-78)
    * --2011--Kyle C Chipman--BMC--**Using Stochastic Causal Trees to Augment Bayesian Networks for Modeling eQTL Datasets**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-7)
    * --2011--Sang-Kyu Jung--BMC--**Visual gene developer: a fully programmable bioinformatics software for synthetic gene optimization**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-340)
    * --2011--Cristian A Gallo--BMC--**Discovering time-lagged rules from microarray data using gene profile classifiers**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-123)
    * --2011--Christian Bender--BMC--**Inferring signalling networks from longitudinal data using sampling based approaches in the R-package 'ddepn'**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-291)
    * --2011--Kahn Rhrissorrakrai--BMC--**MINE: Module Identification in Networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-192)
    * --2011--Xiaofei Nan--BMC--**Leveraging domain information to restructure biological prediction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-S10-S22)
    * --2011--Zafer Aydin--BMC--**Learning sparse models for a dynamic Bayesian network classifier of protein secondary structure**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-154)
    * --2011--Giuseppe Gallone--BMC--**Bio::Homology::InterologWalk - A Perl module to build putative protein-protein interaction networks through interolog mapping**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-289)
    * --2011--Weichen Wang--BMC--**Network-based group variable selection for detecting expression quantitative trait loci (eQTL)**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-269)
    * --2011--Yao Fu--BMC--**Reconstructing genome-wide regulatory network of E. coli using transcriptome data and predicted transcription factor activities**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-233)
    * --2011--Kai Wang--BMC--**CASCADE_SCAN: mining signal transduction network from high-throughput data based on steepest descent method**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-164)
    * --2011--Kim-Anh Lê Cao--BMC--**Sparse PLS discriminant analysis: biologically relevant feature selection and graphical displays for multiclass problems**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-253)
    * --2011--Hadas Birin--BMC--**Efficient algorithms for reconstructing gene content by co-evolution**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-S9-S12)
    * --2011--Yongjin Park--BMC--**Resolving the structure of interactomes with hierarchical agglomerative clustering**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-S1-S44)
    * --2011--Elon Correa--BMC--**A genetic algorithm-Bayesian network approach for the analysis of metabolomics and spectroscopic data: application to the rapid identification of Bacillus spores and classification of Bacillus species**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-33)
    * --2011--Rosemary Braun--BMC--**Partition decoupling for multi-gene analysis of gene expression profiling data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-497)
    * --2011--Xianwen Ren--BMC--**Improving accuracy of protein-protein interaction prediction by considering the converse problem for sequence representation**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-409)
    * --2011--Domenico Fraccalvieri--BMC--**Conformational and functional analysis of molecular dynamics trajectories by Self-Organising Maps**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-158)
    * --2011--Jiamin Xiao--BMC--**Identification of microRNA precursors based on random forest with network-level representation method of stem-loop structure**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-165)
    * --2011--Gustavo Arellano--BMC--**"Antelope": a hybrid-logic model checker for branching-time Boolean GRN analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-490)
    * --2011--Edouard Pauwels--BMC--**Predicting drug side-effect profiles: a chemical fragment-based approach**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-169)
    * --2011--Sangaalofa T Clark--BMC--**A systems approach to identifying correlated gene targets for the loss of colour pigmentation in plants**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-343)
    * --2011--Ozgur Demir-Kavuk--BMC--**Prediction using step-wise L1, L2 regularization and feature selection for small data sets with large number of features**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-412)
    * --2011--Daniel Restrepo-Montoya--BMC--**NClassG+: A classifier for non-classically secreted Gram-positive bacterial proteins**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-21)
    * --2011--Jan Hasenauer--BMC--**Identification of models of heterogeneous cell populations from population snapshot data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-125)
    * --2011--Patrick W Sheppard--BMC--**Quantitative characterization and analysis of the dynamic NF-κB response in microglia**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-276)
    * --2011--Martin Krallinger--BMC--**The Protein-Protein Interaction tasks of BioCreative III: classification/ranking of articles and linking bio-ontology concepts to full text**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-S8-S3)
    * --2011--Bailing Zhang--BMC--**Phenotype Recognition with Combined Features and Random Subspace Classifier Ensemble**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-128)
    * --2011--George D Kritikos--BMC--**Noise reduction in protein-protein interaction graphs by the implementation of a novel weighting scheme**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-239)
    * --2011--Alfredo Benso--BMC--**Building gene expression profile classifiers with a simple and efficient rejection option in R**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-S13-S3)
    * --2011--Marcin Cieślik--BMC--**A lightweight, flow-based toolkit for parallel and distributed bioinformatics pipelines**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-61)
    * --2011--Eduard Schreiner--BMC--**Stereochemical errors and their implications for molecular dynamics simulations**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-190)
    * --2011--Usha K Muppirala--BMC--**Predicting RNA-Protein Interactions Using Only Sequence Information**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-489)
    * --2011--Olga Domanova--BMC--**Toponomics method for the automated quantification of membrane protein translocation**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-370)
    * --2011--Sorayya Malek--BMC--**Assessment of predictive models for chlorophyll-a concentration of a tropical lake**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-S13-S12)
    * --2011--Arturo H Ariño--BMC--**The Biodiversity Informatics Potential Index**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-S15-S4)
    * --2011--Ubbo Visser--BMC--**BioAssay Ontology (BAO): a semantic description of bioassays and high-throughput screening results**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-257)
    * --2011--Ping Shi--BMC--**Top scoring pairs for feature selection in machine learning and applications to cancer outcome prediction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-375)
    * --2011--Mark Needham--BMC--**Hierarchical Parallelization of Gene Differential Association Analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-374)
    * --2011--Ivica Kopriva--BMC--**A mixture model with a reference-based automatic selection of components for disease classification from protein and/or gene expression levels**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-496)
    * --2011--Richard Tzong-Han Tsai--BMC--**Dynamic programming re-ranking for PPI interactor and pair extraction in full-text articles**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-60)
    * --2011--Alfredo A Kalaitzis--BMC--**A Simple Approach to Ranking Differentially Expressed Gene Expression Time Courses through Gaussian Process Regression**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-180)
    * --2011--Hui Zhao--BMC--**Query-based biclustering of gene expression data using Probabilistic Relational Models**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-S1-S37)
    * --2011--Fei Xia--BMC--**FPGA accelerator for protein secondary structure prediction based on the GOR algorithm**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-S1-S5)
    * --2011--Suyu Mei--BMC--**Gene ontology based transfer learning for protein subcellular localization**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-44)
    * --2011--Varun Khanna--BMC--**In silico approach to screen compounds active against parasitic nematodes of major socio-economic importance**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-S13-S25)
    * --2011--Henry Wirth--BMC--**Expression cartography of human tissues using self organizing maps**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-306)
    * --2011--Kailin Tang--BMC--**Discrimination of approved drugs from experimental drugs by learning methods**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-157)
    * --2011--Hiroki Takahashi--BMC--**AMDORAP: Non-targeted metabolic profiling based on high-resolution LC-MS**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-259)
    * --2011--Paul Thompson--BMC--**The BioLexicon: a large-scale terminological resource for biomedical text mining**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-397)
    * --2011--Junliang Shang--BMC--**Performance analysis of novel methods for detecting epistasis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-475)
    * --2011--Kathleen M Doherty--BMC--**A multifaceted analysis of HIV-1 protease multidrug resistance phenotypes**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-477)
    * --2011--Yulong Wang--BMC--**Determinants of antigenicity and specificity in immune response for protein sequences**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-251)
    * --2011--Jason Rudy--BMC--**Empirical comparison of cross-platform normalization methods for gene expression data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-467)
    * --2011--Stephan Gade--BMC--**Graph based fusion of miRNA and mRNA expression data improves clinical outcome prediction in prostate cancer**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-488)
    * --2011--Sabry Razick--BMC--**iRefScape. A Cytoscape plug-in for visualization and data mining of protein interaction data from iRefIndex**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-388)
    * --2011--Yves-Pol Deniélou--BMC--**Bacterial syntenies: an exact approach with gene quorum**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-193)
    * --2011--Andre J Faure--BMC--**Investigating the effect of paralogs on microarray gene-set analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-29)
    * --2011--Natalia Becker--BMC--**Elastic SCAD as a novel penalization method for SVM classification tasks in high-dimensional data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-138)
    * --2011--Ruoying Chen--BMC--**Rigorous assessment and integration of the sequence and structure based features to predict hot spots**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-311)
    * --2011--Masaaki Kotera--BMC--**MUCHA: multiple chemical alignment algorithm to identify building block substructures of orphan secondary metabolites**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-S14-S1)
    * --2011--Satya S Sahoo--BMC--**A unified framework for managing provenance information in translational research**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-461)
    * --2011--Marcus Lechner--BMC--**Proteinortho: Detection of (Co-)orthologs in large-scale analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-124)
    * --2011--Ernesto Iacucci--BMC--**Predicting receptor-ligand pairs through kernel learning**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-336)
    * --2011--Ren-Xiang Yan--BMC--**Outer membrane proteins can be simply identified using secondary structure element alignment**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-76)
    * --2011--Samuel C Flores--BMC--**Predicting protein ligand binding motions with the conformation explorer**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-417)
    * --2011--Irina Tuszynska--BMC--**DARS-RNP and QUASI-RNP: New statistical potentials for protein-RNA docking**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-348)
    * --2011--Paweł Daniluk--BMC--**A novel method to compare protein structures using local descriptors**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-344)
    * --2011--Yoshiharu Sato--BMC--**Meta-analytic approach to the accurate prediction of secreted virulence effectors in gram-negative bacteria**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-442)
    * --2011--Makiko Yoshida--BMC--**SNPInterForest: A new method for detecting epistatic interactions**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-469)
    * --2011--Jonathon T Hill--BMC--**Novel computational analysis of protein binding array data identifies direct targets of Nkx2.2 in the pancreas**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-62)
    * --2011--Xiaosheng Wang--BMC--**Microarray-based cancer prediction using single genes**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-391)
    * --2011--Jacek Blazewicz--BMC--**Protein alignment algorithms with an efficient backtracking routine on multiple GPUs**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-181)
    ## 2010
    * --2010--Miha Štajdohar--BMC--**FragViz: visualization of fragmented networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-475)
    * --2010--Martin T Swain--BMC--**Comparative study of three commonly used continuous deterministic methods for modeling gene regulation networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-459)
    * --2010--Sheng He--BMC--**LucidDraw: Efficiently visualizing complex biochemical networks within MATLAB**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-31)
    * --2010--Ioannis A Maraziotis--BMC--**Gene regulatory networks modelling using a dynamic evolutionary hybrid**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-140)
    * --2010--Mara L Hartsperger--BMC--**Structuring heterogeneous biological information using fuzzy clustering of k-partite graphs**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-522)
    * --2010--Kyoung Ae Kim--BMC--**Systematic calibration of a cell signaling network model**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-202)
    * --2010--Kaname Kojima--BMC--**An efficient biological pathway layout algorithm combining grid-layout and spring embedder for complicated cellular location information**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-335)
    * --2010--Linh M Tran--BMC--**Trimming of mammalian transcriptional networks using network component analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-511)
    * --2010--Jongkwang Kim--BMC--**Discover Protein Complexes in Protein-Protein Interaction Networks Using Parametric Local Modularity**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-521)
    * --2010--Adel Dayarian--BMC--**SOPRA: Scaffolding algorithm for paired reads via statistical optimization**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-345)
    * --2010--Chen Wang--BMC--**Knowledge-guided gene ranking by coordinative component analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-162)
    * --2010--Min Wu--BMC--**Integrating diverse biological and computational sources for reliable protein-protein interactions**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-S7-S8)
    * --2010--Mitra Kabir--BMC--**Reverse engineering gene regulatory network from microarray data using linear time-variant model**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-S1-S56)
    * --2010--Alina Sîrbu--BMC--**Comparison of evolutionary algorithms in gene regulatory network model inference**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-59)
    * --2010--Alex--BMC--**BisoGenet: a new tool for gene network building, visualization and analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-91)
    * --2010--S Yahya Anvar--BMC--**The identification of informative genes from multiple datasets with increasing complexity**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-32)
    * --2010--Suresh Kumar Poovathingal--BMC--**Global parameter estimation methods for stochastic biochemical systems**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-414)
    * --2010--Pingzhao Hu--BMC--**Predicting protein functions by relaxation labelling protein interaction network**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-S1-S64)
    * --2010--Anup Parikh--BMC--**New components of the Dictyostelium PKA pathway revealed by Bayesian analysis of expression data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-163)
    * --2010--Corban G Rivera--BMC--**NeMo: Network Module identification in Cytoscape**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-S1-S61)
    * --2010--Subhadip Basu--BMC--**AMS 3.0: prediction of post-translational modifications**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-210)
    * --2010--Steinn Gudmundsson--BMC--**Computationally efficient flux variability analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-489)
    * --2010--Raffaele Di Natale--BMC--**SING: Subgraph search In Non-homogeneous Graphs**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-96)
    * --2010--Shi Yu--BMC--**Gene prioritization and clustering by multi-view text mining**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-28)
    * --2010--Anil Aswani--BMC--**Nonparametric identification of regulatory interactions from spatial and temporal gene expression data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-413)
    * --2010--Jens Keilwagen--BMC--**Apples and oranges: avoiding different priors in Bayesian DNA sequence analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-149)
    * --2010--Zhong-Ru Xie--BMC--**An interaction-motif-based scoring function for protein-ligand docking**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-298)
    * --2010--Yu-Qing Qiu--BMC--**Detecting disease associated modules and prioritizing active genes based on high throughput data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-26)
    * --2010--Martin Klammer--BMC--**Identifying differentially regulated subnetworks from phosphoproteomic data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-351)
    * --2010--Junbai Wang--BMC--**Quantitative model for inferring dynamic regulation of the tumour suppressor gene p53**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-36)
    * --2010--Benjamin Misselwitz--BMC--**Enhanced CellClassifier: a multi-class classification tool for microscopy images**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-30)
    * --2010--Tianwei Y--BMC--**An exploratory data analysis method to reveal modular latent structures in high-throughput data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-440)
    * --2010--Vinod Ch--BMC--**MTar: a computational microRNA target prediction architecture for human transcriptome**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-S1-S2)
    * --2010--Shirley Hui--BMC--**Proteome scanning to predict PDZ domain interactions using support vector machines**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-507)
    * --2010--Olof Görnerup--BMC--**A model-independent approach to infer hierarchical codon substitution dynamics**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-201)
    * --2010--Ioannis K Valavanis--BMC--**A multifactorial analysis of obesity as CVD risk factor: Use of neural network based methods in a nutrigenetics context**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-453)
    * --2010--Susmita Datta--BMC--**An adaptive optimal ensemble classifier via bagging and rank aggregation with applications to high dimensional data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-427)
    * --2010--Shi Yu--BMC--**L2-norm multiple kernel learning and its application to biomedical data fusion**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-309)
    * --2010--Garrett Jenkinson--BMC--**Thermodynamically consistent Bayesian analysis of closed biochemical reaction systems**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-547)
    * --2010--Florian Battke--BMC--**Mayday - integrative analytics for expression data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-121)
    * --2010--Jérôme Gracy--BMC--**Optimizing structural modeling for a specific protein scaffold: knottins or inhibitor cystine knots**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-535)
    * --2010--Melissa J Morine--BMC--**Bi-directional gene set enrichment and canonical correlation analysis identify key diet-sensitive pathways and biomarkers of metabolic syndrome**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-499)
    * --2010--Nobuyuki Uchikoga--BMC--**Analysis of protein-protein docking decoys using interaction fingerprints: application to the reconstruction of CaM-ligand complexes**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-236)
    * --2010--Xiaojiang Xu--BMC--**Application of machine learning methods to histone methylation ChIP-Seq data reveals H4R3me2 globally represses gene expression**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-396)
    * --2010--Daren Lee--BMC--**MBAT: A scalable informatics system for unifying digital atlasing workflows**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-608)
    * --2010--Yosuke Ozawa--BMC--**Protein complex prediction via verifying and reconstructing the topology of domain-domain interactions**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-350)
    * --2010--Ilenia Giangreco--BMC--**Analysis of X-ray Structures of Matrix Metalloproteinases via Chaotic Map Clustering**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-500)
    * --2010--Marzio Pennisi--BMC--**Modeling the competition between lung metastases and the immune system using agents**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-S7-S13)
    * --2010--Pekka Ruusuvuori--BMC--**Evaluation of methods for detection of fluorescence labeled subcellular objects in microscope images**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-248)
    * --2010--Jan Schellenberger--BMC--**BiGG: a Biochemical Genetic and Genomic knowledgebase of large scale metabolic reconstructions**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-213)
    * --2010--Lisbeth Carstensen--BMC--**Multivariate Hawkes process models of the occurrence of regulatory elements**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-456)
    * --2010--Kenny Daily--BMC--**Data structures and compression algorithms for high-throughput sequencing technologies**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-514)
    * --2010--Feng-Jie Sun--BMC--**The ancient history of the structure of ribonuclease P and the early origins of Archaea**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-153)
    * --2010--Darby Tien-Hao Chang--BMC--**Predicting the protein-protein interactions using primary structures with predicted protein surface**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-S1-S3)
    * --2010--István P Sugár--BMC--**Misty Mountain clustering: application to fast unsupervised flow cytometry gating**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-502)
    * --2010--Prashanti M--BMC--**GOModeler- A tool for hypothesis-testing of functional genomics datasets**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-S6-S29)
    * --2010--Jens Keilwagen--BMC--**Unifying generative and discriminative learning principles**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-98)
    * --2010--Mirva Piippo--BMC--**Pripper: prediction of caspase cleavage sites from whole proteomes**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-320)
    * --2010--SPT Krishnan--BMC--**Towards high performance computing for molecular structure prediction using IBM Cell Broadband Engine - an implementation perspective**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-S1-S36)
    * --2010--Renaud Gaujoux--BMC--**A flexible R package for nonnegative matrix factorization**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-367)
    * --2010--Jia Zeng--BMC--**Multiconstrained gene clustering based on generalized projections**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-164)
    * --2010--Ernst Althaus--BMC--**Computing H/D-Exchange rates of single residues from data of proteolytic fragments**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-424)
    * --2010--Matias Piipari--BMC--**Metamotifs - a generative model for building families of nucleotide position weight matrices**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-348)
    * --2010--Petr Novák--BMC--**Graph-based clustering and characterization of repetitive sequences in next-generation sequencing data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-378)
    * --2010--Jose M Duarte--BMC--**Optimal contact definition for reconstruction of Contact Maps**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-283)
    * --2010--Pengyi Yang--BMC--**A genetic ensemble approach for gene-gene interaction identification**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-524)
    * --2010--Patricia Francis-Lyon--BMC--**Sampling the conformation of protein surface residues for flexible protein docking**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-575)
    * --2010--Aless--BMC--**Structural alphabets derived from attractors in conformational space**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-97)
    * --2010--Alex D Stivala--BMC--**Fast and accurate protein substructure searching with simulated annealing and GPUs**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-446)
    * --2010--Sebastien Tempel--BMC--**ModuleOrganizer: detecting modules in families of transposable elements**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-474)
    * --2010--David-Olivier D Azulay--BMC--**Visualisation tool for peptide fractionation data in proteomics: application to OFFGEL isoelectric focussing**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-371)
    * --2010--Russell L Zaretzki--BMC--**Bias correction and Bayesian analysis of aggregate counts in SAGE libraries**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-72)
    * --2010--Zhanchao Li--BMC--**Classification of G-protein coupled receptors based on support vector machine with maximum relevance minimum redundancy and genetic algorithm**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-325)
    * --2010--Christoph Bartenhagen--BMC--**Comparative study of unsupervised dimension reduction techniques for the visualization of microarray gene expression data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-567)
    * --2010--Russell Schwartz--BMC--**Applying unmixing to gene expression data for tumor phylogeny inference**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-42)
    * --2010--Cheng-Hong Yang--BMC--**Confronting two-pair primer design for enzyme-free SNP genotyping based on a genetic algorithm**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-509)
    * --2010--Márton Münz--BMC--**Dynamics based alignment of proteins: an alternative approach to quantify dynamic similarity**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-188)
    * --2010--Martin Mechelke--BMC--**Robust probabilistic superposition and comparison of protein structures**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-363)
    * --2010--John C Dittmar--BMC--**ScreenMill: A freely available software suite for growth measurement, analysis and visualization of high-throughput screen data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-353)
    * --2010--Andrew K Rider--BMC--**A statistical approach to finding overlooked genetic associations**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-526)
    * --2010--Ashis Kumer Biswas--BMC--**Machine learning approach to predict protein phosphorylation sites by incorporating evolutionary information**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-273)
    * --2010--Karel Zimmermann--BMC--**Amino acid "little Big Bang": Representing amino acid substitution matrices as dot products of Euclidian vectors**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-4)
    * --2010--Mariusz Popenda--BMC--**RNA FRABASE 2.0: an advanced web-accessible database with the capacity to search the three-dimensional fragments within RNA structures**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-11-231)
    ## 2009
    * --2009--Steffen Brasch--BMC--**VANLO - Interactive visual exploration of aligned biological networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-327)
    * --2009--Falk Schreiber--BMC--**A generic algorithm for layout of biological networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-375)
    * --2009--Franck Picard--BMC--**Deciphering the connectivity structure of biological networks using MixNet**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S6-S17)
    * --2009--Tatsunori B Hashimoto--BMC--**BFL: a node and edge betweenness based fast layout algorithm for large scale networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-19)
    * --2009--James Anderson--BMC--**On validation and invalidation of biological models**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-132)
    * --2009--Tomáš Hirmajer--BMC--**DOTcvpSB, a software toolbox for dynamic optimization in systems biology**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-199)
    * --2009--Xinan Yang--BMC--**Mechanism-anchored profiling derived from epigenetic networks predicts outcome in acute lymphoblastic leukemia**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S9-S6)
    * --2009--David J Klinke II--BMC--**An empirical Bayesian approach for model-based inference of cellular signaling networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-371)
    * --2009--George Zheng--BMC--**Service-based analysis of biological pathways**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S10-S6)
    * --2009--Kim-Anh Lê Cao--BMC--**Sparse canonical methods for biological data integration: application to a cross-platform study**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-34)
    * --2009--Feng Luo--BMC--**Core and periphery structures in protein interaction networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S4-S8)
    * --2009--Gonzalo Guillén-Gosálbez--BMC--**Identifying quantitative operation principles in metabolic pathways: a systematic method for searching feasible enzyme activity patterns leading to cellular adaptive responses**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-386)
    * --2009--Holger Fröhlich--BMC--**Deterministic Effects Propagation Networks for reconstructing protein signaling networks from multiple interventions**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-322)
    * --2009--Rui-Sheng Wang--BMC--**Modeling post-transcriptional regulation activity of small non-coding RNAs in Escherichia coli**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S4-S6)
    * --2009--Steve Pettifer--BMC--**Visualising biological data: a semantic approach to tool and database integration**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S6-S19)
    * --2009--Johanna Mazur--BMC--**Reconstructing nonlinear dynamic models of gene regulation using stochastic sampling**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-448)
    * --2009--Gunnar W Kla--BMC--**A new graph-based method for pairwise global network alignment**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S1-S59)
    * --2009--Yanni Zhu--BMC--**Network-based support vector machine for classification of microarray samples**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S1-S21)
    * --2009--James Vlasblom--BMC--**Markov clustering versus affinity propagation for the partitioning of protein interaction graphs**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-99)
    * --2009--Ujjwal Maulik--BMC--**Combining Pareto-optimal clusters using supervised learning for identifying co-expressed genes**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-27)
    * --2009--Michael Hecker--BMC--**Integrative modeling of transcriptional regulation in response to antirheumatic therapy**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-262)
    * --2009--Luciano Milanesi--BMC--**Trends in modeling Biomedical Complex Systems**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S12-I1)
    * --2009--Zixiang Xu--BMC--**Genome-scale analysis to the impact of gene deletion on the metabolism of E. coli: constraint-based simulation approach**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S1-S62)
    * --2009--Yang Li--BMC--**designGG: an R-package and web tool for the optimal design of genetical genomics experiments**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-188)
    * --2009--Ettore Mosca--BMC--**Identification of functionally related genes using data mining and data integration: a breast cancer case study**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S12-S8)
    * --2009--Yiming Ying--BMC--**Enhanced protein fold recognition through a novel data integration approach**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-267)
    * --2009--Atsushi Niida--BMC--**Gene set-based module discovery in the breast cancer transcriptome**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-71)
    * --2009--Kevin Y Yip--BMC--**Multi-level learning: improving the prediction of protein, domain and residue interactions by allowing information flow between levels**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-241)
    * --2009--Christopher W Maier--BMC--**Ultra-Structure database design methodology for managing systems biology data and analyses**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-254)
    * --2009--Junwan Liu--BMC--**Biclustering of microarray data with MOSPO based on crowding distance**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S4-S9)
    * --2009--Iftach Nachman--BMC--**BRNI: Modular analysis of transcriptional regulatory programs**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-155)
    * --2009--Cuilan Gao--BMC--**Graph ranking for exploratory gene data analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S11-S19)
    * --2009--Emanuel Schwarz--BMC--**Clinical bioinformatics for complex disorders: a schizophrenia case study**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S12-S6)
    * --2009--Marcello Castellano--BMC--**A bioinformatics knowledge discovery in text application for grid computing**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S6-S23)
    * --2009--Filippo Menolascina--BMC--**Developing optimal input design strategies in cancer systems biology with applications to microfluidic device engineering**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S12-S4)
    * --2009--James R Green--BMC--**PCI-SS: MISO dynamic nonlinear protein secondary structure prediction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-222)
    * --2009--Chengpeng Bi--BMC--**DNA motif alignment by evolving a population of Markov chains**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S1-S13)
    * --2009--Nazar Zaki--BMC--**Protein-protein interaction based on pairwise similarity**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-150)
    * --2009--Gregor Rot--BMC--**dictyExpress: a Dictyostelium discoideum gene expression database with an explorative data analysis web-based interface**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-265)
    * --2009--Michalis Zervakis--BMC--**Outcome prediction based on microarray analysis: a critical perspective on methods**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-53)
    * --2009--Michael Dondrup--BMC--**EMMA 2 – A MAGE-compliant system for the collaborative analysis and integration of microarray data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-50)
    * --2009--Paola Bertolazzi--BMC--**Learning to classify species with barcodes**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S14-S7)
    * --2009--Jignesh R Parikh--BMC--**multiplierz: an extensible API based desktop environment for proteomics data analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-364)
    * --2009--Gernot Stocker--BMC--**iLAP: a workflow-driven software for experimental protocol development, data acquisition and analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-390)
    * --2009--Enrico Glaab--BMC--**ArrayMining: a modular web-application for microarray analysis combining ensemble and consensus methods with cross-study normalization**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-358)
    * --2009--Göran Sperber--BMC--**RetroTector online, a rational tool for analysis of retroviral elements in small and medium size vertebrate genomic sequences**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S6-S4)
    * --2009--Helena Strömbergsson--BMC--**A chemogenomics view on protein-ligand spaces**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S6-S13)
    * --2009--Erick Antezana--BMC--**BioGateway: a semantic systems biology tool for the life sciences**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S10-S11)
    * --2009--Stefan R Maetschke--BMC--**Exploiting structural and topological information to improve prediction of RNA-protein binding sites**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-341)
    * --2009--María del Mar Roldán-García--BMC--**KA-SB: from data integration to large scale reasoning**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S10-S5)
    * --2009--Wei Guan--BMC--**Ovarian cancer detection from metabolomic liquid chromatography/mass spectrometry data by support vector machines**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-259)
    * --2009--Yi Jia--BMC--**Towards comprehensive structural motif mining for better fold annotation in the "twilight zone" of sequence dissimilarity**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S1-S46)
    * --2009--An-Min Zou--BMC--**Quality assessment of tandem mass spectra using support vector machine (SVM)**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S1-S49)
    * --2009--Flavien Quintus--BMC--**Ligand scaffold hopping combining 3D maximal substructure search and molecular similarity**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-245)
    * --2009--Monica Chiogna--BMC--**A comparison on effects of normalisations in the detection of differentially expressed genes**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-61)
    * --2009--Aless--BMC--**A Perl procedure for protein identification by Peptide Mass Fingerprinting**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S12-S11)
    * --2009--Bo Jin--BMC--**Feature selection for fMRI-based deception detection**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S9-S15)
    * --2009--Fatemeh Zare-Mirakabad--BMC--**New scoring schema for finding motifs in DNA Sequences**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-93)
    * --2009--Martin H van Vliet--BMC--**Knowledge driven decomposition of tumor expression profiles**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S1-S20)
    * --2009--Lei Deng--BMC--**Prediction of protein-protein interaction sites using an ensemble method**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-426)
    * --2009--Sung Hee Park--BMC--**Prediction of protein-protein interaction types using association rule based classification**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-36)
    * --2009--Vitali Sintchenko--BMC--**Towards bioinformatics assisted infectious disease control**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S2-S10)
    * --2009--Timothy Nugent--BMC--**Transmembrane protein topology prediction using support vector machines**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-159)
    * --2009--Sebastian Schönherr--BMC--**eCOMPAGT – efficient Combination and Management of Phenotypes and Genotypes for Genetic Epidemiology**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-139)
    * --2009--Frank PY Lin--BMC--**In silico prioritisation of candidate genes for prokaryotic gene function discovery: an application of phylogenetic profiles**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-86)
    * --2009--Waranyu Wongseree--BMC--**Detecting purely epistatic multi-locus interactions by an omnibus permutation test on ensembles of two-locus analyses**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-294)
    * --2009--Anna Korhonen--BMC--**The first step in the development of text mining technology for cancer risk assessment: identifying and organizing scientific evidence in risk assessment literature**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-303)
    * --2009--Varun Khanna--BMC--**Physicochemical property space distribution among human metabolites, drugs and toxins**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S15-S10)
    * --2009--Ren-Xiang Yan--BMC--**DescFold: A web server for protein fold recognition**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-416)
    * --2009--Christian Schwager--BMC--**TableButler – a Windows based tool for processing large data tables generated with high-throughput methods**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-235)
    * --2009--Richard D Pearson--BMC--**puma: a Bioconductor package for propagating uncertainty in microarray analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-211)
    * --2009--Federica Chiappori--BMC--**In silico saturation mutagenesis and docking screening for the analysis of protein-ligand interaction: the Endothelial Protein C Receptor case study**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-S12-S3)
    * --2009--Antonis Koussounadis--BMC--**Improving classification in protein structure databases using text mining**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-129)
    ## 2008
    * --2008--Liang Sun--BMC--**Adaptive diffusion kernel learning from biological networks for protein function prediction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-162)
    * --2008--Erwin P Gianch--BMC--**Predicting biological system objectives de novo from internal state measurements**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-43)
    * --2008--Meng P Tan--BMC--**Microarray data mining: A novel optimization-based approach to uncover biologically coherent structures**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-268)
    * --2008--Miguel Rocha--BMC--**Natural computation meta-heuristics for the in silico optimization of microbial strains**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-499)
    * --2008--Iaroslav Ispolatov--BMC--**Detection of the dominant direction of information flow and feedback links in densely interconnected regulatory networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-424)
    * --2008--Yuji Zhang--BMC--**Network motif-based identification of transcription factor-target gene relationships by integrating multi-source biological data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-203)
    * --2008--Philippe Veber--BMC--**Inferring the role of transcription factors in regulatory networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-228)
    * --2008--Nicola Zamboni--BMC--**anNET: a tool for network-embedded thermodynamic analysis of quantitative metabolome data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-199)
    * --2008--Wolfram Gronwald--BMC--**Evolutionary Pareto-optimization of stably folding peptides**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-109)
    * --2008--Cédric Auliac--BMC--**Evolutionary approaches for the reverse-engineering of gene regulatory networks: A study on a biologically realistic dataset**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-91)
    * --2008--Alfredo Braunstein--BMC--**Estimating the size of the solution space of metabolic networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-240)
    * --2008--Rileen Sinha--BMC--**Improved identification of conserved cassette exons using Bayesian networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-477)
    * --2008--Cuong C To--BMC--**Supervised inference of gene-regulatory networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-2)
    * --2008--Grace S Shieh--BMC--**Inferring transcriptional compensation interactions in yeast via stepwise structure equation modeling**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-134)
    * --2008--Chen Wang--BMC--**Motif-directed network component analysis for regulatory network inference**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S1-S21)
    * --2008--Rodrigo Santamaría--BMC--**A visual analytics approach for understanding biclustering results from microarray data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-247)
    * --2008--Ari Rantanen--BMC--**An analytic and systematic framework for estimating metabolic flux ratios from 13C tracer experiments**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-266)
    * --2008--Peter A DiMaggio Jr--BMC--**Biclustering via optimal re-ordering of data matrices in systems biology: rigorous methods and comparative studies**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-458)
    * --2008--Harri T Kiiveri--BMC--**A general approach to simultaneous model fitting and variable elimination in response models for biological data with many more variables than observations**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-195)
    * --2008--Aswath Manoharan--BMC--**Optimizations for the EcoPod field identification tool**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-150)
    * --2008--Yinyin Yuan--BMC--**Partial mixture model for tight clustering of gene expression time-course**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-287)
    * --2008--Katia S Guimarães--BMC--**Interrogating domain-domain interactions with parsimony based approaches**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-171)
    * --2008--Irene Kouskoumvekaki--BMC--**Identification of biomarkers for genotyping Aspergilli using non-linear methods for clustering and classification**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-59)
    * --2008--Marko Tscherepanow--BMC--**An incremental approach to automated protein localisation**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-445)
    * --2008--Silvana Penco--BMC--**New application of intelligent agents in sporadic amyotrophic lateral sclerosis identifies unexpected specific genetic background**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-254)
    * --2008--Yitan Zhu--BMC--**caBIG™ VISDA: Modeling, visualization, and discovery for cluster analysis of genomic data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-383)
    * --2008--Ismael Navas-Delgado--BMC--**AMMO-Prot: amine system project 3D-model finder**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S4-S5)
    * --2008--Cheng-Wei Cheng--BMC--**Predicting RNA-binding sites of proteins using support vector machines and evolutionary information**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S12-S6)
    * --2008--Zhenqiu Liu--BMC--**Gene and pathway identification with L
    p
    penalized Bayesian logistic regression**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-412)
    * --2008--Michael Gormley--BMC--**Expression profiles of switch-like genes accurately classify tissue and infectious disease phenotypes in model-based classification**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-486)
    * --2008--Johannes Tuikkala--BMC--**Missing value imputation improves clustering and interpretation of gene expression microarray data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-202)
    * --2008--Xiangchao Gan--BMC--**Discovering biclusters in gene expression data based on high-dimensional linear geometries**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-209)
    * --2008--Zhenqiang Su--BMC--**Very Important Pool (VIP) genes – an application for microarray-based molecular signatures**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S9-S9)
    * --2008--Wen-Lin Huang--BMC--**ProLoc-GO: Utilizing informative Gene Ontology terms for sequence-based prediction of protein subcellular localization**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-80)
    * --2008--Celine Scornavacca--BMC--**PhySIC_IST: cleaning source trees to infer more informative supertrees**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-413)
    * --2008--Chiara Parravicini--BMC--**GPR17: Molecular modeling and dynamics studies of the 3-D structure and purinergic ligand binding features in comparison with P2Y receptors**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-263)
    * --2008--Rainer Merkl--BMC--**H2r: Identification of evolutionary important residues by means of an entropy based analysis of multiple sequence alignments**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-151)
    * --2008--Markus Bundschus--BMC--**Extraction of semantic biomedical relations from text using conditional random fields**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-207)
    * --2008--Paulo C Carvalho--BMC--**PatternLab for proteomics: a tool for differential shotgun proteomics**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-316)
    * --2008--Yan Xin Huang--BMC--**Pep-3D-Search: a method for B-cell epitope prediction based on mimotope analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-538)
    * --2008--Paul D Yoo--BMC--**SiteSeek: Post-translational modification analysis using adaptive locality-effective kernel methods and new profiles**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-272)
    * --2008--Huiling Xiong--BMC--**Using Generalized Procrustes Analysis (GPA) for normalization of cDNA microarray data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-25)
    * --2008--Kieran O'Neill--BMC--**OntoDas – a tool for facilitating the construction of complex queries to the Gene Ontology**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-437)
    * --2008--Madhavi Ganapathiraju--BMC--**Transmembrane helix prediction using amino acid property features and latent semantic analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S1-S4)
    * --2008--Pelin Armutlu--BMC--**Classification of drug molecules considering their IC50 values using mixed-integer linear programming based hyper-boxes method**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-411)
    * --2008--Thierry Rème--BMC--**A new method for class prediction based on signed-rank algorithms applied to Affymetrix® microarray experiments**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-16)
    * --2008--AKMA Baten--BMC--**Fast splice site detection using information content and feature reduction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S12-S8)
    * --2008--Ce Zheng--BMC--**Prediction of beta-turns at over 80% accuracy based on an ensemble of predicted secondary structures and multiple alignments**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-430)
    * --2008--Lianyi Han--BMC--**Developing and validating predictive decision tree models from mining chemical structural fingerprints and high–throughput screening data in PubChem**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-401)
    * --2008--Gregory A Wilkin--BMC--**A practical comparison of two K-Means clustering algorithms**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S6-S19)
    * --2008--Nan Li--BMC--**Prediction of protein-protein binding site by using core interface residue and support vector machine**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-553)
    * --2008--Maris Lapins--BMC--**Proteochemometric modeling of HIV protease susceptibility**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-181)
    * --2008--Rekin's Janky--BMC--**Evaluation of phylogenetic footprint discovery for predicting bacterial cis-regulatory elements and revealing their evolution**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-37)
    * --2008--Daniel Quest--BMC--**MTAP: The Motif Tool Assessment Platform**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S9-S6)
    * --2008--Dadabhai T Singh--BMC--**Comparative phyloinformatics of virus genes at micro and macro levels in a distributed computing environment**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S1-S23)
    * --2008--Emmanuel Perrodou--BMC--**A new protein linear motif benchmark for multiple sequence alignment software**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-213)
    * --2008--Jonathan R Manning--BMC--**The contrasting properties of conservation and correlated phylogeny in protein functional residue prediction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-51)
    * --2008--Hongya Zhao--BMC--**Multivariate hierarchical Bayesian model for differential gene expression analysis in microarray experiments**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S1-S9)
    * --2008--Hiroyuki Mishima--BMC--**Application of the Linux cluster for exhaustive window haplotype analysis using the FBAT and Unphased programs**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S6-S10)
    * --2008--Bernhard Gschloessl--BMC--**HECTAR: A method to predict subcellular targeting in heterokonts**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-393)
    * --2008--Shen Jean Lim--BMC--**The value of position-specific scoring matrices for assessment of protein allegenicity**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S12-S21)
    * --2008--Yuchen Yang--BMC--**MiRTif: a support vector machine-based microRNA target interaction filter**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S12-S4)
    * --2008--Ramon Diaz-Uriarte--BMC--**SignS: a parallelized, open-source, freely available, web-based tool for gene selection and molecular signatures for survival and censored data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-30)
    * --2008--Firoz Anwar--BMC--**Pol II promoter prediction using characteristic 4-mer motifs: a machine learning approach**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-414)
    * --2008--Guo-Zheng Li--BMC--**Asymmetric bagging and feature selection for activities prediction of drug molecules**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S6-S7)
    * --2008--Michael Richter--BMC--**JCoast – A biologist-centric software tool for data mining and comparison of prokaryotic (meta)genomes**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-177)
    * --2008--Abhishek Tripathi--BMC--**Simple integrative preprocessing preserves what is shared in data sources**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-111)
    * --2008--Catia Pesquita--BMC--**Metrics for GO based protein semantic similarity: a systematic evaluation**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S5-S4)
    * --2008--Yumi Mizuno--BMC--**Predicted mouse peroxisome-targeted proteins and their actual subcellular locations**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S12-S16)
    * --2008--Claudia Chica--BMC--**A tree-based conservation scoring method for short linear motifs in multiple alignments of protein sequences**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-229)
    * --2008--Qingwu Yang--BMC--**Predicting protein folding pathways at the mesoscopic level based on native interactions between secondary structure elements**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-320)
    * --2008--Sabry Razick--BMC--**iRefIndex: A consolidated protein interaction database with provenance**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-405)
    * --2008--Luigi Buonaguro--BMC--**Gene expression profile of peripheral blood mononuclear cells in response to HIV-VLPs stimulation**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-9-S2-S5)
    ## 2007
    * --2007--Reuben Thomas--BMC--**A model-based optimization framework for the inference of regulatory interactions using time-course DNA microarray expression data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-228)
    * --2007--Jean-Philippe Vert--BMC--**A new pairwise kernel for biological network inference with support vector machines**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S10-S8)
    * --2007--Kaname Kojima--BMC--**An efficient grid layout algorithm for biological networks utilizing various biological attributes**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-76)
    * --2007--Anastasia Yartseva--BMC--**Incremental and unifying modelling formalism for biological interaction networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-433)
    * --2007--Xiaohua Hu--BMC--**Mining and state-space modeling and verification of sub-networks from large-scale biomolecular networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-324)
    * --2007--Mudita Singhal--BMC--**A domain-based approach to predict protein-protein interactions**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-199)
    * --2007--Brian J Holden--BMC--**An exploration of alternative visualisations of the basic helix-loop-helix protein interaction network**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-289)
    * --2007--Franck Rapaport--BMC--**Classification of microarray data using gene networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-35)
    * --2007--Markus Bauer--BMC--**Accurate multiple sequence-structure alignment of RNA sequences using combinatorial optimization**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-271)
    * --2007--Florian Markowetz--BMC--**Inferring cellular networks – a review**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S6-S5)
    * --2007--Michael Weitzel--BMC--**The topology of metabolic isotope labeling networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-315)
    * --2007--Jongrae Kim--BMC--**Least-squares methods for identifying biochemical regulatory networks from noisy measurements**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-8)
    * --2007--Rainer Opgen-Rhein--BMC--**Learning causal networks from systems biology time course data: an effective model selection procedure for the vector autoregressive process**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S2-S3)
    * --2007--Tom Michoel--BMC--**Validating module network learning algorithms using simulated data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S2-S5)
    * --2007--Aless--BMC--**Dynamic simulation of regulatory networks using SQUAD**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-462)
    * --2007--Marco Vilela--BMC--**Automated smoother for the numerical decoupling of dynamics models**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-305)
    * --2007--Chang Sik Kim--BMC--**Bayesian Orthogonal Least Squares (BOLS) algorithm for reverse engineering of gene regulatory networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-251)
    * --2007--Doheon Lee--BMC--**BioCAD: an information fusion platform for bio-network inference and analysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S9-S2)
    * --2007--Suman Datta--BMC--**Accelerated search for biomolecular network models to interpret high-throughput experimental data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-258)
    * --2007--Carlos Rodríguez-Penagos--BMC--**Automatic reconstruction of a bacterial regulatory network using Natural Language Processing**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-293)
    * --2007--Blaise Gassend--BMC--**Learning biophysically-motivated parameters for alpha helix prediction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S5-S3)
    * --2007--Ana P Teixeira--BMC--**Hybrid elementary flux analysis/nonparametric modeling: application for bioprocess control**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-30)
    * --2007--Matthew Yeung--BMC--**Estimation of the number of extreme pathways for metabolic networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-363)
    * --2007--Eunkyoung Jung--BMC--**Artificial neural network models for prediction of intestinal permeability of oligopeptides**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-245)
    * --2007--Guillaume Launay--BMC--**Recognizing protein–protein interfaces with empirical potentials and reduced amino acid alphabets**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-270)
    * --2007--Hon Nian Chua--BMC--**Using indirect protein interactions for the prediction of Gene Ontology functions**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S4-S8)
    * --2007--Johannes J M--BMC--**Modeling biochemical transformation processes and information processing with Narrator**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-103)
    * --2007--Markus Koschorreck--BMC--**Reduced modeling of signal transduction – a modular approach**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-336)
    * --2007--Shawn Cokus--BMC--**An improved method for identifying functionally linked proteins using phylogenetic profiles**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S4-S7)
    * --2007--Holger Froehlich--BMC--**Large scale statistical inference of signaling pathways from RNAi and microarray data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-386)
    * --2007--Lars Kuepfer--BMC--**Efficient classification of complete parameter regions based on semidefinite programming**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-12)
    * --2007--Kyoung-Jae Won--BMC--**An evolutionary method for learning HMM structure: prediction of protein secondary structure**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-357)
    * --2007--Jingjing Ye--BMC--**On the analysis of glycomics mass spectrometry data via the regularized area under the ROC curve**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-477)
    * --2007--Wataru Fujibuchi--BMC--**Classification of heterogeneous microarray data by maximum entropy kernel**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-267)
    * --2007--Arijit Chakravarty--BMC--**A novel ensemble learning method for de novo computational identification of DNA binding sites**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-249)
    * --2007--Andrew S Peek--BMC--**Improving model predictions for RNA interference activities that use support vector machine regression by combining and filtering features**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-182)
    * --2007--Limin Fu--BMC--**FLAME, a novel fuzzy clustering method for the analysis of DNA microarray data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-3)
    * --2007--Henning Redestig--BMC--**Transcription factor target prediction using multiple short expression time series from Arabidopsis thaliana**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-454)
    * --2007--Lars Gidskehaug--BMC--**A framework for significance analysis of gene expression data using dimension reduction methods**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-346)
    * --2007--Hyunsoo Kim--BMC--**Extracting unrecognized gene relationships from the biomedical literature via matrix factorizations**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S9-S6)
    * --2007--David Talavera--BMC--**A procedure for identifying homologous alternative splicing events**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-260)
    * --2007--Marc Strickert--BMC--**Correlation-maximizing surrogate gene space for visual mining of gene expression patterns in developing barley endosperm tissue**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-165)
    * --2007--Tobias Wittkop--BMC--**Large scale clustering of protein sequences with FORCE -A layout based heuristic for weighted cluster editing**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-396)
    * --2007--Susana Vinga--BMC--**Local Renyi entropic profiles of DNA sequences**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-393)
    * --2007--Yanjun Qi--BMC--**A mixture of feature experts approach for protein-protein interaction prediction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S10-S6)
    * --2007--Malik Yousef--BMC--**Recursive Cluster Elimination (RCE) for classification and feature selection from gene expression data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-144)
    * --2007--Wasinee Rungsarityotin--BMC--**Identifying protein complexes directly from high-throughput TAP data with Markov random fields**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-482)
    * --2007--Chris Thachuk--BMC--**A replica exchange Monte Carlo algorithm for protein folding in the HP model**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-342)
    * --2007--Guangan Hu--BMC--**Selection of long oligonucleotides for gene expression microarrays using weighted rank-sum strategy**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-350)
    * --2007--Sneh Lata--BMC--**Analysis and prediction of antibacterial peptides**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-263)
    * --2007--Corinne Dahinden--BMC--**Penalized likelihood for sparse contingency tables with an application to full-length cDNA libraries**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-476)
    * --2007--Kana Shimizu--BMC--**Predicting mostly disordered proteins by using structure-unknown protein data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-78)
    * --2007--Modan K Das--BMC--**A survey of DNA motif finding algorithms**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S7-S21)
    * --2007--Sören Sonnenburg--BMC--**Accurate splice site prediction using support vector machines**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S10-S7)
    * --2007--Menaka Rajapakse--BMC--**Predicting peptides binding to MHC class II molecules using multi-objective evolutionary algorithms**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-459)
    * --2007--Ryoko Morioka--BMC--**Predicting state transitions in the transcriptome and metabolome using a linear dynamical system model**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-343)
    * --2007--Cornelia Caragea--BMC--**Glycosylation site prediction using ensembles of Support Vector Machine classifiers**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-438)
    * --2007--Meng Wang--BMC--**Context based mixture model for cell phase identification in automated fluorescence microscopy**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-32)
    * --2007--Andrea Splendiani--BMC--**The Genopolis Microarray Database**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S1-S21)
    * --2007--Roger A Craig--BMC--**Phylogenetic tree information aids supervised learning for predicting protein-protein interaction based on distance matrices**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-6)
    * --2007--Qiwen Dong--BMC--**Exploiting residue-level and profile-level interface propensities for usage in binding sites prediction of proteins**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-147)
    * --2007--Gabriele A Trombetti--BMC--**Data handling strategies for high throughput pyrosequencers**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S1-S22)
    * --2007--Florian Teichert--BMC--**SABERTOOTH: protein structural alignment based on a vectorial structure representation**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-425)
    * --2007--Harmen J Bussemaker--BMC--**Dissecting complex transcriptional responses using pathway-level scores based on prior information**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S6-S6)
    * --2007--Aki Vehtari--BMC--**A novel Bayesian approach to quantify clinical variables and to determine their spectroscopic counterparts in 1H NMR metabonomic data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-S2-S8)
    * --2007--Chrysi Konstantinou-Kirtay--BMC--**Scoring functions and enrichment: a case study on Hsp90**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-27)
    ## 2006
    * --2006--David J Reiss--BMC--**Integrated biclustering of heterogeneous genome-wide datasets for the inference of global regulatory networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-280)
    * --2006--Yoshiya Matsubara--BMC--**Parameter estimation for stiff equations of biosystems using radial basis function networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-230)
    * --2006--Maria Rodriguez-Fern--BMC--**Novel metaheuristic for parameter estimation in nonlinear dynamic biological systems**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-483)
    * --2006--Bor-Sen Chen--BMC--**On the attenuation and amplification of molecular noise in genetic regulatory networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-52)
    * --2006--Gunnar Rätsch--BMC--**Learning Interpretable SVMs for Biological Sequence Classification**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S1-S9)
    * --2006--I Emrah Nikerel--BMC--**A method for estimation of elasticities in metabolic networks using steady state and dynamic metabolomics data and linlog kinetics**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-540)
    * --2006--Adam A Margolin--BMC--**ARACNE: An Algorithm for the Reconstruction of Gene Regulatory Networks in a Mammalian Cellular Context**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S1-S7)
    * --2006--Tsuyoshi Kato--BMC--**Network-based de-noising improves prediction from microarray data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S1-S4)
    * --2006--Weichun Huang--BMC--**Optimized mixed Markov models for motif identification**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-279)
    * --2006--Ethan G Cerami--BMC--**cPath: open source software for collecting, storing, and querying biological pathways**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-497)
    * --2006--Jianghui Xiong--BMC--**Genome wide prediction of protein function via a generic knowledge discovery approach based on evidence integration**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-268)
    * --2006--Sylvain Brohée--BMC--**Evaluation of clustering algorithms for protein-protein interaction networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-488)
    * --2006--Kazuhito Shida--BMC--**GibbsST: a Gibbs sampling method for motif discovery with enhanced resistance to local optima**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-486)
    * --2006--Chung-Yen Lin--BMC--**Fly-DPI: database of protein interactomes for D. melanogaster in the approach of systems biology**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S5-S18)
    * --2006--Srinka Ghosh--BMC--**Rank-statistics based enrichment-site prediction algorithm developed for chromatin immunoprecipitation on chip experiments**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-434)
    * --2006--Br--BMC--**Spectral embedding finds meaningful (relevant) structure in image and microarray data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-74)
    * --2006--Jean-Marc Schwartz--BMC--**Quantitative elementary mode analysis of metabolic pathways: the example of yeast glycolysis**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-186)
    * --2006--Rakesh Kaundal--BMC--**Machine learning techniques in disease forecasting: a case study on rice blast prediction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-485)
    * --2006--Rainer König--BMC--**Discovering functional gene expression patterns in the metabolic network of Escherichia coli with wavelets transforms**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-119)
    * --2006--Monica Chagoyen--BMC--**Discovering semantic features in the literature: a foundation for building functional associations**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-41)
    * --2006--Ling Qin--BMC--**An improved ant colony algorithm with diversified solutions based on the immune strategy**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S4-S3)
    * --2006--Alison A Motsinger--BMC--**GPNN: Power studies and applications of a neural network method for detecting gene-gene interactions in studies of human disease**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-39)
    * --2006--Susmita Datta--BMC--**Evaluation of clustering algorithms for gene expression data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S4-S17)
    * --2006--Chung-Tsai Su--BMC--**Protein disorder prediction by condensed PSSM considering propensity for order or disorder**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-319)
    * --2006--Kenzie D MacIsaac--BMC--**An improved map of conserved regulatory sites for Saccharomyces cerevisiae**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-113)
    * --2006--Hongbo Zhu--BMC--**NOXclass: prediction of protein-protein interaction types**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-27)
    * --2006--Lawrence JK Wee--BMC--**SVM-based prediction of caspase substrate cleavage sites**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S5-S14)
    * --2006--Ester Vilaprinyo--BMC--**Use of physiological constraints to identify quantitative design principles for gene expression in yeast adaptation to heat shock**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-184)
    * --2006--AKMA Baten--BMC--**Splice site identification using probabilistic parameters and SVM classification**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S5-S15)
    * --2006--Xiuzhen Huang--BMC--**Maximum common subgraph: some upper bound and lower bound results**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S4-S6)
    * --2006--Yian A Chen--BMC--**A multivariate prediction model for microarray cross-hybridization**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-101)
    * --2006--Bartek Wilczyński--BMC--**Using local gene expression similarities to discover regulatory binding site modules**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-505)
    * --2006--Chaoyang Zhang--BMC--**Parallelization of multicategory support vector machines (PMC-SVM) for classifying microarray data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S4-S15)
    * --2006--Kazuharu Arakawa--BMC--**GEM System: automatic prototyping of cell-wide metabolic pathway models from genomes**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-168)
    * --2006--Mikael Bodén--BMC--**Prediction of protein continuum secondary structure with probabilistic models based on NMR solved structures**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-68)
    * --2006--Stephen Winters-Hil--BMC--**Nanopore Detector based analysis of single-molecule conformational kinetics and binding interactions**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S2-S21)
    * --2006--Alex--BMC--**Genome BLAST distance phylogenies inferred from whole plastid and whole mitochondrion genome sequences**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-350)
    * --2006--Stefano Lise--BMC--**Docking protein domains in contact space**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-310)
    * --2006--Natalia V Petrova--BMC--**Prediction of catalytic residues using Support Vector Machine with selected protein sequence and structural properties**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-312)
    * --2006--Martin Hoffmann--BMC--**Robust computational reconstitution – a new method for the comparative analysis of gene expression in tissues and isolated cell fractions**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-369)
    * --2006--Prabakaran Subramani--BMC--**Feature selection using Haar wavelet power spectrum**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-432)
    * --2006--Youping Deng--BMC--**Development of computations in bioscience and bioinformatics and its application: review of the Symposium of Computations in Bioinformatics and Bioscience (SCBB06)**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S4-S1)
    * --2006--Kang Peng--BMC--**Length-dependent prediction of protein intrinsic disorder**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-208)
    * --2006--Yu Xue--BMC--**PPSP: prediction of PK-specific phosphorylation site with Bayesian decision theory**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-163)
    * --2006--Pantelis G Bagos--BMC--**Algorithms for incorporating prior topological information in HMMs: application to transmembrane proteins**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-189)
    * --2006--Maksim V Plikus--BMC--**PubFocus: semantic MEDLINE/PubMed citations analytics through integration of controlled biomedical dictionaries and ranking algorithm**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-424)
    * --2006--Xian Wang--BMC--**Missing value estimation for DNA microarray gene expression data by Support Vector Regression imputation and orthogonal coding scheme**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-32)
    * --2006--Jianjun Hu--BMC--**EMD: an ensemble algorithm for discovering regulatory motifs in DNA sequences**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-342)
    * --2006--Teresa Haynes--BMC--**A quantitative analysis of secondary RNA structure using domination based parameters on trees**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-108)
    * --2006--Manju R Mamtani--BMC--**A simple method to combine multiple molecular biomarkers for dichotomous diagnostic classification**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-442)
    * --2006--Jonathan D Wren--BMC--**Proceedings of the Third Annual Conference of the MidSouth Computational Biology and Bioinformatics Society**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S2-S1)
    * --2006--Denis C Bauer--BMC--**STAR: predicting recombination sites from amino acid sequence**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-437)
    * --2006--Feng Li--BMC--**Construction of a nasopharyngeal carcinoma 2D/MS repository with Open Source XML Database – Xindice**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-13)
    * --2006--Jesper Salomon--BMC--**Predicting Class II MHC-Peptide binding: a kernel based approach using similarity scores**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-501)
    * --2006--Scott Montgomerie--BMC--**Improving the accuracy of protein secondary structure prediction using structural alignment**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-301)
    * --2006--Jiangning Song--BMC--**Predicting residue-wise contact orders in proteins by support vector regression**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-425)
    * --2006--Peter L Chang--BMC--**Structure alignment based on coding of local geometric measures**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-346)
    * --2006--Stephan Waack--BMC--**Score-based prediction of genomic islands in prokaryotic genomes using hidden Markov models**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-142)
    * --2006--Pufeng Du--BMC--**Prediction of protein submitochondria locations by hybridizing pseudo-amino acid composition with various physicochemical features of segmented sequence**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-518)
    * --2006--Mehdi Pirooznia--BMC--**SVM Classifier – a comprehensive java interface for support vector machine classification of microarray data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-S4-S25)
    * --2006--Cheong Xin Chan--BMC--**Detecting recombination in evolving nucleotide sequences**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-412)
    * --2006--Ingolf Sommer--BMC--**Improving the quality of protein structure models by selecting from alignment alternatives**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-7-364)
    ## 2005
    * --2005--Kapil G Gadkar--BMC--**Iterative approach to model identification of biological networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-155)
    * --2005--Barbara Di Camillo--BMC--**A quantization method based on threshold optimization for microarray short time series**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-S4-S11)
    * --2005--Dongxiao Zhu--BMC--**Structural comparison of metabolic networks in selected single cell organisms**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-8)
    * --2005--Robert G Beiko--BMC--**GANN: Genetic algorithm neural networks for the detection of conserved combinations of features in DNA**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-36)
    * --2005--AM Cohen--BMC--**Using co-occurrence network structure to extract synonymous gene and protein names from MEDLINE abstracts**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-103)
    * --2005--Kiran Raosaheb Patil--BMC--**Evolutionary programming as a platform for in silico metabolic engineering**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-308)
    * --2005--Xiaoxing Liu--BMC--**An Entropy-based gene selection method for cancer classification using microarray data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-76)
    * --2005--Markey C McNutt--BMC--**Human promoter genomic composition demonstrates non-random groupings that reflect general cellular function**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-259)
    * --2005--Nicola Zamboni--BMC--**FiatFlux – a software for metabolic flux analysis from 13C-glucose experiments**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-209)
    * --2005--Chin-Jen Ku--BMC--**The distance-profile representation and its application to detection of distantly related protein families**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-282)
    * --2005--Blaise TF Alako--BMC--**CoPub Mapper: mining MEDLINE based on search term co-publication**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-51)
    * --2005--Michael Maurer--BMC--**MARS: Microarray analysis, retrieval, and storage system**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-101)
    * --2005--Yunlong Liu--BMC--**Principal component analysis for predicting transcription-factor binding motifs from array-derived data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-276)
    * --2005--Claudio Lottaz--BMC--**stam – a Bioconductor compliant R package for structured analysis of microarray data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-211)
    * --2005--Giuliano Armano--BMC--**A Hybrid Genetic-Neural System for Predicting Protein Secondary Structure**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-S4-S3)
    * --2005--Piero Fariselli--BMC--**A new decoding algorithm for hidden Markov models improves the prediction of the topology of all-beta membrane proteins**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-S4-S12)
    * --2005--Jannick Dyrløv Bendtsen--BMC--**Prediction of twin-arginine signal peptides**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-6-167)
    ## 2004
    * --2004--Hao Chen--BMC--**Content-rich biological network constructed by mining PubMed abstracts**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-147)
    * --2004--Christine Brun--BMC--**Clustering proteins from interaction networks for the prediction of cellular functions**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-95)
    * --2004--David Adalsteinsson--BMC--**Biochemical Network Stochastic Simulator (BioNetS): software for stochastic modeling of biochemical networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-24)
    * --2004--Bahrad A Sokhansanj--BMC--**Linear fuzzy gene network models obtained from microarray data by exhaustive search**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-108)
    * --2004--Peter Meinicke--BMC--**Oligo kernels for datamining on biological sequences: a case study on prokaryotic translation initiation sites**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-169)
    * --2004--John A Berger--BMC--**Optimized LOWESS normalization parameter selection for DNA microarray data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-194)
    * --2004--Yasuyuki Tomita--BMC--**Artificial neural network approach for selection of susceptible single nucleotide polymorphisms and construction of prediction model on childhood allergic asthma**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-120)
    * --2004--William H Majoros--BMC--**An empirical analysis of training protocols for probabilistic gene finders**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-206)
    * --2004--Kayvan Najarian--BMC--**A novel Mixture Model Method for identification of differentially expressed genes from DNA microarray data**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-201)
    * --2004--Yvan Saeys--BMC--**Feature selection for splice site prediction: A new method using EDA-based feature ranking**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-64)
    * --2004--Gustavo Camps-Valls--BMC--**Profiled support vector machines for antisense oligonucleotide efficacy prediction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-135)
    * --2004--Arunachalam Vinayagam--BMC--**Applying Support Vector Machines for Gene ontology based gene function prediction**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-116)
    * --2004--Qinghua Cui--BMC--**Esub8: A novel tool to predict protein subcellular localizations in eukaryotic organisms**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-66)
    * --2004--Drew Lett--BMC--**Interaction profile-based protein classification of death domain**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-75)
    * --2004--Remko de Knikker--BMC--**A web services choreography scenario for interoperating bioinformatics applications**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-25)
    * --2004--Rainer Merkl--BMC--**SIGI: score-based identification of genomic islands**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-22)
    ## 2003
    * --2003--Gary D Bader--BMC--**An automated method for finding molecular complexes in large protein interaction networks**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-4-2)
    * --2003--Marylyn D Ritchie--BMC--**Optimizationof neural network architecture using genetic programming improvesdetection and modeling of gene-gene interactions in studies of humandiseases**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-4-28)
    * --2003--Robert M Hubley--BMC--**Evolutionary algorithms for the selection of single nucleotide polymorphisms**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-4-30)
    * --2003--Samuel Kaski--BMC--**Trustworthiness and metrics in visualizing similarity of gene expression**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-4-48)
    ## 2002
    * --2002--Lan Ma--BMC--**Quantifying robustness of biochemical network models**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-3-38)
    * --2002--Pierre Dönnes--BMC--**Prediction of MHC class I binding peptides, using SVMHC**-- [PDF](https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-3-25)
